// phosphor-react-native/lib/module/bold/Alarm.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Alarm = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "alarm-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 28a100 100 0 1 0 100 100A100.11 100.11 0 0 0 128 28Zm0 176a76 76 0 1 1 76-76 76.08 76.08 0 0 1-76 76ZM32.49 64.49a12 12 0 1 1-17-17l32-32a12 12 0 1 1 17 17Zm208 0a12 12 0 0 1-17 0l-32-32a12 12 0 0 1 17-17l32 32a12 12 0 0 1 0 17ZM176 116a12 12 0 0 1 0 24h-48a12 12 0 0 1-12-12V80a12 12 0 0 1 24 0v36Z"
}));
export default Alarm;
