// phosphor-react-native/lib/module/bold/ArticleMedium.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArticleMedium = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "article-medium-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M60 132a12 12 0 0 1-12 12H24a12 12 0 0 1 0-24h4V64h-4a12 12 0 0 1 0-24h16a12 12 0 0 1 10.18 5.63L80 93.36l29.82-47.72A12 12 0 0 1 120 40h16a12 12 0 0 1 0 24h-4v56h4a12 12 0 0 1 0 24h-24a12 12 0 0 1-4-23.3V93.84l-17.82 28.52a12 12 0 0 1-20.36 0L52 93.84v26.86a12 12 0 0 1 8 11.3Zm116-28h64a12 12 0 0 0 0-24h-64a12 12 0 0 0 0 24Zm64 16h-64a12 12 0 0 0 0 24h64a12 12 0 0 0 0-24Zm0 40H72a12 12 0 0 0 0 24h168a12 12 0 0 0 0-24Zm0 40H72a12 12 0 0 0 0 24h168a12 12 0 0 0 0-24Z"
}));
export default ArticleMedium;
