// phosphor-react-native/lib/module/bold/BatteryVerticalHigh.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BatteryVerticalHigh = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "battery-vertical-high-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M92 12a12 12 0 0 1 12-12h48a12 12 0 0 1 0 24h-48a12 12 0 0 1-12-12Zm112 48v168a28 28 0 0 1-28 28H80a28 28 0 0 1-28-28V60a28 28 0 0 1 28-28h96a28 28 0 0 1 28 28Zm-24 0a4 4 0 0 0-4-4H80a4 4 0 0 0-4 4v168a4 4 0 0 0 4 4h96a4 4 0 0 0 4-4Zm-24 52h-56a12 12 0 0 0 0 24h56a12 12 0 0 0 0-24Zm0 40h-56a12 12 0 0 0 0 24h56a12 12 0 0 0 0-24Zm0 40h-56a12 12 0 0 0 0 24h56a12 12 0 0 0 0-24Z"
}));
export default BatteryVerticalHigh;
