// phosphor-react-native/lib/module/bold/BatteryHigh.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BatteryHigh = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "battery-high-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M196 52H28A28 28 0 0 0 0 80v96a28 28 0 0 0 28 28h168a28 28 0 0 0 28-28V80a28 28 0 0 0-28-28Zm4 124a4 4 0 0 1-4 4H28a4 4 0 0 1-4-4V80a4 4 0 0 1 4-4h168a4 4 0 0 1 4 4Zm-56-72v48a12 12 0 0 1-24 0v-48a12 12 0 0 1 24 0Zm-40 0v48a12 12 0 0 1-24 0v-48a12 12 0 0 1 24 0Zm-40 0v48a12 12 0 0 1-24 0v-48a12 12 0 0 1 24 0Zm192 0v48a12 12 0 0 1-24 0v-48a12 12 0 0 1 24 0Z"
}));
export default BatteryHigh;
