// phosphor-react-native/lib/module/bold/BatteryCharging.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BatteryCharging = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "battery-charging-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M196 52H28A28 28 0 0 0 0 80v96a28 28 0 0 0 28 28h168a28 28 0 0 0 28-28V80a28 28 0 0 0-28-28Zm4 124a4 4 0 0 1-4 4H28a4 4 0 0 1-4-4V80a4 4 0 0 1 4-4h168a4 4 0 0 1 4 4Zm56-72v48a12 12 0 0 1-24 0v-48a12 12 0 0 1 24 0Zm-113.62 18a12 12 0 0 1 0 12l-16 28a12 12 0 1 1-20.84-11.9l5.78-10.1H100a12 12 0 0 1-10.42-18l16-28a12 12 0 0 1 20.84 12l-5.74 10H132a12 12 0 0 1 10.38 6Z"
}));
export default BatteryCharging;
