// phosphor-react-native/lib/module/bold/BluetoothSlash.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BluetoothSlash = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "bluetooth-slash-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m216.88 207.93-160-176a12 12 0 1 0-17.76 16.14L107 122.75 48.8 166.4a12 12 0 1 0 14.4 19.2L108 152v72a12 12 0 0 0 19.2 9.6l47.91-35.94 24 26.41a12 12 0 0 0 17.76-16.14ZM132 200v-48l5 3.77 21.87 24.06ZM108 59.74V32a12 12 0 0 1 19.2-9.6l64 48a12 12 0 0 1 0 19.2l-27.1 20.33a12 12 0 0 1-14.4-19.2L164 80l-32-24v3.74a12 12 0 0 1-24 0Z"
}));
export default BluetoothSlash;
