// phosphor-react-native/lib/module/bold/Brandy.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Brandy = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "brandy-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M228 88a99.63 99.63 0 0 0-16.18-54.55 12 12 0 0 0-10-5.45H54.23a12 12 0 0 0-10 5.45A99.63 99.63 0 0 0 28 88a100.15 100.15 0 0 0 88 99.28V212H88a12 12 0 0 0 0 24h80a12 12 0 0 0 0-24h-28v-24.72A100.15 100.15 0 0 0 228 88ZM61.05 52H195a75.43 75.43 0 0 1 8.1 24H53a75.43 75.43 0 0 1 8.05-24Zm67 112a76.12 76.12 0 0 1-75-64H203a76.12 76.12 0 0 1-75 64Z"
}));
export default Brandy;
