// phosphor-react-native/lib/module/bold/BriefcaseMetal.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BriefcaseMetal = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "briefcase-metal-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 52h-36V40a28 28 0 0 0-28-28h-48a28 28 0 0 0-28 28v12H40a20 20 0 0 0-20 20v128a20 20 0 0 0 20 20h176a20 20 0 0 0 20-20V72a20 20 0 0 0-20-20ZM44 116h168v40H44Zm56-76a4 4 0 0 1 4-4h48a4 4 0 0 1 4 4v12h-56Zm112 36v16H44V76ZM44 196v-16h168v16Z"
}));
export default BriefcaseMetal;
