// phosphor-react-native/lib/module/bold/Buildings.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Buildings = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "buildings-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M240 204h-12V96a20 20 0 0 0-20-20h-36V32a20 20 0 0 0-28.45-18.12l-104 48.54A20.06 20.06 0 0 0 28 80.55V204H16a12 12 0 0 0 0 24h224a12 12 0 0 0 0-24Zm-36-104v104h-32V100ZM52 83.09l96-44.79V204H52ZM132 112v12a12 12 0 0 1-24 0v-12a12 12 0 0 1 24 0Zm-40 0v12a12 12 0 0 1-24 0v-12a12 12 0 0 1 24 0Zm0 52v12a12 12 0 0 1-24 0v-12a12 12 0 0 1 24 0Zm40 0v12a12 12 0 0 1-24 0v-12a12 12 0 0 1 24 0Z"
}));
export default Buildings;
