// phosphor-react-native/lib/module/bold/Calculator.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Calculator = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "calculator-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M200 20H56a20 20 0 0 0-20 20v176a20 20 0 0 0 20 20h144a20 20 0 0 0 20-20V40a20 20 0 0 0-20-20Zm-4 192H60V44h136ZM80 76a12 12 0 0 1 12-12h72a12 12 0 0 1 0 24H92a12 12 0 0 1-12-12Zm40 52a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm48 0a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm-48 48a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm48 0a16 16 0 1 1-16-16 16 16 0 0 1 16 16Z"
}));
export default Calculator;
