// phosphor-react-native/lib/module/bold/Cardholder.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Cardholder = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cardholder-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 44H48a28 28 0 0 0-28 28v112a28 28 0 0 0 28 28h160a28 28 0 0 0 28-28V72a28 28 0 0 0-28-28ZM48 68h160a4 4 0 0 1 4 4v16h-44.81a20 20 0 0 0-19.59 16 20 20 0 0 1-39.2 0 20 20 0 0 0-19.59-16H44V72a4 4 0 0 1 4-4Zm160 120H48a4 4 0 0 1-4-4v-72h41.66a44 44 0 0 0 84.68 0H212v72a4 4 0 0 1-4 4Z"
}));
export default Cardholder;
