// phosphor-react-native/lib/module/bold/CarSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CarSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "car-simple-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M240 108h-8.2l-26.72-60.12A20 20 0 0 0 186.8 36H69.2a20 20 0 0 0-18.28 11.88L24.2 108H16a12 12 0 0 0 0 24h4v76a20 20 0 0 0 20 20h24a20 20 0 0 0 20-20v-12h88v12a20 20 0 0 0 20 20h24a20 20 0 0 0 20-20v-76h4a12 12 0 0 0 0-24ZM71.8 60h112.4l21.33 48H50.47ZM212 204h-16v-20a12 12 0 0 0-12-12H72a12 12 0 0 0-12 12v20H44v-72h168Z"
}));
export default CarSimple;
