// phosphor-react-native/lib/module/bold/CastleTurret.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CastleTurret = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "castle-turret-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M200 20H56a20 20 0 0 0-20 20v44.69a19.86 19.86 0 0 0 5.86 14.14L52 109v107a20 20 0 0 0 20 20h112a20 20 0 0 0 20-20V109l10.14-10.14A19.86 19.86 0 0 0 220 84.69V40a20 20 0 0 0-20-20Zm-60 192h-24v-60a12 12 0 0 1 24 0Zm56-129-10.14 10.17a19.86 19.86 0 0 0-5.86 14.14V212h-16v-60a36 36 0 0 0-72 0v60H76V107.31a19.86 19.86 0 0 0-5.86-14.14L60 83V44h28v24a12 12 0 0 0 24 0V44h32v24a12 12 0 0 0 24 0V44h28Z"
}));
export default CastleTurret;
