// phosphor-react-native/lib/module/bold/ChalkboardSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChalkboardSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chalkboard-simple-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M20 160V56a20 20 0 0 1 20-20h176a20 20 0 0 1 20 20v104a12 12 0 0 1-24 0V60H44v100a12 12 0 0 1-24 0Zm232 40a12 12 0 0 1-12 12H16a12 12 0 0 1 0-24h92v-28a12 12 0 0 1 12-12h64a12 12 0 0 1 12 12v28h44a12 12 0 0 1 12 12Zm-120-12h40v-16h-40Z"
}));
export default ChalkboardSimple;
