// phosphor-react-native/lib/module/bold/ChartScatter.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChartScatter = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chart-scatter-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M236 208a12 12 0 0 1-12 12H32a12 12 0 0 1-12-12V48a12 12 0 0 1 24 0v148h180a12 12 0 0 1 12 12Zm-120-60a16 16 0 1 0 16-16 16 16 0 0 0-16 16Zm-8-40a16 16 0 1 0-16-16 16 16 0 0 0 16 16Zm-32 72a16 16 0 1 0-16-16 16 16 0 0 0 16 16Zm96-48a16 16 0 1 0-16-16 16 16 0 0 0 16 16Zm24-40a16 16 0 1 0-16-16 16 16 0 0 0 16 16Zm-24 72a16 16 0 1 0 16-16 16 16 0 0 0-16 16Z"
}));
export default ChartScatter;
