// phosphor-react-native/lib/module/bold/ClosedCaptioning.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ClosedCaptioning = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "closed-captioning-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 44H32a20 20 0 0 0-20 20v128a20 20 0 0 0 20 20h192a20 20 0 0 0 20-20V64a20 20 0 0 0-20-20Zm-4 144H36V68h184ZM52 128a44 44 0 0 1 66-38.11 12 12 0 0 1-12 20.78 20 20 0 1 0 0 34.66 12 12 0 1 1 12 20.77A44 44 0 0 1 52 128Zm80 0a44 44 0 0 1 66-38.11 12 12 0 0 1-12 20.78 20 20 0 1 0 0 34.66 12 12 0 1 1 12 20.77 44 44 0 0 1-66-38.1Z"
}));
export default ClosedCaptioning;
