// phosphor-react-native/lib/module/bold/CloudX.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CloudX = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cloud-x-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M160 36a92.09 92.09 0 0 0-81 48.36A68 68 0 1 0 72 220h88a92 92 0 0 0 0-184Zm0 160H72a44 44 0 0 1-1.82-88A91.86 91.86 0 0 0 68 128a12 12 0 0 0 24 0 68 68 0 1 1 68 68Zm32.49-75.51L177 136l15.52 15.51a12 12 0 0 1-17 17L160 153l-15.51 15.52a12 12 0 1 1-17-17L143 136l-15.52-15.51a12 12 0 1 1 17-17L160 119l15.51-15.52a12 12 0 0 1 17 17Z"
}));
export default CloudX;
