// phosphor-react-native/lib/module/bold/ContactlessPayment.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ContactlessPayment = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "contactless-payment-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M152.58 66.35a130.61 130.61 0 0 1 0 123.3 12 12 0 1 1-21.17-11.3 106.7 106.7 0 0 0 0-100.7 12 12 0 1 1 21.16-11.3Zm-52.22 11.06a12 12 0 0 0-5 16.23 73 73 0 0 1 0 68.72 12 12 0 1 0 21.18 11.28 97 97 0 0 0 0-91.28 12 12 0 0 0-16.18-4.95ZM236 128A108 108 0 1 1 128 20a108.12 108.12 0 0 1 108 108Zm-24 0a84 84 0 1 0-84 84 84.09 84.09 0 0 0 84-84Z"
}));
export default ContactlessPayment;
