// phosphor-react-native/lib/module/bold/CurrencyEth.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CurrencyEth = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "currency-eth-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m225.44 120.59-88-112a12 12 0 0 0-18.88 0l-88 112a12 12 0 0 0 0 14.82l.6.76a3.72 3.72 0 0 0 .44.56l87 110.68a12 12 0 0 0 18.88 0l88-112a12 12 0 0 0-.04-14.82ZM140 50.7l57.12 72.7-57.12 26Zm-24 98.66-57.12-26L116 50.7Zm0 26.37v29.57l-36.15-46Zm24 0 36.15-16.43-36.15 46Z"
}));
export default CurrencyEth;
