// phosphor-react-native/lib/module/bold/Engine.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Engine = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "engine-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M244 112a12 12 0 0 0-12 12v16h-16v-8a20 20 0 0 0-20-20h-11l-34.17-34.14A19.86 19.86 0 0 0 136.69 72H124V56h16a12 12 0 0 0 0-24H84a12 12 0 0 0 0 24h16v16H60a20 20 0 0 0-20 20v48H24v-16a12 12 0 0 0-24 0v56a12 12 0 0 0 24 0v-16h16v12.69a19.86 19.86 0 0 0 5.86 14.14l35.31 35.31A19.86 19.86 0 0 0 95.31 232h41.38a19.86 19.86 0 0 0 14.14-5.86L185 192h11a20 20 0 0 0 20-20v-8h16v16a12 12 0 0 0 24 0v-56a12 12 0 0 0-12-12Zm-52 56h-8.69a19.86 19.86 0 0 0-14.14 5.86L135 208H97l-33-33V96h71l34.14 34.14a19.86 19.86 0 0 0 14.17 5.86H192Z"
}));
export default Engine;
