// phosphor-react-native/lib/module/bold/Equalizer.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Equalizer = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "equalizer-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M80 108a12 12 0 0 1-12 12H24a12 12 0 0 1 0-24h44a12 12 0 0 1 12 12Zm-12 28H24a12 12 0 0 0 0 24h44a12 12 0 0 0 0-24Zm0 40H24a12 12 0 0 0 0 24h44a12 12 0 0 0 0-24Zm82-40h-44a12 12 0 0 0 0 24h44a12 12 0 0 0 0-24Zm0 40h-44a12 12 0 0 0 0 24h44a12 12 0 0 0 0-24Zm38-96h44a12 12 0 0 0 0-24h-44a12 12 0 0 0 0 24Zm44 16h-44a12 12 0 0 0 0 24h44a12 12 0 0 0 0-24Zm0 40h-44a12 12 0 0 0 0 24h44a12 12 0 0 0 0-24Zm0 40h-44a12 12 0 0 0 0 24h44a12 12 0 0 0 0-24Z"
}));
export default Equalizer;
