// phosphor-react-native/lib/module/bold/EscalatorDown.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const EscalatorDown = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "escalator-down-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M167.51 88.49a12 12 0 0 1 17-17L188 75V48a12 12 0 0 1 24 0v27l3.51-3.52a12 12 0 0 1 17 17l-24 24a12 12 0 0 1-17 0ZM244 160v40a20 20 0 0 1-20 20h-52.5a20.08 20.08 0 0 1-14.7-6.44L66.75 116H32a20 20 0 0 1-20-20V56a20 20 0 0 1 20-20h52.5a20.08 20.08 0 0 1 14.7 6.44L189.25 140H224a20 20 0 0 1 20 20Zm-24 4h-32.5a20.08 20.08 0 0 1-14.7-6.44L82.75 60H36v32h32.5a20.08 20.08 0 0 1 14.7 6.44L173.25 196H220Z"
}));
export default EscalatorDown;
