// phosphor-react-native/lib/module/bold/EscalatorUp.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const EscalatorUp = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "escalator-up-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 36h-52.5a20.07 20.07 0 0 0-14.69 6.43L66.75 140H32a20 20 0 0 0-20 20v40a20 20 0 0 0 20 20h52.5a20.07 20.07 0 0 0 14.69-6.43L189.25 116H224a20 20 0 0 0 20-20V56a20 20 0 0 0-20-20Zm-4 56h-32.5a20.07 20.07 0 0 0-14.69 6.43L82.75 196H36v-32h32.5a20.07 20.07 0 0 0 14.69-6.43L173.25 60H220Zm12.49 75.51a12 12 0 0 1-17 17L212 181v27a12 12 0 0 1-24 0v-27l-3.51 3.52a12 12 0 0 1-17-17l24-24a12 12 0 0 1 17 0Z"
}));
export default EscalatorUp;
