// phosphor-react-native/lib/module/bold/FadersHorizontal.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FadersHorizontal = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "faders-horizontal-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M28 80a12 12 0 0 1 12-12h28a12 12 0 0 1 0 24H40a12 12 0 0 1-12-12Zm188 84h-32v-12a12 12 0 0 0-24 0v48a12 12 0 0 0 24 0v-12h32a12 12 0 0 0 0-24Zm-84 0H40a12 12 0 0 0 0 24h92a12 12 0 0 0 0-24Zm-24-48a12 12 0 0 0 12-12V92h96a12 12 0 0 0 0-24h-96V56a12 12 0 0 0-24 0v48a12 12 0 0 0 12 12Z"
}));
export default FadersHorizontal;
