// phosphor-react-native/lib/module/bold/FigmaLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FigmaLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "figma-logo-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M190.15 96A44 44 0 0 0 160 20H88a44 44 0 0 0-30.15 76 43.9 43.9 0 0 0 1.23 65.12A48 48 0 1 0 140 196v-28.83A44 44 0 0 0 190.15 96ZM180 64a20 20 0 0 1-20 20h-20V44h20a20 20 0 0 1 20 20ZM68 64a20 20 0 0 1 20-20h28v40H88a20 20 0 0 1-20-20Zm20 84a20 20 0 0 1 0-40h28v40H88Zm28 48a24 24 0 1 1-24-24h24Zm44-48a20 20 0 1 1 20-20 20 20 0 0 1-20 20Z"
}));
export default FigmaLogo;
