// phosphor-react-native/lib/module/bold/FilmReel.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FilmReel = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "film-reel-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 212h-28.2a107.94 107.94 0 1 0-67.8 24h96a12 12 0 0 0 0-24ZM44 128a84 84 0 1 1 84 84 84.09 84.09 0 0 1-84-84Zm64-44a20 20 0 1 1 20 20 20 20 0 0 1-20-20Zm20 108a20 20 0 1 1 20-20 20 20 0 0 1-20 20Zm24-64a20 20 0 1 1 20 20 20 20 0 0 1-20-20Zm-48 0a20 20 0 1 1-20-20 20 20 0 0 1 20 20Z"
}));
export default FilmReel;
