// phosphor-react-native/lib/module/bold/FolderSimpleMinus.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FolderSimpleMinus = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "folder-simple-minus-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M104 132h48a12 12 0 0 1 0 24h-48a12 12 0 0 1 0-24Zm132-44v112.89A19.13 19.13 0 0 1 216.89 220H40a20 20 0 0 1-20-20V64a20 20 0 0 1 20-20h53.33a20.12 20.12 0 0 1 12 4L132 68h84a20 20 0 0 1 20 20Zm-24 4h-81.33a20.12 20.12 0 0 1-12-4L92 68H44v128h168Z"
}));
export default FolderSimpleMinus;
