// phosphor-react-native/lib/module/bold/FramerLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FramerLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "framer-logo-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M212 96V32a12 12 0 0 0-12-12H56a12 12 0 0 0-8 21l48.44 43H56a12 12 0 0 0-12 12v64a12 12 0 0 0 3.52 8.49l72 72A12 12 0 0 0 140 232v-60h60a12 12 0 0 0 8-21l-48.41-43H200a12 12 0 0 0 12-12Zm-43.56 52H128a12 12 0 0 0-12 12v43l-48-48v-47h55.44ZM188 84h-55.44l-45-40H188Z"
}));
export default FramerLogo;
