// phosphor-react-native/lib/module/bold/GitBranch.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GitBranch = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "git-branch-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M228 64a36 36 0 1 0-48 33.94V104a12 12 0 0 1-12 12H96a35.88 35.88 0 0 0-12 2.06V97.94a36 36 0 1 0-24 0v60.12a36 36 0 1 0 24 0V152a12 12 0 0 1 12-12h72a36 36 0 0 0 36-36v-6.06A36.07 36.07 0 0 0 228 64ZM72 52a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm0 152a12 12 0 1 1 12-12 12 12 0 0 1-12 12ZM192 76a12 12 0 1 1 12-12 12 12 0 0 1-12 12Z"
}));
export default GitBranch;
