// phosphor-react-native/lib/module/bold/GitMerge.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GitMerge = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "git-merge-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 108a36.07 36.07 0 0 0-33.94 24H152a36.16 36.16 0 0 1-28.8-14.4L99.62 86.16A36 36 0 1 0 68 89.94v76.12a36 36 0 1 0 24 0V116l12 16a60.27 60.27 0 0 0 48 24h22.06A36 36 0 1 0 208 108ZM80 44a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm0 168a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm128-56a12 12 0 1 1 12-12 12 12 0 0 1-12 12Z"
}));
export default GitMerge;
