// phosphor-react-native/lib/module/bold/GlobeSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GlobeSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "globe-simple-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 20a108 108 0 1 0 108 108A108.12 108.12 0 0 0 128 20Zm83.13 96h-31.57a155.5 155.5 0 0 0-19.51-65.63A84.23 84.23 0 0 1 211.13 116Zm-110.62 24h55c-2.25 26.69-12 51.46-27.49 69.85-15.56-18.39-25.26-43.16-27.51-69.85Zm0-24c2.25-26.69 11.95-51.46 27.49-69.85 15.54 18.39 25.24 43.16 27.49 69.85ZM96 50.37A155.5 155.5 0 0 0 76.44 116H44.87A84.23 84.23 0 0 1 96 50.37ZM44.87 140h31.57A155.5 155.5 0 0 0 96 205.63 84.23 84.23 0 0 1 44.87 140Zm115.18 65.63A155.5 155.5 0 0 0 179.56 140h31.57a84.23 84.23 0 0 1-51.08 65.63Z"
}));
export default GlobeSimple;
