// phosphor-react-native/lib/module/bold/GlobeStand.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GlobeStand = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "globe-stand-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 172a76 76 0 1 0-76-76 76.08 76.08 0 0 0 76 76Zm0-128a52 52 0 1 1-52 52 52.06 52.06 0 0 1 52-52Zm12 167.38V220h20a12 12 0 0 1 0 24H96a12 12 0 0 1 0-24h20v-8.62A116 116 0 0 1 12 97.12a115.3 115.3 0 0 1 32.29-81.43A12 12 0 1 1 61.6 32.31a92 92 0 0 0 130.09 130.08 12 12 0 1 1 16.62 17.31A115.12 115.12 0 0 1 140 211.38Z"
}));
export default GlobeStand;
