// phosphor-react-native/lib/module/bold/GoogleCardboardLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GoogleCardboardLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "google-cardboard-logo-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 44H32a20 20 0 0 0-20 20v128a20 20 0 0 0 20 20h60.69a19.86 19.86 0 0 0 14.14-5.86L128 185l21.17 21.17a19.86 19.86 0 0 0 14.14 5.83H224a20 20 0 0 0 20-20V64a20 20 0 0 0-20-20Zm-4 144h-55l-22.83-22.83a20 20 0 0 0-28.28 0L91 188H36V68h184ZM82 160a34 34 0 1 0-34-34 34 34 0 0 0 34 34Zm0-44a10 10 0 1 1-10 10 10 10 0 0 1 10-10Zm92 44a34 34 0 1 0-34-34 34 34 0 0 0 34 34Zm0-44a10 10 0 1 1-10 10 10 10 0 0 1 10-10Z"
}));
export default GoogleCardboardLogo;
