// phosphor-react-native/lib/module/bold/GooglePlayLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GooglePlayLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "google-play-logo-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M225.79 110.7 58 14.65a20.24 20.24 0 0 0-20.12.06A19.62 19.62 0 0 0 28 31.84v192.32a19.62 19.62 0 0 0 9.91 17.13 20.22 20.22 0 0 0 20.12.06l167.76-96a19.76 19.76 0 0 0 0-34.6ZM52 203V53l75 75Zm92-58 12.4 12.4-58 33.2ZM98.41 65.43l58 33.2L144 111ZM178 145l-17-17 17-17 29.72 17Z"
}));
export default GooglePlayLogo;
