// phosphor-react-native/lib/module/bold/Gradient.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Gradient = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "gradient-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M20 108a12 12 0 0 1 12-12h76a12 12 0 0 1 0 24H32a12 12 0 0 1-12-12Zm204-12h-76a12 12 0 0 0 0 24h76a12 12 0 0 0 0-24ZM68 136H32a12 12 0 0 0 0 24h36a12 12 0 0 0 0-24Zm156 0h-36a12 12 0 0 0 0 24h36a12 12 0 0 0 0-24ZM96 148a12 12 0 0 0 12 12h40a12 12 0 0 0 0-24h-40a12 12 0 0 0-12 12Zm-44 28H32a12 12 0 0 0 0 24h20a12 12 0 0 0 0-24Zm56 0H92a12 12 0 0 0 0 24h16a12 12 0 0 0 0-24Zm56 0h-16a12 12 0 0 0 0 24h16a12 12 0 0 0 0-24Zm60 0h-20a12 12 0 0 0 0 24h20a12 12 0 0 0 0-24ZM32 80h192a12 12 0 0 0 0-24H32a12 12 0 0 0 0 24Z"
}));
export default Gradient;
