// phosphor-react-native/lib/module/bold/HandPalm.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const HandPalm = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "hand-palm-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M188 84a32 32 0 0 0-8 1V60a32 32 0 0 0-43.21-30A32 32 0 0 0 76 44v1a32 32 0 0 0-40 31v76a92 92 0 0 0 184 0v-36a32 32 0 0 0-32-32Zm8 68a68 68 0 0 1-136 0V76a8 8 0 0 1 16 0v44a12 12 0 0 0 24 0V44a8 8 0 0 1 16 0v68a12 12 0 0 0 24 0V60a8 8 0 0 1 16 0v65.4a52.09 52.09 0 0 0-40 50.6 12 12 0 0 0 24 0 28 28 0 0 1 28-28 12 12 0 0 0 12-12v-20a8 8 0 0 1 16 0Z"
}));
export default HandPalm;
