// phosphor-react-native/lib/module/bold/HeartBreak.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const HeartBreak = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "heart-break-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M178 28a65.6 65.6 0 0 0-46.67 19.33L128 50.66l-3.33-3.33A66 66 0 0 0 12 94c0 72.34 105.81 130.14 110.31 132.57a12 12 0 0 0 11.38 0C138.19 224.14 244 166.34 244 94a66.08 66.08 0 0 0-66-66Zm-5.49 142.36a328.69 328.69 0 0 1-44.51 31.8 328.69 328.69 0 0 1-44.51-31.8C61.82 151.77 36 123.42 36 94a42 42 0 0 1 71.7-29.7l3.3 3.33-7.52 7.52a12 12 0 0 0 0 17l21.7 21.7-13.7 13.69a12 12 0 0 0 17 17l22.18-22.19a12 12 0 0 0 0-17L129 83.63l19.3-19.33A42 42 0 0 1 220 94c0 29.42-25.82 57.77-47.49 76.36Z"
}));
export default HeartBreak;
