// phosphor-react-native/lib/module/bold/IceCream.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const IceCream = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "ice-cream-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M212 86.7a84 84 0 0 0-168 0A28 28 0 0 0 56 140h1l60.54 106a12 12 0 0 0 20.84 0L199 140h1a28 28 0 0 0 12-53.3Zm-84 129.11L84.68 140h26.07L141 193Zm26.86-47L138.39 140h32.93ZM200 116H56a4 4 0 0 1 0-8 12 12 0 0 0 12-12v-8a60 60 0 0 1 120 0v8a12 12 0 0 0 12 12 4 4 0 0 1 0 8Z"
}));
export default IceCream;
