// phosphor-react-native/lib/module/bold/Images.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Images = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "images-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 36H80a20 20 0 0 0-20 20v12H40a20 20 0 0 0-20 20v112a20 20 0 0 0 20 20h144a20 20 0 0 0 20-20v-20h12a20 20 0 0 0 20-20V56a20 20 0 0 0-20-20ZM84 60h128v45.09l-3.23-3.23a20 20 0 0 0-28.28 0L165.31 117l-35.17-35.14a20 20 0 0 0-28.28 0L84 99.72Zm96 136H44V92h16v68a20 20 0 0 0 20 20h100Zm-96-40v-22.34l32-32 40.83 40.83a12 12 0 0 0 17 0l20.83-20.83L212 139v17Z"
}));
export default Images;
