// phosphor-react-native/lib/module/bold/IntersectSquare.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const IntersectSquare = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "intersect-square-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 84h-44V40a12 12 0 0 0-12-12H40a12 12 0 0 0-12 12v120a12 12 0 0 0 12 12h44v44a12 12 0 0 0 12 12h120a12 12 0 0 0 12-12V96a12 12 0 0 0-12-12ZM52 148V52h96v32H96a12 12 0 0 0-12 12v52Zm56-23 23 23h-23Zm40 6-23-23h23Zm56 73h-96v-32h52a12 12 0 0 0 12-12v-52h32Z"
}));
export default IntersectSquare;
