// phosphor-react-native/lib/module/bold/Lightning.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Lightning = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "lightning-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M219.68 117.26a12 12 0 0 0-7.47-8.5l-54.44-20.41 14-70a12 12 0 0 0-20.54-10.54l-112 120a12 12 0 0 0 4.56 19.43l54.44 20.41-14 70a12 12 0 0 0 20.54 10.54l112-120a12 12 0 0 0 2.91-10.93Zm-103.63 83.67 7.72-38.58a12 12 0 0 0-7.56-13.59L69 131.07l70.93-76-7.72 38.58a12 12 0 0 0 7.56 13.59L187 124.93Z"
}));
export default Lightning;
