// phosphor-react-native/lib/module/bold/LightningSlash.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const LightningSlash = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "lightning-slash-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M56.88 31.93a12 12 0 1 0-17.76 16.14L75.9 88.52l-36.67 39.29a12 12 0 0 0 4.56 19.43l54.44 20.41-14 70a12 12 0 0 0 20.54 10.54l58.9-63.11 35.45 39a12 12 0 0 0 17.76-16.14Zm59.17 169 7.72-38.58a12 12 0 0 0-7.56-13.59L69 131.07l23.07-24.73 55.38 60.92Zm-4.64-132.87a12 12 0 0 1-.59-17l40.41-43.25a12 12 0 0 1 20.54 10.54l-14 70 54.44 20.41a12 12 0 0 1 4.56 19.43l-16.82 18a12 12 0 0 1-17.55-16.37l4.58-4.91-47.19-17.69a12 12 0 0 1-7.56-13.59L140 55.07l-11.63 12.41a12 12 0 0 1-16.96.58Z"
}));
export default LightningSlash;
