// phosphor-react-native/lib/module/bold/ListDashes.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ListDashes = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "list-dashes-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M84 64a12 12 0 0 1 12-12h120a12 12 0 0 1 0 24H96a12 12 0 0 1-12-12Zm132 52H96a12 12 0 0 0 0 24h120a12 12 0 0 0 0-24Zm0 64H96a12 12 0 0 0 0 24h120a12 12 0 0 0 0-24ZM56 52H40a12 12 0 0 0 0 24h16a12 12 0 0 0 0-24Zm0 64H40a12 12 0 0 0 0 24h16a12 12 0 0 0 0-24Zm0 64H40a12 12 0 0 0 0 24h16a12 12 0 0 0 0-24Z"
}));
export default ListDashes;
