// phosphor-react-native/lib/module/bold/ListMagnifyingGlass.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ListMagnifyingGlass = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "list-magnifying-glass-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M28 64a12 12 0 0 1 12-12h176a12 12 0 0 1 0 24H40a12 12 0 0 1-12-12Zm12 76h64a12 12 0 0 0 0-24H40a12 12 0 0 0 0 24Zm80 40H40a12 12 0 0 0 0 24h80a12 12 0 0 0 0-24Zm120.49 20.49a12 12 0 0 1-17 0l-18.08-18.08a44 44 0 1 1 17-17l18.08 18.07a12 12 0 0 1 0 17.01ZM184 164a20 20 0 1 0-20-20 20 20 0 0 0 20 20Z"
}));
export default ListMagnifyingGlass;
