// phosphor-react-native/lib/module/bold/MediumLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const MediumLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "medium-logo-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M68 60a68 68 0 1 0 68 68 68.07 68.07 0 0 0-68-68Zm0 112a44 44 0 1 1 44-44 44.05 44.05 0 0 1-44 44ZM184 60c-23.63 0-36 34.21-36 68s12.37 68 36 68 36-34.21 36-68-12.37-68-36-68Zm0 111.87c-3.74-2.16-12-17.09-12-43.87s8.26-41.71 12-43.87c3.74 2.16 12 17.09 12 43.87s-8.26 41.71-12 43.87ZM256 72v112a12 12 0 0 1-24 0V72a12 12 0 0 1 24 0Z"
}));
export default MediumLogo;
