// phosphor-react-native/lib/module/bold/MegaphoneSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const MegaphoneSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "megaphone-simple-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m221.7 82.83-176.1-54A20 20 0 0 0 20 48v144a20 20 0 0 0 19.94 20 20.38 20.38 0 0 0 5.66-.81l78.4-24.05V192a20 20 0 0 0 20 20h32a20 20 0 0 0 20-20v-26.95l25.7-7.88A20.12 20.12 0 0 0 236 138v-36a20.12 20.12 0 0 0-14.3-19.17ZM124 162l-80 24.58V53.42L124 78Zm48 26h-24v-8.22l24-7.36Zm40-53-64 19.63v-69.3L212 105Z"
}));
export default MegaphoneSimple;
