// phosphor-react-native/lib/module/bold/MusicNotesPlus.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const MusicNotesPlus = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "music-notes-plus-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M236 56a12 12 0 0 1-12 12h-12v12a12 12 0 0 1-24 0V68h-12a12 12 0 0 1 0-24h12V32a12 12 0 0 1 24 0v12h12a12 12 0 0 1 12 12Zm-16 68v48a40 40 0 1 1-24-36.65V124a12 12 0 0 1 24 0Zm-24 48a16 16 0 1 0-16 16 16 16 0 0 0 16-16ZM92 121.37V204a40 40 0 1 1-24-36.65V64a12 12 0 0 1 9.09-11.64l52-13a12 12 0 0 1 5.82 23.28L92 73.37v23.26l53.09-13.27a12 12 0 0 1 5.82 23.28ZM68 204a16 16 0 1 0-16 16 16 16 0 0 0 16-16Z"
}));
export default MusicNotesPlus;
