// phosphor-react-native/lib/module/bold/NavigationArrow.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const NavigationArrow = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "navigation-arrow-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M230.7 94.46 54.81 29.21l-.25-.09a20 20 0 0 0-25.44 25.44l.09.25L94.46 230.7A20 20 0 0 0 113.3 244h.35a20 20 0 0 0 18.77-14.12l22.93-74.53 74.53-22.93a20 20 0 0 0 .82-38ZM146.27 133A20 20 0 0 0 133 146.27l-20 65.28L54.8 54.8 211.55 113Z"
}));
export default NavigationArrow;
