// phosphor-react-native/lib/module/bold/NumberSquareEight.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const NumberSquareEight = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "number-square-eight-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 28H48a20 20 0 0 0-20 20v160a20 20 0 0 0 20 20h160a20 20 0 0 0 20-20V48a20 20 0 0 0-20-20Zm-4 176H52V52h152ZM88 152a40 40 0 1 0 67.6-28.91 36 36 0 1 0-55.2 0A39.87 39.87 0 0 0 88 152Zm40 16a16 16 0 1 1 16-16 16 16 0 0 1-16 16Zm-12-68a12 12 0 1 1 12 12 12 12 0 0 1-12-12Z"
}));
export default NumberSquareEight;
