// phosphor-react-native/lib/module/bold/NumberSquareZero.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const NumberSquareZero = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "number-square-zero-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 188c28.26 0 48-24.67 48-60s-19.74-60-48-60-48 24.67-48 60 19.74 60 48 60Zm0-96c23.33 0 24 32.32 24 36s-.67 36-24 36-24-32.32-24-36 .67-36 24-36Zm80-64H48a20 20 0 0 0-20 20v160a20 20 0 0 0 20 20h160a20 20 0 0 0 20-20V48a20 20 0 0 0-20-20Zm-4 176H52V52h152Z"
}));
export default NumberSquareZero;
