// phosphor-react-native/lib/module/bold/NumberCircleZero.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const NumberCircleZero = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "number-circle-zero-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 20a108 108 0 1 0 108 108A108.12 108.12 0 0 0 128 20Zm0 192a84 84 0 1 1 84-84 84.09 84.09 0 0 1-84 84Zm0-144c-28.26 0-48 24.67-48 60s19.74 60 48 60 48-24.67 48-60-19.74-60-48-60Zm0 96c-23.33 0-24-32.32-24-36s.67-36 24-36 24 32.32 24 36-.67 36-24 36Z"
}));
export default NumberCircleZero;
