// phosphor-react-native/lib/module/bold/PaintRoller.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PaintRoller = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "paint-roller-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M232 84h-16V64a20 20 0 0 0-20-20H52a20 20 0 0 0-20 20v20H16a12 12 0 0 0 0 24h16v20a20 20 0 0 0 20 20h144a20 20 0 0 0 20-20v-20h12v43l-97.5 27.8A20.09 20.09 0 0 0 116 198v34a12 12 0 0 0 24 0v-30.95l97.5-27.85A20.09 20.09 0 0 0 252 154v-50a20 20 0 0 0-20-20Zm-40 40H56V68h136Z"
}));
export default PaintRoller;
