// phosphor-react-native/lib/module/bold/PaintBrushHousehold.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PaintBrushHousehold = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "paint-brush-household-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M233.47 22.53a36 36 0 0 0-50.91 0c-.16.15-.3.31-.45.47L130 81l-6.22-6.21a28 28 0 0 0-39.6 0L7.52 151.51a12 12 0 0 0 0 17l80 80a12 12 0 0 0 17 0l76.7-76.7a28 28 0 0 0 0-39.6L175 126l58-52.08c.16-.15.32-.29.47-.45a36 36 0 0 0 0-50.94ZM96 223l-11-11 19.51-19.52a12 12 0 0 0-17-17L68 195l-7-7 19.51-19.52a12 12 0 0 0-17-17L44 171l-11-11 39-39 63 63ZM216.69 56.28l-60.9 54.65a20 20 0 0 0-.78 29l9.2 9.2a4 4 0 0 1 0 5.66L152 167l-63-63 12.2-12.2a4 4 0 0 1 5.66 0L116 101a20 20 0 0 0 29-.78l54.65-60.9a12 12 0 0 1 17 17Z"
}));
export default PaintBrushHousehold;
