// phosphor-react-native/lib/module/bold/PaperPlaneTilt.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PaperPlaneTilt = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "paper-plane-tilt-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M230.14 25.86a20 20 0 0 0-19.57-5.11l-.22.07L18.44 79a20 20 0 0 0-3 37.28l84.32 40 40 84.32a19.81 19.81 0 0 0 18 11.44c.57 0 1.15 0 1.73-.07A19.82 19.82 0 0 0 177 237.56l58.18-191.91a1.42 1.42 0 0 0 .07-.22 20 20 0 0 0-5.11-19.57ZM157 220.92l-33.72-71.19 45.25-45.25a12 12 0 0 0-17-17l-45.25 45.25L35.08 99 210 46Z"
}));
export default PaperPlaneTilt;
