// phosphor-react-native/lib/module/bold/PencilSimpleSlash.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PencilSimpleSlash = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "pencil-simple-slash-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M56.88 31.93a12 12 0 1 0-17.76 16.14l45.64 50.2-50.9 50.9A19.86 19.86 0 0 0 28 163.31V208a20 20 0 0 0 20 20h44.69a19.85 19.85 0 0 0 14.14-5.86l47.43-47.42 44.87 49.35a12 12 0 1 0 17.76-16.14ZM91 204H52v-39l48.92-48.92 37.18 40.89ZM230.15 70.54l-44.69-44.68a20 20 0 0 0-28.28 0l-36 36a12 12 0 0 0 14.55 18.82L175 120l-.54.53a12 12 0 1 0 17 17l38.67-38.67a20 20 0 0 0 .02-28.32ZM192 103l-39-39 18.34-18.34 39 39Z"
}));
export default PencilSimpleSlash;
