// phosphor-react-native/lib/module/bold/PhosphorLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PhosphorLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "phosphor-logo-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M220 96a76.08 76.08 0 0 0-76-76H64a12 12 0 0 0-12 12v128a84.09 84.09 0 0 0 84 84 12 12 0 0 0 12-12v-60.11A76.09 76.09 0 0 0 220 96ZM76 77.81 115.48 148H76Zm48 36.38L84.52 44H124ZM77.22 172H124v46.79A60.18 60.18 0 0 1 77.22 172ZM148 147.83V44.17a52 52 0 0 1 0 103.66Z"
}));
export default PhosphorLogo;
