// phosphor-react-native/lib/module/bold/Playlist.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Playlist = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "playlist-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M28 64a12 12 0 0 1 12-12h176a12 12 0 0 1 0 24H40a12 12 0 0 1-12-12Zm12 76h116a12 12 0 0 0 0-24H40a12 12 0 0 0 0 24Zm68 40H40a12 12 0 0 0 0 24h68a12 12 0 0 0 0-24Zm143.49-52.55a12 12 0 0 1-14.94 8L212 128.13V192a36 36 0 1 1-24-33.94V112a12 12 0 0 1 15.45-11.49l40 12a12 12 0 0 1 8.04 14.94ZM188 192a12 12 0 1 0-12 12 12 12 0 0 0 12-12Z"
}));
export default Playlist;
