// phosphor-react-native/lib/module/bold/Quotes.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Quotes = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "quotes-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M100 52H40a20 20 0 0 0-20 20v64a20 20 0 0 0 20 20h56v4a28 28 0 0 1-28 28 12 12 0 0 0 0 24 52.06 52.06 0 0 0 52-52V72a20 20 0 0 0-20-20Zm-4 80H44V76h52Zm120-80h-60a20 20 0 0 0-20 20v64a20 20 0 0 0 20 20h56v4a28 28 0 0 1-28 28 12 12 0 0 0 0 24 52.06 52.06 0 0 0 52-52V72a20 20 0 0 0-20-20Zm-4 80h-52V76h52Z"
}));
export default Quotes;
