// phosphor-react-native/lib/module/bold/ShieldWarning.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ShieldWarning = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "shield-warning-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 36H48a20 20 0 0 0-20 20v58.8c0 92.36 78.1 123 93.75 128.18a19.63 19.63 0 0 0 12.49 0C149.9 237.78 228 207.16 228 114.8V56a20 20 0 0 0-20-20Zm-4 78.8c0 73.55-60.52 99.52-76 105-15.47-5.42-76-31.39-76-104.95V60h152ZM116 132V96a12 12 0 0 1 24 0v36a12 12 0 0 1-24 0Zm-4 40a16 16 0 1 1 16 16 16 16 0 0 1-16-16Z"
}));
export default ShieldWarning;
