// phosphor-react-native/lib/module/bold/Shower.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Shower = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "shower-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M68 236a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm16-48a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm-64 0a16 16 0 1 0 16 16 16 16 0 0 0-16-16Zm32 0a16 16 0 1 0-16-16 16 16 0 0 0 16 16ZM256 40a12 12 0 0 1-12 12h-23l-25.81 25.79-21.45 125.54a20 20 0 0 1-33.86 10.8l-98-98a20 20 0 0 1 10.84-33.88l125.5-21.44 26.95-26.95A19.86 19.86 0 0 1 219.31 28H244a12 12 0 0 1 12 12Zm-86.68 46.68-105 17.94 87.07 87.07Z"
}));
export default Shower;
