// phosphor-react-native/lib/module/bold/SimCard.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SimCard = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "sim-card-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m216.49 79.51-56-56A12 12 0 0 0 152 20H56a20 20 0 0 0-20 20v176a20 20 0 0 0 20 20h144a20 20 0 0 0 20-20V88a12 12 0 0 0-3.51-8.49ZM196 212H60V44h87l49 49ZM88 112a12 12 0 0 0-12 12v64a12 12 0 0 0 12 12h80a12 12 0 0 0 12-12v-64a12 12 0 0 0-12-12Zm12 24h16v40h-16Zm56 40h-16v-40h16Z"
}));
export default SimCard;
