// phosphor-react-native/lib/module/bold/SketchLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SketchLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "sketch-logo-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m249 96.1-56-64a12 12 0 0 0-9-4.1H72a12 12 0 0 0-9 4.1l-56 64a12 12 0 0 0 .26 16.09l112 120a12 12 0 0 0 17.54 0l112-120a12 12 0 0 0 .2-16.09ZM213.55 92H182l-30-40h26.55ZM71.88 116l21.19 53-49.46-53Zm86.4 0L128 191.69 97.72 116ZM104 92l24-32 24 32Zm80.12 24h28.27l-49.46 53ZM77.45 52H104L74 92H42.45Z"
}));
export default SketchLogo;
