// phosphor-react-native/lib/module/bold/SmileyNervous.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SmileyNervous = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "smiley-nervous-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M184.49 176.49a12 12 0 0 1-17 0L160 169l-7.51 7.52a12 12 0 0 1-17 0L128 169l-7.51 7.52a12 12 0 0 1-17 0L96 169l-7.51 7.52a12 12 0 0 1-17-17l16-16a12 12 0 0 1 17 0L112 151l7.51-7.52a12 12 0 0 1 17 0L144 151l7.51-7.52a12 12 0 0 1 17 0l16 16a12 12 0 0 1-.02 17.01ZM236 128A108 108 0 1 1 128 20a108.12 108.12 0 0 1 108 108Zm-24 0a84 84 0 1 0-84 84 84.09 84.09 0 0 0 84-84Zm-120-4a16 16 0 1 0-16-16 16 16 0 0 0 16 16Zm72 0a16 16 0 1 0-16-16 16 16 0 0 0 16 16Z"
}));
export default SmileyNervous;
