// phosphor-react-native/lib/module/bold/SmileyXEyes.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SmileyXEyes = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "smiley-x-eyes-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 20a108 108 0 1 0 108 108A108.12 108.12 0 0 0 128 20Zm0 192a84 84 0 1 1 84-84 84.09 84.09 0 0 1-84 84Zm60.49-92.49a12 12 0 0 1-17 17L164 129l-7.51 7.52a12 12 0 0 1-17-17L147 112l-7.52-7.51a12 12 0 0 1 17-17L164 95l7.51-7.52a12 12 0 0 1 17 17L181 112Zm-72 17a12 12 0 0 1-17 0L92 129l-7.51 7.52a12 12 0 0 1-17-17L75 112l-7.52-7.51a12 12 0 0 1 17-17L92 95l7.51-7.52a12 12 0 0 1 17 17L109 112l7.52 7.51a12 12 0 0 1-.03 16.98ZM144 180a16 16 0 1 1-16-16 16 16 0 0 1 16 16Z"
}));
export default SmileyXEyes;
