// phosphor-react-native/lib/module/bold/SpeakerSimpleHigh.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SpeakerSimpleHigh = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "speaker-simple-high-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M157.27 21.22a12 12 0 0 0-12.64 1.31L75.88 76H32a20 20 0 0 0-20 20v64a20 20 0 0 0 20 20h43.88l68.75 53.47A12 12 0 0 0 164 224V32a12 12 0 0 0-6.73-10.78ZM140 199.47l-52.63-40.94A12 12 0 0 0 80 156H36v-56h44a12 12 0 0 0 7.37-2.53L140 56.54ZM204 104v48a12 12 0 0 1-24 0v-48a12 12 0 0 1 24 0Zm36-16v80a12 12 0 0 1-24 0V88a12 12 0 0 1 24 0Z"
}));
export default SpeakerSimpleHigh;
