// phosphor-react-native/lib/module/bold/SpotifyLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SpotifyLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "spotify-logo-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 20a108 108 0 1 0 108 108A108.12 108.12 0 0 0 128 20Zm0 192a84 84 0 1 1 84-84 84.09 84.09 0 0 1-84 84Zm66.59-84.36a12 12 0 0 1-16.24 4.93 106.7 106.7 0 0 0-100.7 0A11.83 11.83 0 0 1 72 134a12 12 0 0 1-5.66-22.58 130.61 130.61 0 0 1 123.3 0 12 12 0 0 1 4.95 16.22Zm-16 36a12 12 0 0 1-16.23 5 73 73 0 0 0-68.72 0 12 12 0 0 1-11.28-21.18 97 97 0 0 1 91.28 0 12 12 0 0 1 4.95 16.18Z"
}));
export default SpotifyLogo;
