// phosphor-react-native/lib/module/bold/Storefront.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Storefront = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "storefront-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M236 96a12 12 0 0 0-.44-3.3L221.2 42.51A20.08 20.08 0 0 0 202 28H54a20.08 20.08 0 0 0-19.2 14.51L20.46 92.7A12 12 0 0 0 20 96v16a43.94 43.94 0 0 0 16 33.92V208a20 20 0 0 0 20 20h144a20 20 0 0 0 20-20v-62.08A43.94 43.94 0 0 0 236 112Zm-24 16a20 20 0 0 1-40 0v-4h40Zm-168 0v-4h40v4a20 20 0 0 1-40 0Zm64-4h40v4a20 20 0 0 1-40 0ZM57.05 52H199l9.14 32H47.91ZM196 204H60v-48.19c1.32.12 2.65.19 4 .19a43.86 43.86 0 0 0 32-13.85 43.89 43.89 0 0 0 64 0A43.86 43.86 0 0 0 192 156c1.35 0 2.68-.07 4-.19Z"
}));
export default Storefront;
