// phosphor-react-native/lib/module/bold/Subtract.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Subtract = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "subtract-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M178 78A84 84 0 1 0 78 178 84 84 0 1 0 178 78Zm42 82a60.75 60.75 0 0 1-.38 6.65l-44-44a83.62 83.62 0 0 0 4-19.39A59.83 59.83 0 0 1 220 160ZM36 96a60 60 0 1 1 60 60 60.07 60.07 0 0 1-60-60Zm67.28 83.66a83.62 83.62 0 0 0 19.39-4l44 44a60.75 60.75 0 0 1-6.67.34 59.83 59.83 0 0 1-56.72-40.34Zm88.53 31.18-46.73-46.73a85 85 0 0 0 19-19l46.73 46.73a60.45 60.45 0 0 1-19 19Z"
}));
export default Subtract;
