// phosphor-react-native/lib/module/bold/SunDim.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SunDim = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "sun-dim-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M116 36v-4a12 12 0 0 1 24 0v4a12 12 0 0 1-24 0Zm80 92a68 68 0 1 1-68-68 68.07 68.07 0 0 1 68 68Zm-24 0a44 44 0 1 0-44 44 44.05 44.05 0 0 0 44-44ZM51.51 68.49a12 12 0 1 0 17-17l-4-4a12 12 0 0 0-17 17Zm0 119-4 4a12 12 0 0 0 17 17l4-4a12 12 0 1 0-17-17ZM196 72a12 12 0 0 0 8.49-3.51l4-4a12 12 0 0 0-17-17l-4 4A12 12 0 0 0 196 72Zm8.49 115.51a12 12 0 0 0-17 17l4 4a12 12 0 0 0 17-17ZM48 128a12 12 0 0 0-12-12h-4a12 12 0 0 0 0 24h4a12 12 0 0 0 12-12Zm80 80a12 12 0 0 0-12 12v4a12 12 0 0 0 24 0v-4a12 12 0 0 0-12-12Zm96-92h-4a12 12 0 0 0 0 24h4a12 12 0 0 0 0-24Z"
}));
export default SunDim;
