// phosphor-react-native/lib/module/bold/Syringe.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Syringe = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "syringe-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m240.49 63.51-48-48a12 12 0 0 0-17 17L191 48l-23 23-31.51-31.49a12 12 0 1 0-17 17L123 60l-81.14 81.17A19.86 19.86 0 0 0 36 155.31V203l-20.49 20.51a12 12 0 0 0 17 17L53 220h47.72a19.86 19.86 0 0 0 14.14-5.86L196 133l3.51 3.52a12 12 0 0 0 17-17L185 88l23-23 15.51 15.52a12 12 0 1 0 17-17ZM99 196H60v-39l14-14 17.51 17.52a12 12 0 0 0 17-17L91 126l11-11 17.51 17.52a12 12 0 0 0 17-17L119 98l21-21 39 39Z"
}));
export default Syringe;
