// phosphor-react-native/lib/module/bold/TagChevron.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TagChevron = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "tag-chevron-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m240.63 116.91-42.63-64A19.93 19.93 0 0 0 181.32 44H24a12 12 0 0 0-9.88 18.82l45 65.18-45 65.18A12 12 0 0 0 24 212h157.32a19.93 19.93 0 0 0 16.68-8.91l42.67-64a19.94 19.94 0 0 0-.04-22.18ZM179.18 188H46.87l33.65-48.74a1.63 1.63 0 0 0 .11-.17 19.91 19.91 0 0 0 0-22.18 1.63 1.63 0 0 0-.11-.17L46.87 68h132.31l40 60Z"
}));
export default TagChevron;
