// phosphor-react-native/lib/module/bold/TelegramLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TelegramLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "telegram-logo-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M239.49 23.16a13 13 0 0 0-13.23-2.26L23.6 100.21a18.22 18.22 0 0 0 3.12 34.86L76 144.74V200a20 20 0 0 0 34.4 13.88l22.67-23.51L170.35 223a20 20 0 0 0 32.7-10.54l40.62-176.55a13 13 0 0 0-4.18-12.75Zm-92.08 54.36-62.19 44.57-34.43-6.75ZM100 190.06v-28.71l15 13.15Zm81.16 10.52-73.88-64.77 106.31-76.18Z"
}));
export default TelegramLogo;
