// phosphor-react-native/lib/module/bold/TextAa.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TextAa = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "text-aa-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M90.86 50.89a12 12 0 0 0-21.72 0l-64 136a12 12 0 0 0 21.71 10.22L42.44 164h75.12l15.58 33.11a12 12 0 0 0 21.72-10.22ZM53.74 140 80 84.18 106.27 140ZM200 92c-13.85 0-24.77 3.86-32.45 11.48a12 12 0 1 0 16.9 17c3-3 8.26-4.52 15.55-4.52 9.34 0 17.19 5.16 19.37 12.1A47.32 47.32 0 0 0 200 124c-24.26 0-44 17.94-44 40s19.74 40 44 40a47.18 47.18 0 0 0 22-5.38 12 12 0 0 0 22-6.62v-60c0-22.06-19.74-40-44-40Zm0 88c-11 0-20-7.18-20-16s9-16 20-16 20 7.18 20 16-9 16-20 16Z"
}));
export default TextAa;
