// phosphor-react-native/lib/module/bold/TextHFour.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TextHFour = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "text-h-four-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M256 176a12 12 0 0 1-12 12v20a12 12 0 0 1-24 0v-20h-36a12 12 0 0 1-11.38-15.79l24-72a12 12 0 0 1 22.76 7.58L200.65 164H220v-20a12 12 0 0 1 24 0v20a12 12 0 0 1 12 12ZM144 44a12 12 0 0 0-12 12v48H52V56a12 12 0 0 0-24 0v120a12 12 0 0 0 24 0v-48h80v48a12 12 0 0 0 24 0V56a12 12 0 0 0-12-12Z"
}));
export default TextHFour;
