// phosphor-react-native/lib/module/bold/TextColumns.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TextColumns = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "text-columns-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M120 64a12 12 0 0 1-12 12H40a12 12 0 0 1 0-24h68a12 12 0 0 1 12 12Zm-12 28H40a12 12 0 0 0 0 24h68a12 12 0 0 0 0-24Zm0 40H40a12 12 0 0 0 0 24h68a12 12 0 0 0 0-24Zm0 40H40a12 12 0 0 0 0 24h68a12 12 0 0 0 0-24Zm40-96h68a12 12 0 0 0 0-24h-68a12 12 0 0 0 0 24Zm68 16h-68a12 12 0 0 0 0 24h68a12 12 0 0 0 0-24Zm0 40h-68a12 12 0 0 0 0 24h68a12 12 0 0 0 0-24Zm0 40h-68a12 12 0 0 0 0 24h68a12 12 0 0 0 0-24Z"
}));
export default TextColumns;
