// phosphor-react-native/lib/module/bold/Ticket.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Ticket = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "ticket-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M228 108.4a20 20 0 0 0 16-19.59V64a20 20 0 0 0-20-20H32a20 20 0 0 0-20 20v24.81a20 20 0 0 0 16 19.59 20 20 0 0 1 0 39.2 20 20 0 0 0-16 19.59V192a20 20 0 0 0 20 20h192a20 20 0 0 0 20-20v-24.81a20 20 0 0 0-16-19.59 20 20 0 0 1 0-39.2ZM36 170.34a44 44 0 0 0 0-84.68V68h48v120H36Zm184 0V188H108V68h112v17.66a44 44 0 0 0 0 84.68Z"
}));
export default Ticket;
