// phosphor-react-native/lib/module/bold/Translate.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Translate = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "translate-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m242.73 210.63-56-112a12 12 0 0 0-21.46 0l-20.52 41A84.2 84.2 0 0 1 106 126.22 107.48 107.48 0 0 0 131.33 68H152a12 12 0 0 0 0-24h-52V32a12 12 0 0 0-24 0v12H24a12 12 0 0 0 0 24h83.13A83.69 83.69 0 0 1 88 110.35 84 84 0 0 1 75.6 91a12 12 0 1 0-21.81 10A107.55 107.55 0 0 0 70 126.24 83.54 83.54 0 0 1 24 140a12 12 0 0 0 0 24 107.47 107.47 0 0 0 64-21.07 108.4 108.4 0 0 0 45.39 19.44l-24.13 48.26a12 12 0 1 0 21.46 10.73L143.41 196h65.17l12.68 25.36a12 12 0 1 0 21.47-10.73ZM155.41 172 176 130.83 196.58 172Z"
}));
export default Translate;
