// phosphor-react-native/lib/module/bold/UserGear.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const UserGear = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "user-gear-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M152.5 156.53a72 72 0 1 0-89 0 124 124 0 0 0-48.69 35.75 12 12 0 0 0 18.38 15.44C52.54 184.69 79.11 172 108 172c37 0 61.12 19.42 74.81 35.72a12 12 0 1 0 18.38-15.44 124 124 0 0 0-48.69-35.75ZM60 100a48 48 0 1 1 48 48 48.05 48.05 0 0 1-48-48Zm182.73 41.23-3.31 1.08 2 2.82a12 12 0 1 1-19.41 14.1l-2.01-2.82-2 2.82a12 12 0 1 1-19.41-14.1l2-2.82-3.31-1.08a12 12 0 0 1 7.42-22.82l3.31 1.07V116a12 12 0 0 1 24 0v3.48l3.31-1.07a12 12 0 0 1 7.42 22.82Z"
}));
export default UserGear;
