// phosphor-react-native/lib/module/bold/VinylRecord.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const VinylRecord = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "vinyl-record-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 20a108 108 0 1 0 108 108A108.12 108.12 0 0 0 128 20Zm0 192a84 84 0 1 1 84-84 84.09 84.09 0 0 1-84 84Zm0-124a40 40 0 0 0-40 40 12 12 0 0 1-24 0 64.07 64.07 0 0 1 64-64 12 12 0 0 1 0 24Zm64 40a64.07 64.07 0 0 1-64 64 12 12 0 0 1 0-24 40 40 0 0 0 40-40 12 12 0 0 1 24 0Zm-64 20a20 20 0 1 1 20-20 20 20 0 0 1-20 20Z"
}));
export default VinylRecord;
