// phosphor-react-native/lib/module/bold/WebhooksLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const WebhooksLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "webhooks-logo-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M114.59 188A52 52 0 0 1 12 176a51.54 51.54 0 0 1 10.41-31.21 12 12 0 1 1 19.18 14.41A27.74 27.74 0 0 0 36 176a28 28 0 0 0 56 0 12 12 0 0 1 12-12h88a12 12 0 0 1 0 24Zm-56.88-1.78a12 12 0 0 0 16.51-3.93l43-69.94a12 12 0 0 0-3.93-16.51 28 28 0 1 1 40.34-35.06 12 12 0 1 0 22-9.59 52 52 0 1 0-84.42 57.62l-37.47 60.9a12 12 0 0 0 3.97 16.51ZM192 124a51.89 51.89 0 0 0-16.3 2.62L138.22 65.7a12 12 0 0 0-20.44 12.58l43 69.93a12 12 0 0 0 16.52 3.93A28 28 0 1 1 192 204a12 12 0 0 0 0 24 52 52 0 0 0 0-104Z"
}));
export default WebhooksLogo;
