// phosphor-react-native/lib/module/bold/YinYang.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const YinYang = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "yin-yang-bold__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 20a108 108 0 1 0 108 108A108.12 108.12 0 0 0 128 20ZM44 128a84.09 84.09 0 0 1 84-84 36 36 0 0 1 0 72 60 60 0 0 0-58.81 71.9A83.73 83.73 0 0 1 44 128Zm84 84a36 36 0 0 1 0-72 60 60 0 0 0 58.81-71.9A83.94 83.94 0 0 1 128 212Zm16-36a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm-32-96a16 16 0 1 1 16 16 16 16 0 0 1-16-16Z"
}));
export default YinYang;
