// phosphor-react-native/lib/module/regular/BatteryVerticalHigh.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BatteryVerticalHigh = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "battery-vertical-high__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M88 8a8 8 0 0 1 8-8h64a8 8 0 0 1 0 16H96a8 8 0 0 1-8-8Zm112 48v168a24 24 0 0 1-24 24H80a24 24 0 0 1-24-24V56a24 24 0 0 1 24-24h96a24 24 0 0 1 24 24Zm-16 0a8 8 0 0 0-8-8H80a8 8 0 0 0-8 8v168a8 8 0 0 0 8 8h96a8 8 0 0 0 8-8Zm-24 56H96a8 8 0 0 0 0 16h64a8 8 0 0 0 0-16Zm0 40H96a8 8 0 0 0 0 16h64a8 8 0 0 0 0-16Zm0 40H96a8 8 0 0 0 0 16h64a8 8 0 0 0 0-16Z"
}));
export default BatteryVerticalHigh;
