// phosphor-react-native/lib/module/regular/BracketsAngle.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BracketsAngle = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "brackets-angle__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M86.75 44.3 33.48 128l53.27 83.7a8 8 0 0 1-2.46 11.05A7.91 7.91 0 0 1 80 224a8 8 0 0 1-6.76-3.71l-56-88a8 8 0 0 1 0-8.59l56-88a8 8 0 1 1 13.5 8.59Zm152 79.41-56-88a8 8 0 1 0-13.5 8.59l53.27 83.7-53.27 83.7a8 8 0 0 0 2.46 11.05A7.91 7.91 0 0 0 176 224a8 8 0 0 0 6.76-3.71l56-88a8 8 0 0 0-.01-8.58Z"
}));
export default BracketsAngle;
