// phosphor-react-native/lib/module/regular/BugDroid.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BugDroid = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "bug-droid__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m191.83 43.48 13.83-13.82a8 8 0 1 0-11.32-11.32L179.79 32.9a87.81 87.81 0 0 0-103.58 0L61.66 18.34a8 8 0 0 0-11.32 11.32l13.83 13.82A87.72 87.72 0 0 0 40 104v40a88 88 0 0 0 176 0v-40a87.72 87.72 0 0 0-24.17-60.52ZM128 32a72.08 72.08 0 0 1 72 72v8H56v-8a72.08 72.08 0 0 1 72-72Zm0 184a72.08 72.08 0 0 1-72-72v-16h144v16a72.08 72.08 0 0 1-72 72Zm16-132a12 12 0 1 1 12 12 12 12 0 0 1-12-12Zm-56 0a12 12 0 1 1 12 12 12 12 0 0 1-12-12Z"
}));
export default BugDroid;
