// phosphor-react-native/lib/module/regular/Cardholder.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Cardholder = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cardholder__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 48H48a24 24 0 0 0-24 24v112a24 24 0 0 0 24 24h160a24 24 0 0 0 24-24V72a24 24 0 0 0-24-24ZM40 96h176v16h-48.81a16 16 0 0 0-15.67 12.81 24 24 0 0 1-47 0A16 16 0 0 0 88.81 112H40Zm8-32h160a8 8 0 0 1 8 8v8H40v-8a8 8 0 0 1 8-8Zm160 128H48a8 8 0 0 1-8-8v-56h48.8a40 40 0 0 0 78.39 0H216v56a8 8 0 0 1-8 8Z"
}));
export default Cardholder;
