// phosphor-react-native/lib/module/regular/ChartScatter.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChartScatter = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chart-scatter__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M232 208a8 8 0 0 1-8 8H32a8 8 0 0 1-8-8V48a8 8 0 0 1 16 0v152h184a8 8 0 0 1 8 8Zm-100-48a12 12 0 1 0-12-12 12 12 0 0 0 12 12Zm-24-56a12 12 0 1 0-12-12 12 12 0 0 0 12 12Zm-32 72a12 12 0 1 0-12-12 12 12 0 0 0 12 12Zm96-48a12 12 0 1 0-12-12 12 12 0 0 0 12 12Zm24-40a12 12 0 1 0-12-12 12 12 0 0 0 12 12Zm-20 76a12 12 0 1 0 12-12 12 12 0 0 0-12 12Z"
}));
export default ChartScatter;
