// phosphor-react-native/lib/module/regular/CloudFog.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CloudFog = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cloud-fog__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M120 200H72a8 8 0 0 1 0-16h48a8 8 0 0 1 0 16Zm64-16h-24a8 8 0 0 0 0 16h24a8 8 0 0 0 0-16Zm-24 32h-56a8 8 0 0 0 0 16h56a8 8 0 0 0 0-16Zm72-124a76.08 76.08 0 0 1-76 76H76a52 52 0 0 1 0-104 53.26 53.26 0 0 1 8.92.76A76.08 76.08 0 0 1 232 92Zm-16 0a60.06 60.06 0 0 0-120-3.54 8 8 0 0 1-16-.92q.21-3.66.77-7.23A38.11 38.11 0 0 0 76 80a36 36 0 0 0 0 72h80a60.07 60.07 0 0 0 60-60Z"
}));
export default CloudFog;
