// phosphor-react-native/lib/module/regular/ContactlessPayment.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ContactlessPayment = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "contactless-payment__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm0 192a88 88 0 1 1 88-88 88.1 88.1 0 0 1-88 88ZM97.07 100.26a59.33 59.33 0 0 1 0 55.48 8 8 0 1 1-14.14-7.48 42.79 42.79 0 0 0 0-40.52 8 8 0 0 1 14.14-7.48Zm56-32a126.67 126.67 0 0 1 0 119.54 8 8 0 0 1-14.07-7.57 110.62 110.62 0 0 0 0-104.46 8 8 0 0 1 14.12-7.54Zm-28 16a93 93 0 0 1 0 87.52 8 8 0 1 1-14.12-7.52 77 77 0 0 0 0-72.48 8 8 0 1 1 14.12-7.52Z"
}));
export default ContactlessPayment;
