// phosphor-react-native/lib/module/regular/CursorClick.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CursorClick = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cursor-click__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m169.64 134.33 44.77-19.46a16 16 0 0 0-1.41-29.8L52.92 32.8A16 16 0 0 0 32.8 52.92L85.07 213a15.83 15.83 0 0 0 14.41 11h.79a15.83 15.83 0 0 0 14.6-9.59l19.46-44.77L184 219.31a16 16 0 0 0 22.63 0l12.68-12.68a16 16 0 0 0 0-22.63Zm-69.48 73.76.06-.05Zm95.15-.09-49.66-49.67a16 16 0 0 0-26 4.94l-19.42 44.65L48 48l159.87 52.21-44.64 19.41a16 16 0 0 0-4.94 26L208 195.31ZM88 24v-8a8 8 0 0 1 16 0v8a8 8 0 0 1-16 0ZM8 96a8 8 0 0 1 8-8h8a8 8 0 0 1 0 16h-8a8 8 0 0 1-8-8Zm112.85-67.58 8-16a8 8 0 0 1 14.31 7.16l-8 16a8 8 0 1 1-14.31-7.16Zm-81.69 96a8 8 0 0 1-3.58 10.74l-16 8a8 8 0 0 1-7.16-14.31l16-8a8 8 0 0 1 10.74 3.57Z"
}));
export default CursorClick;
