// phosphor-react-native/lib/module/regular/Detective.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Detective = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "detective__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M248 112h-27.92l-47.5-65.41a16 16 0 0 0-25.31-.72l-12.85 14.9-.2.23a7.95 7.95 0 0 1-12.44 0l-.2-.23-12.85-14.9a16 16 0 0 0-25.31.72L35.92 112H8a8 8 0 0 0 0 16h240a8 8 0 0 0 0-16ZM96.34 56l.19.23 12.85 14.89a24 24 0 0 0 37.24 0l12.85-14.89c.06-.08.1-.15.17-.23l40.66 56H55.69ZM180 144a36 36 0 0 0-35.77 32h-32.46a36 36 0 1 0-1.83 16h36.12A36 36 0 1 0 180 144ZM76 200a20 20 0 1 1 20-20 20 20 0 0 1-20 20Zm104 0a20 20 0 1 1 20-20 20 20 0 0 1-20 20Z"
}));
export default Detective;
