// phosphor-react-native/lib/module/regular/DiceFour.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const DiceFour = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "dice-four__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M192 32H64a32 32 0 0 0-32 32v128a32 32 0 0 0 32 32h128a32 32 0 0 0 32-32V64a32 32 0 0 0-32-32Zm16 160a16 16 0 0 1-16 16H64a16 16 0 0 1-16-16V64a16 16 0 0 1 16-16h128a16 16 0 0 1 16 16Zm-96-92a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm56 0a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm-56 56a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm56 0a12 12 0 1 1-12-12 12 12 0 0 1 12 12Z"
}));
export default DiceFour;
