// phosphor-react-native/lib/module/regular/Equalizer.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Equalizer = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "equalizer__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M80 96a8 8 0 0 1-8 8H24a8 8 0 0 1 0-16h48a8 8 0 0 1 8 8Zm-8 24H24a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm0 32H24a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm0 32H24a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm80-64h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm0 32h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm0 32h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm80-96h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm-48-16h48a8 8 0 0 0 0-16h-48a8 8 0 0 0 0 16Zm48 48h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm0 32h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm0 32h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Z"
}));
export default Equalizer;
