// phosphor-react-native/lib/module/regular/EscalatorDown.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const EscalatorDown = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "escalator-down__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M170.34 85.66a8 8 0 0 1 11.32-11.32L192 84.69V48a8 8 0 0 1 16 0v36.69l10.34-10.35a8 8 0 0 1 11.32 11.32l-24 24a8 8 0 0 1-11.32 0ZM240 160v40a16 16 0 0 1-16 16h-52.5a16.06 16.06 0 0 1-11.76-5.15L68.5 112H32a16 16 0 0 1-16-16V56a16 16 0 0 1 16-16h52.5a16.06 16.06 0 0 1 11.76 5.15L187.5 144H224a16 16 0 0 1 16 16Zm-16 0h-36.5a16.06 16.06 0 0 1-11.76-5.15L84.5 56H32v40h36.5a16.06 16.06 0 0 1 11.76 5.15L171.5 200H224Z"
}));
export default EscalatorDown;
