// phosphor-react-native/lib/module/regular/FileVideo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FileVideo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "file-video__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M148.23 153.21a8 8 0 0 0-7.75-.39l-21.22 10.39A16 16 0 0 0 104 152H48a16 16 0 0 0-16 16v40a16 16 0 0 0 16 16h56a16 16 0 0 0 15.44-11.87l20.84 11A8 8 0 0 0 152 216v-56a8 8 0 0 0-3.77-6.79ZM104 208H48v-40h56v40Zm32-5.24-16-8.42v-13.68l16-7.83Zm77.66-120.42-56-56A8 8 0 0 0 152 24H56a16 16 0 0 0-16 16v88a8 8 0 0 0 16 0V40h88v48a8 8 0 0 0 8 8h48v120h-24a8 8 0 0 0 0 16h24a16 16 0 0 0 16-16V88a8 8 0 0 0-2.34-5.66ZM160 51.31 188.69 80H160Z"
}));
export default FileVideo;
