// phosphor-react-native/lib/module/regular/FilmReel.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FilmReel = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "film-reel__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 216h-40.64A103.95 103.95 0 1 0 128 232h96a8 8 0 0 0 0-16ZM40 128a88 88 0 1 1 88 88 88.1 88.1 0 0 1-88-88Zm88-24a24 24 0 1 0-24-24 24 24 0 0 0 24 24Zm0-32a8 8 0 1 1-8 8 8 8 0 0 1 8-8Zm24 104a24 24 0 1 0-24 24 24 24 0 0 0 24-24Zm-32 0a8 8 0 1 1 8 8 8 8 0 0 1-8-8Zm56-24a24 24 0 1 0-24-24 24 24 0 0 0 24 24Zm0-32a8 8 0 1 1-8 8 8 8 0 0 1 8-8Zm-96-16a24 24 0 1 0 24 24 24 24 0 0 0-24-24Zm0 32a8 8 0 1 1 8-8 8 8 0 0 1-8 8Z"
}));
export default FilmReel;
