// phosphor-react-native/lib/module/regular/FlowArrow.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FlowArrow = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "flow-arrow__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m245.66 74.34-32-32a8 8 0 0 0-11.32 11.32L220.69 72H192a74.49 74.49 0 0 0-28.35 6.73c-13.62 6.29-30.83 19.71-35.54 48-5.32 31.94-29.1 39.22-41 40.86a40 40 0 1 0 .18 16.06 71.65 71.65 0 0 0 20.84-5.65c13.62-6 30.87-19.4 35.76-48.69C150.65 88.77 190.34 88 192 88h28.69l-18.35 18.34a8 8 0 0 0 11.32 11.32l32-32a8 8 0 0 0 0-11.32ZM48 200a24 24 0 1 1 24-24 24 24 0 0 1-24 24Z"
}));
export default FlowArrow;
