// phosphor-react-native/lib/module/regular/Gradient.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Gradient = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "gradient__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M24 104a8 8 0 0 1 8-8h80a8 8 0 0 1 0 16H32a8 8 0 0 1-8-8Zm200-8h-80a8 8 0 0 0 0 16h80a8 8 0 0 0 0-16ZM72 136H32a8 8 0 0 0 0 16h40a8 8 0 0 0 0-16Zm152 0h-40a8 8 0 0 0 0 16h40a8 8 0 0 0 0-16Zm-128 8a8 8 0 0 0 8 8h48a8 8 0 0 0 0-16h-48a8 8 0 0 0-8 8Zm-40 32H32a8 8 0 0 0 0 16h24a8 8 0 0 0 0-16Zm56 0H88a8 8 0 0 0 0 16h24a8 8 0 0 0 0-16Zm56 0h-24a8 8 0 0 0 0 16h24a8 8 0 0 0 0-16Zm56 0h-24a8 8 0 0 0 0 16h24a8 8 0 0 0 0-16ZM32 72h192a8 8 0 0 0 0-16H32a8 8 0 0 0 0 16Z"
}));
export default Gradient;
