// phosphor-react-native/lib/module/regular/HandTap.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const HandTap = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "hand-tap__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M48 76a60 60 0 0 1 120 0 8 8 0 0 1-16 0 44 44 0 0 0-88 0 8 8 0 0 1-16 0Zm140 44a27.9 27.9 0 0 0-13.36 3.39A28 28 0 0 0 136 106.7V76a28 28 0 0 0-56 0v80l-3.82-6.13a28 28 0 0 0-48.41 28.17l29.32 50a8 8 0 1 0 13.8-8.04L41.6 170a12 12 0 1 1 20.78-12l.14.23 18.68 30A8 8 0 0 0 96 184V76a12 12 0 0 1 24 0v68a8 8 0 0 0 16 0v-12a12 12 0 0 1 24 0v20a8 8 0 0 0 16 0v-4a12 12 0 0 1 24 0v36c0 21.61-7.1 36.3-7.16 36.42a8 8 0 0 0 3.58 10.73 7.9 7.9 0 0 0 3.58.85 8 8 0 0 0 7.16-4.42c.37-.73 8.85-18 8.85-43.58v-36A28 28 0 0 0 188 120Z"
}));
export default HandTap;
