// phosphor-react-native/lib/module/regular/Jeep.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Jeep = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "jeep__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M240 96h-9.53l-9.29-43.35A16.08 16.08 0 0 0 205.53 40H50.47a16.08 16.08 0 0 0-15.65 12.65L25.53 96H16a8 8 0 0 0 0 16h8v96a16 16 0 0 0 16 16h24a16 16 0 0 0 16-16v-24h96v24a16 16 0 0 0 16 16h24a16 16 0 0 0 16-16v-96h8a8 8 0 0 0 0-16ZM50.47 56h155.06l8.57 40H41.9ZM64 208H40v-24h24Zm128 0v-24h24v24Zm24-40h-64v-32a8 8 0 0 0-16 0v32h-16v-32a8 8 0 0 0-16 0v32H40v-56h176ZM56 140a12 12 0 1 1 12 12 12 12 0 0 1-12-12Zm120 0a12 12 0 1 1 12 12 12 12 0 0 1-12-12Z"
}));
export default Jeep;
