// phosphor-react-native/lib/module/regular/Lockers.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Lockers = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "lockers__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M192 72a8 8 0 0 1-8 8h-24a8 8 0 0 1 0-16h24a8 8 0 0 1 8 8Zm-8 24h-24a8 8 0 0 0 0 16h24a8 8 0 0 0 0-16ZM72 80h24a8 8 0 0 0 0-16H72a8 8 0 0 0 0 16Zm24 16H72a8 8 0 0 0 0 16h24a8 8 0 0 0 0-16Zm128-48v176a8 8 0 0 1-16 0v-16h-72v16a8 8 0 0 1-16 0v-16H48v16a8 8 0 0 1-16 0V48a16 16 0 0 1 16-16h160a16 16 0 0 1 16 16ZM120 192V48H48v144Zm16 0h72V48h-72Z"
}));
export default Lockers;
