// phosphor-react-native/lib/module/regular/NavigationArrow.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const NavigationArrow = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "navigation-arrow__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M229.33 98.21 53.41 33l-.16-.05a16 16 0 0 0-20.35 20.3 1 1 0 0 0 .05.16l65.26 175.92A15.77 15.77 0 0 0 113.28 240h.3a15.77 15.77 0 0 0 15-11.29l23.56-76.56 76.56-23.56a16 16 0 0 0 .62-30.38ZM224 113.3l-76.56 23.56a16 16 0 0 0-10.58 10.58L113.3 224l-.06-.17L48 48l175.82 65.22.16.06Z"
}));
export default NavigationArrow;
