// phosphor-react-native/lib/module/regular/ShieldChevron.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ShieldChevron = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "shield-chevron__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 40H48a16 16 0 0 0-16 16v58.77c0 89.61 75.82 119.34 91 124.39a15.53 15.53 0 0 0 10 0c15.2-5.05 91-34.78 91-124.39V56a16 16 0 0 0-16-16Zm-80 184c-9.26-3.08-43.29-16.32-63.87-49.5L128 129.76l63.87 44.71C171.31 207.61 137.34 220.85 128 224Zm80-109.18c0 17.64-3.36 32.63-8.72 45.34l-66.69-46.68a8 8 0 0 0-9.18 0l-66.69 46.65c-5.36-12.71-8.72-27.7-8.72-45.34V56h160Z"
}));
export default ShieldChevron;
