// phosphor-react-native/lib/module/fill/ArticleMedium.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArticleMedium = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "article-medium-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M56 144a8 8 0 0 1-8 8H24a8 8 0 0 1 0-16h8V72h-8a8 8 0 0 1 0-16h16a8 8 0 0 1 6.78 3.74L80 112.91l33.22-53.15A8 8 0 0 1 120 56h16a8 8 0 0 1 0 16h-8v64h8a8 8 0 0 1 0 16h-24a8 8 0 0 1 0-16V91.89l-25.22 40.35a8 8 0 0 1-13.56 0L48 91.89V136a8 8 0 0 1 8 8Zm184-24h-72a8 8 0 0 0-8 8v16a8 8 0 0 0 8 8h72a8 8 0 0 0 8-8v-16a8 8 0 0 0-8-8Zm0 48H72a8 8 0 0 0-8 8v16a8 8 0 0 0 8 8h168a8 8 0 0 0 8-8v-16a8 8 0 0 0-8-8Zm0-96h-72a8 8 0 0 0-8 8v16a8 8 0 0 0 8 8h72a8 8 0 0 0 8-8V80a8 8 0 0 0-8-8Z"
}));
export default ArticleMedium;
