// phosphor-react-native/lib/module/fill/AsteriskSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const AsteriskSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "asterisk-simple-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm59.15 87.35L141 131.12l33.13 39.76a8 8 0 0 1-12.3 10.24L128 140.5l-33.85 40.62a8 8 0 0 1-12.3-10.24L115 131.12l-46.15-19.77a8 8 0 1 1 6.3-14.7L120 115.87V70a8 8 0 0 1 16 0v45.87l44.85-19.22a8 8 0 1 1 6.3 14.7Z"
}));
export default AsteriskSimple;
