// phosphor-react-native/lib/module/fill/Asterisk.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Asterisk = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "asterisk-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm64.12 133.14a8 8 0 0 1-8.24 13.72L136 142.13V194a8 8 0 0 1-16 0v-51.87l-47.88 28.73a8 8 0 0 1-8.24-13.72L112.45 128 63.88 98.86a8 8 0 0 1 8.24-13.72L120 113.87V62a8 8 0 0 1 16 0v51.87l47.88-28.73a8 8 0 1 1 8.24 13.72L143.55 128Z"
}));
export default Asterisk;
