// phosphor-react-native/lib/module/fill/Bandaids.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Circle, Path } from "react-native-svg";
const Bandaids = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "bandaids-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Circle, {
  cx: 128,
  cy: 128,
  r: 12,
  fill: props.color
}), /*#__PURE__*/React.createElement(Path, {
  d: "m184.57 128 27.71-27.72a40 40 0 1 0-56.56-56.56L128 71.43l-27.72-27.71a40 40 0 1 0-56.56 56.56L71.43 128l-27.71 27.72a40 40 0 1 0 56.56 56.56L128 184.57l27.72 27.71a40 40 0 1 0 56.56-56.56ZM128 94.06 161.94 128 128 161.94 94.06 128Zm39-39A24 24 0 1 1 201 89l-27.72 27.72-33.97-33.97ZM89 201a24 24 0 1 1-34-34l27.72-27.72 33.94 33.94Z"
}));
export default Bandaids;
