// phosphor-react-native/lib/module/fill/BezierCurve.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BezierCurve = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "bezier-curve-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M221 144.4A96.26 96.26 0 0 0 181 88h59a8 8 0 0 0 0-16h-81a32 32 0 0 0-62 0H16a8 8 0 0 0 0 16h59a96.26 96.26 0 0 0-40 56.4A32 32 0 1 0 71 184h114a32 32 0 1 0 36-39.6ZM40 192a16 16 0 1 1 16-16 16 16 0 0 1-16 16Zm88-128a16 16 0 1 1-16 16 16 16 0 0 1 16-16Zm88 128a16 16 0 1 1 16-16 16 16 0 0 1-16 16Z"
}));
export default BezierCurve;
