// phosphor-react-native/lib/module/fill/Boat.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Boat = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "boat-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M221.06 110.63 208 106.27V56a16 16 0 0 0-16-16h-56V24a8 8 0 0 0-16 0v16H64a16 16 0 0 0-16 16v50.27l-13.06 4.36A16 16 0 0 0 24 125.82v34.28a7.73 7.73 0 0 0 .31 2.19c15.72 55.09 86.12 74.06 100.07 77.3a16.19 16.19 0 0 0 7.24 0c13.95-3.24 84.35-22.21 100.07-77.3a7.73 7.73 0 0 0 .31-2.19v-34.28a16 16 0 0 0-10.94-15.19ZM136 168a8 8 0 0 1-16 0v-56a8 8 0 0 1 16 0Zm56-67.07-61.47-20.52a8 8 0 0 0-5.06 0L64 100.93V56h128Z"
}));
export default Boat;
