// phosphor-react-native/lib/module/fill/CastleTurret.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CastleTurret = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "castle-turret-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M200 24h-8a16 16 0 0 0-16 16v16h-28V40a16 16 0 0 0-16-16h-8a16 16 0 0 0-16 16v16H80V40a16 16 0 0 0-16-16h-8a16 16 0 0 0-16 16v44.69A15.86 15.86 0 0 0 44.69 96L56 107.31V216a16 16 0 0 0 16 16h112a16 16 0 0 0 16-16V107.31L211.31 96A15.86 15.86 0 0 0 216 84.69V40a16 16 0 0 0-16-16Zm-48 192h-48v-64a24 24 0 0 1 48 0Z"
}));
export default CastleTurret;
