// phosphor-react-native/lib/module/fill/ChartScatter.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChartScatter = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chart-scatter-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 40H40a16 16 0 0 0-16 16v144a16 16 0 0 0 16 16h176a16 16 0 0 0 16-16V56a16 16 0 0 0-16-16Zm-28 32a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm0 56a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm-40-16a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm-24-40a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm-24 56a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm100 64H56a8 8 0 0 1-8-8V72a8 8 0 0 1 16 0v104h136a8 8 0 0 1 0 16Z"
}));
export default ChartScatter;
