// phosphor-react-native/lib/module/fill/Checks.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Checks = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "checks-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 48H32a16 16 0 0 0-16 16v128a16 16 0 0 0 16 16h192a16 16 0 0 0 16-16V64a16 16 0 0 0-16-16ZM80 160a8 8 0 0 1-5.66-2.34l-24-24a8 8 0 0 1 11.32-11.32L80 140.69l50.34-50.35a8 8 0 0 1 11.32 11.32l-56 56A8 8 0 0 1 80 160Zm133.66-58.34-56 56a8 8 0 0 1-11.32 0l-16-16a8 8 0 0 1 11.32-11.32L152 140.69l50.34-50.35a8 8 0 0 1 11.32 11.32Z"
}));
export default Checks;
