// phosphor-react-native/lib/module/fill/CheckSquareOffset.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CheckSquareOffset = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "check-square-offset-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m125.66 165.66-56 56a8 8 0 0 1-11.32 0l-24-24a8 8 0 0 1 11.32-11.32L64 204.69l50.34-50.35a8 8 0 0 1 11.32 11.32ZM208 32H48a16 16 0 0 0-16 16v108.23a4 4 0 0 0 4.41 4 32 32 0 0 1 26.22 9.16 1.93 1.93 0 0 0 2.74 0l32-32a32 32 0 0 1 45.26 45.26l-34.55 34.54a4 4 0 0 0 2.83 6.83H208a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16Z"
}));
export default CheckSquareOffset;
