// phosphor-react-native/lib/module/fill/ContactlessPayment.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ContactlessPayment = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "contactless-payment-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24ZM97.07 155.74a8 8 0 1 1-14.14-7.48 42.79 42.79 0 0 0 0-40.52 8 8 0 0 1 14.14-7.48 59.33 59.33 0 0 1 0 55.48Zm28 16a8 8 0 1 1-14.12-7.52 77.07 77.07 0 0 0 0-72.48 8 8 0 1 1 14.12-7.52 93 93 0 0 1-.01 87.54Zm28 16a8 8 0 0 1-14.07-7.51 110.62 110.62 0 0 0 0-104.46 8 8 0 0 1 14.12-7.54 126.67 126.67 0 0 1-.05 119.54Z"
}));
export default ContactlessPayment;
