// phosphor-react-native/lib/module/fill/Cookie.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Cookie = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cookie-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 120a40 40 0 0 1-40-40 8 8 0 0 0-8-8 40 40 0 0 1-40-40 8 8 0 0 0-8-8 104 104 0 1 0 104 104 8 8 0 0 0-8-8ZM75.51 99.51a12 12 0 1 1 0 17 12 12 0 0 1 0-17Zm25 73a12 12 0 1 1 0-17 12 12 0 0 1-.02 16.98Zm23-40a12 12 0 1 1 17 0 12 12 0 0 1-17-.02Zm41 48a12 12 0 1 1 0-17 12 12 0 0 1-.02 16.98Z"
}));
export default Cookie;
