// phosphor-react-native/lib/module/fill/Couch.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Couch = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "couch-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M248 120v48a16 16 0 0 1-16 16h-8v16a8 8 0 0 1-16 0v-16H48v16a8 8 0 0 1-16 0v-16h-8a16 16 0 0 1-16-16v-48a16 16 0 0 1 16-16h8a16 16 0 0 1 16 16v16a8 8 0 0 0 8 8h144a8 8 0 0 0 8-8v-16a16 16 0 0 1 16-16h8a16 16 0 0 1 16 16Zm-56 0a32 32 0 0 1 32-32h12a4 4 0 0 0 4-4V72a16 16 0 0 0-16-16h-84a4 4 0 0 0-4 4v68h56ZM20 88h12a32 32 0 0 1 32 32v8h56V60a4 4 0 0 0-4-4H32a16 16 0 0 0-16 16v12a4 4 0 0 0 4 4Z"
}));
export default Couch;
