// phosphor-react-native/lib/module/fill/CurrencyRub.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CurrencyRub = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "currency-rub-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M176 108a28 28 0 0 1-28 28h-36V80h36a28 28 0 0 1 28 28Zm56 20A104 104 0 1 1 128 24a104.11 104.11 0 0 1 104 104Zm-40-20a44.05 44.05 0 0 0-44-44h-44a8 8 0 0 0-8 8v64H80a8 8 0 0 0 0 16h16v16H80a8 8 0 0 0 0 16h16v16a8 8 0 0 0 16 0v-16h40a8 8 0 0 0 0-16h-40v-16h36a44.05 44.05 0 0 0 44-44Z"
}));
export default CurrencyRub;
