// phosphor-react-native/lib/module/fill/CurrencyEur.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CurrencyEur = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "currency-eur-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm0 80a8 8 0 0 1 0 16H88v16h32a8 8 0 0 1 0 16H89.89a44.25 44.25 0 0 0 72.6 18.74 8 8 0 1 1 11 11.6A60.22 60.22 0 0 1 73.36 152H64a8 8 0 0 1 0-16h8v-16h-8a8 8 0 0 1 0-16h9.12a60.21 60.21 0 0 1 100.39-31.48 8 8 0 1 1-11 11.6A44.24 44.24 0 0 0 89.56 104Z"
}));
export default CurrencyEur;
