// phosphor-react-native/lib/module/fill/CurrencyGbp.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CurrencyGbp = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "currency-gbp-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm40 168H80a8 8 0 0 1 0-16h4a20 20 0 0 0 20-20v-20H80a8 8 0 0 1 0-16h24V96a40 40 0 0 1 60-34.64 8 8 0 1 1-8 13.85A24 24 0 0 0 120 96v24h24a8 8 0 0 1 0 16h-24v20a35.79 35.79 0 0 1-6.08 20H168a8 8 0 0 1 0 16Z"
}));
export default CurrencyGbp;
