// phosphor-react-native/lib/module/fill/CurrencyNgn.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CurrencyNgn = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "currency-ngn-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M88 93.63 110.61 120H88Zm80 68.74V136h-22.61ZM232 128A104 104 0 1 1 128 24a104.11 104.11 0 0 1 104 104Zm-32 0a8 8 0 0 0-8-8h-8V72a8 8 0 0 0-16 0v48h-36.32L86.07 66.79A8 8 0 0 0 72 72v48h-8a8 8 0 0 0 0 16h8v48a8 8 0 0 0 16 0v-48h36.32l45.61 53.21A8 8 0 0 0 176 192a8 8 0 0 0 8-8v-48h8a8 8 0 0 0 8-8Z"
}));
export default CurrencyNgn;
