// phosphor-react-native/lib/module/fill/DiamondsFour.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const DiamondsFour = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "diamonds-four-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M86.9 73.82a16.72 16.72 0 0 1 0-23.65l29.27-29.28a16.75 16.75 0 0 1 23.66 0l29.27 29.28a16.72 16.72 0 0 1 0 23.65l-29.27 29.28a16.75 16.75 0 0 1-23.66 0Zm148.21 42.35-29.28-29.28a16.74 16.74 0 0 0-23.65 0l-29.28 29.28a16.75 16.75 0 0 0 0 23.66l29.28 29.28a16.74 16.74 0 0 0 23.65 0l29.28-29.28a16.75 16.75 0 0 0 0-23.66Zm-132 0L73.82 86.89a16.74 16.74 0 0 0-23.65 0l-29.28 29.28a16.75 16.75 0 0 0 0 23.66l29.28 29.28a16.74 16.74 0 0 0 23.65 0l29.28-29.28a16.75 16.75 0 0 0 0-23.66Zm36.73 36.73a16.75 16.75 0 0 0-23.66 0L86.9 182.18a16.72 16.72 0 0 0 0 23.65l29.27 29.28a16.75 16.75 0 0 0 23.66 0l29.27-29.28a16.72 16.72 0 0 0 0-23.65Z"
}));
export default DiamondsFour;
