// phosphor-react-native/lib/module/fill/DiceFive.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const DiceFive = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "dice-five-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M192 32H64a32 32 0 0 0-32 32v128a32 32 0 0 0 32 32h128a32 32 0 0 0 32-32V64a32 32 0 0 0-32-32ZM92 176a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-72a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm36 36a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm36 36a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-72a12 12 0 1 1 12-12 12 12 0 0 1-12 12Z"
}));
export default DiceFive;
