// phosphor-react-native/lib/module/fill/DotsNine.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const DotsNine = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "dots-nine-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 32H48a16 16 0 0 0-16 16v160a16 16 0 0 0 16 16h160a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16ZM76 192a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-52a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-52a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm52 104a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-52a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-52a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm52 104a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-52a12 12 0 1 1 12-12 12 12 0 0 1-12 12Zm0-52a12 12 0 1 1 12-12 12 12 0 0 1-12 12Z"
}));
export default DotsNine;
