// phosphor-react-native/lib/module/fill/EscalatorDown.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const EscalatorDown = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "escalator-down-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M170.34 85.66a8 8 0 0 1 11.32-11.32L192 84.69V48a8 8 0 0 1 16 0v36.69l10.34-10.35a8 8 0 0 1 11.32 11.32l-24 24a8 8 0 0 1-11.32 0ZM224 144h-36.5L96.26 45.15A16.06 16.06 0 0 0 84.5 40H32a16 16 0 0 0-16 16v40a16 16 0 0 0 16 16h36.5l91.24 98.85A16.06 16.06 0 0 0 171.5 216H224a16 16 0 0 0 16-16v-40a16 16 0 0 0-16-16Z"
}));
export default EscalatorDown;
