// phosphor-react-native/lib/module/fill/EscalatorUp.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const EscalatorUp = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "escalator-up-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M240 56v40a16 16 0 0 1-16 16h-36.5l-91.25 98.85A16 16 0 0 1 84.5 216H32a16 16 0 0 1-16-16v-40a16 16 0 0 1 16-16h36.5l91.25-98.85A16 16 0 0 1 171.5 40H224a16 16 0 0 1 16 16Zm-34.34 90.34a8 8 0 0 0-11.32 0l-24 24a8 8 0 0 0 11.32 11.32L192 171.31V208a8 8 0 0 0 16 0v-36.69l10.34 10.35a8 8 0 0 0 11.32-11.32Z"
}));
export default EscalatorUp;
