// phosphor-react-native/lib/module/fill/Factory.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Factory = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "factory-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M240 208h-16v-72.98a.76.76 0 0 0 0-.15L209 29.74A16.08 16.08 0 0 0 193.06 16h-18.12a16.08 16.08 0 0 0-15.84 13.74l-11.56 80.91L108.8 81.6A8 8 0 0 0 96 88v32L44.8 81.6A8 8 0 0 0 32 88v120H16a8 8 0 0 0 0 16h224a8 8 0 0 0 0-16Zm-132-24H80a8 8 0 0 1 0-16h28a8 8 0 0 1 0 16Zm68 0h-28a8 8 0 0 1 0-16h28a8 8 0 0 1 0 16Zm-5.33-56-8.53-6.4 12.8-89.6h18.12l13.72 96Z"
}));
export default Factory;
