// phosphor-react-native/lib/module/fill/FloppyDiskBack.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FloppyDiskBack = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "floppy-disk-back-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 32h-20a4 4 0 0 0-4 4v44a16 16 0 0 1-16 16H88.27A8.17 8.17 0 0 1 80 88.53 8 8 0 0 1 88 80h76a4 4 0 0 0 4-4V36a4 4 0 0 0-4-4H91.31A15.86 15.86 0 0 0 80 36.69L36.69 80A15.86 15.86 0 0 0 32 91.31V208a16 16 0 0 0 16 16h160a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16Zm-80 152a32 32 0 1 1 32-32 32 32 0 0 1-32 32Z"
}));
export default FloppyDiskBack;
