// phosphor-react-native/lib/module/fill/Flower.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Flower = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "flower-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M210.35 129.36c-.81-.47-1.7-.92-2.62-1.36.92-.44 1.81-.89 2.62-1.36a40 40 0 1 0-40-69.28c-.81.47-1.65 1-2.48 1.59.08-1 .13-2 .13-3a40 40 0 0 0-80 0c0 .94 0 1.94.13 3-.83-.57-1.67-1.12-2.48-1.59a40 40 0 1 0-40 69.28c.81.47 1.7.92 2.62 1.36-.92.44-1.81.89-2.62 1.36a40 40 0 1 0 40 69.28c.81-.47 1.65-1 2.48-1.59-.08 1-.13 2-.13 2.95a40 40 0 0 0 80 0c0-.94-.05-1.94-.13-2.95.83.57 1.67 1.12 2.48 1.59a39.79 39.79 0 0 0 19.94 5.36 40.43 40.43 0 0 0 10.42-1.38 40 40 0 0 0 9.64-73.28ZM128 156a28 28 0 1 1 28-28 28 28 0 0 1-28 28Z"
}));
export default Flower;
