// phosphor-react-native/lib/module/fill/Gavel.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Gavel = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "gavel-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M52.69 99.31a16 16 0 0 1 0-22.63l64-64a16 16 0 0 1 22.63 22.63l-64 64a16 16 0 0 1-22.63 0Zm190.63 17.37a16 16 0 0 0-22.63 0l-64 64a16 16 0 0 0 0 22.62 16 16 0 0 0 22.63 0l64-64a16 16 0 0 0 0-22.62Zm-37.66-18.34-48-48a8 8 0 0 0-11.31 0l-56 56a8 8 0 0 0 0 11.31L103 130.34l-74.31 74.35a16 16 0 0 0 22.62 22.62L125.66 153l12.69 12.69a8 8 0 0 0 11.31 0l56-56a8 8 0 0 0 0-11.35Z"
}));
export default Gavel;
