// phosphor-react-native/lib/module/fill/GitBranch.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GitBranch = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "git-branch-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 64a32 32 0 1 0-40 31v9a16 16 0 0 1-16 16H96a31.71 31.71 0 0 0-16 4.31V95a32 32 0 1 0-16 0v66a32 32 0 1 0 16 0v-9a16 16 0 0 1 16-16h72a32 32 0 0 0 32-32v-9a32.06 32.06 0 0 0 24-31ZM56 64a16 16 0 1 1 16 16 16 16 0 0 1-16-16Zm32 128a16 16 0 1 1-16-16 16 16 0 0 1 16 16Z"
}));
export default GitBranch;
