// phosphor-react-native/lib/module/fill/GitDiff.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GitDiff = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "git-diff-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M120 160v48a8 8 0 0 1-8 8H64a8 8 0 0 1 0-16h28.69L64.4 171.72a55.67 55.67 0 0 1-16.4-39.6V95a32 32 0 1 1 16 0v37.13a39.71 39.71 0 0 0 11.72 28.27L104 188.69V160a8 8 0 0 1 16 0Zm88 1v-37.12a55.67 55.67 0 0 0-16.4-39.6L163.31 56H192a8 8 0 0 0 0-16h-48a8 8 0 0 0-8 8v48a8 8 0 0 0 16 0V67.31l28.28 28.29A39.71 39.71 0 0 1 192 123.88V161a32 32 0 1 0 16 0Z"
}));
export default GitDiff;
