// phosphor-react-native/lib/module/fill/GitPullRequest.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GitPullRequest = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "git-pull-request-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M104 64a32 32 0 1 0-40 31v66a32 32 0 1 0 16 0V95a32.06 32.06 0 0 0 24-31ZM88 192a16 16 0 1 1-16-16 16 16 0 0 1 16 16Zm144 0a32 32 0 1 1-40-31v-37.12a39.71 39.71 0 0 0-11.72-28.28L152 67.31V96a8 8 0 0 1-16 0V48a8 8 0 0 1 8-8h48a8 8 0 0 1 0 16h-28.69l28.29 28.28a55.67 55.67 0 0 1 16.4 39.6V161a32.06 32.06 0 0 1 24 31Z"
}));
export default GitPullRequest;
