// phosphor-react-native/lib/module/fill/Globe.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Globe = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "globe-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm78.37 64h-35.43a142.39 142.39 0 0 0-20.26-45 88.37 88.37 0 0 1 55.69 45ZM128 40.11c12 13 21 29.55 26.37 47.89h-52.74C107 69.66 116 53.13 128 40.11ZM96 128a145.29 145.29 0 0 1 2-24h60a145.72 145.72 0 0 1 0 48H98a145.29 145.29 0 0 1-2-24Zm5.63 40h52.74C149 186.34 140 202.87 128 215.89c-12-13.02-21-29.55-26.37-47.89Zm49.05 45a142.39 142.39 0 0 0 20.26-45h35.43a88.37 88.37 0 0 1-55.69 45Zm23.53-61a161.79 161.79 0 0 0 0-48h38.46a88.15 88.15 0 0 1 0 48Z"
}));
export default Globe;
