// phosphor-react-native/lib/module/fill/Keyboard.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Keyboard = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "keyboard-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M223.51 48h-191A16.51 16.51 0 0 0 16 64.49v127A16.51 16.51 0 0 0 32.49 208h191A16.51 16.51 0 0 0 240 191.51v-127A16.51 16.51 0 0 0 223.51 48ZM64 168H48a8 8 0 0 1 0-16h16a8 8 0 0 1 0 16Zm96 0H96a8 8 0 0 1 0-16h64a8 8 0 0 1 0 16Zm48 0h-16a8 8 0 0 1 0-16h16a8 8 0 0 1 0 16Zm0-32H48a8 8 0 0 1 0-16h160a8 8 0 0 1 0 16Zm0-32H48a8 8 0 0 1 0-16h160a8 8 0 0 1 0 16Z"
}));
export default Keyboard;
