// phosphor-react-native/lib/module/fill/ListDashes.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ListDashes = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "list-dashes-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 56v16a8 8 0 0 1-8 8H96a8 8 0 0 1-8-8V56a8 8 0 0 1 8-8h120a8 8 0 0 1 8 8ZM56 48H40a8 8 0 0 0-8 8v16a8 8 0 0 0 8 8h16a8 8 0 0 0 8-8V56a8 8 0 0 0-8-8Zm160 64H96a8 8 0 0 0-8 8v16a8 8 0 0 0 8 8h120a8 8 0 0 0 8-8v-16a8 8 0 0 0-8-8Zm-160 0H40a8 8 0 0 0-8 8v16a8 8 0 0 0 8 8h16a8 8 0 0 0 8-8v-16a8 8 0 0 0-8-8Zm160 64H96a8 8 0 0 0-8 8v16a8 8 0 0 0 8 8h120a8 8 0 0 0 8-8v-16a8 8 0 0 0-8-8Zm-160 0H40a8 8 0 0 0-8 8v16a8 8 0 0 0 8 8h16a8 8 0 0 0 8-8v-16a8 8 0 0 0-8-8Z"
}));
export default ListDashes;
