// phosphor-react-native/lib/module/fill/Lockers.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Lockers = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "lockers-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 32H48a16 16 0 0 0-16 16v176a8 8 0 0 0 16 0v-16h72v16a8 8 0 0 0 16 0v-16h72v16a8 8 0 0 0 16 0V48a16 16 0 0 0-16-16ZM96 112H56a8 8 0 0 1 0-16h40a8 8 0 0 1 0 16Zm0-32H56a8 8 0 0 1 0-16h40a8 8 0 0 1 0 16Zm40 104a8 8 0 0 1-16 0V56a8 8 0 0 1 16 0Zm64-72h-40a8 8 0 0 1 0-16h40a8 8 0 0 1 0 16Zm0-32h-40a8 8 0 0 1 0-16h40a8 8 0 0 1 0 16Z"
}));
export default Lockers;
