// phosphor-react-native/lib/module/fill/MegaphoneSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const MegaphoneSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "megaphone-simple-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m220.54 86.66-176.06-54A16 16 0 0 0 24 48v144a16 16 0 0 0 16 16 16 16 0 0 0 4.52-.65L128 181.73V192a16 16 0 0 0 16 16h32a16 16 0 0 0 16-16v-29.9l28.54-8.75A16.09 16.09 0 0 0 232 138v-36a16.09 16.09 0 0 0-11.46-15.34ZM176 192h-32v-15.18l32-9.82Zm40-54h-.11L144 160.08V79.91L215.89 102h.11v36Z"
}));
export default MegaphoneSimple;
