// phosphor-react-native/lib/module/fill/MonitorPlay.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const MonitorPlay = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "monitor-play-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M168 224a8 8 0 0 1-8 8H96a8 8 0 0 1 0-16h64a8 8 0 0 1 8 8Zm64-160v112a24 24 0 0 1-24 24H48a24 24 0 0 1-24-24V64a24 24 0 0 1 24-24h160a24 24 0 0 1 24 24Zm-68 56a8 8 0 0 0-3.71-6.75l-44-28A8 8 0 0 0 104 92v56a8 8 0 0 0 12.29 6.75l44-28A8 8 0 0 0 164 120Z"
}));
export default MonitorPlay;
