// phosphor-react-native/lib/module/fill/NewspaperClipping.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const NewspaperClipping = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "newspaper-clipping-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M72 104h32v48H72Zm160-48v160a8 8 0 0 1-11.58 7.15L192 208.94l-28.42 14.21a8 8 0 0 1-7.16 0L128 208.94l-28.42 14.21a8 8 0 0 1-7.16 0L64 208.94l-28.42 14.21A8 8 0 0 1 24 216V56a16 16 0 0 1 16-16h176a16 16 0 0 1 16 16ZM120 96a8 8 0 0 0-8-8H64a8 8 0 0 0-8 8v64a8 8 0 0 0 8 8h48a8 8 0 0 0 8-8Zm80 48a8 8 0 0 0-8-8h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 8-8Zm0-32a8 8 0 0 0-8-8h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 8-8Z"
}));
export default NewspaperClipping;
