// phosphor-react-native/lib/module/fill/NumberEight.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const NumberEight = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "number-eight-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M108 92a20 20 0 1 1 20 20 20 20 0 0 1-20-20Zm20 36a28 28 0 1 0 28 28 28 28 0 0 0-28-28Zm88-88v176a16 16 0 0 1-16 16H56a16 16 0 0 1-16-16V40a16 16 0 0 1 16-16h144a16 16 0 0 1 16 16Zm-44 116a44 44 0 0 0-20.23-37 36 36 0 1 0-47.54 0A44 44 0 1 0 172 156Z"
}));
export default NumberEight;
