// phosphor-react-native/lib/module/fill/PawPrint.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PawPrint = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "paw-print-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M240 108a28 28 0 1 1-28-28 28 28 0 0 1 28 28Zm-168 0a28 28 0 1 0-28 28 28 28 0 0 0 28-28Zm20-20a28 28 0 1 0-28-28 28 28 0 0 0 28 28Zm72 0a28 28 0 1 0-28-28 28 28 0 0 0 28 28Zm23.12 60.86a35.3 35.3 0 0 1-16.87-21.14 44 44 0 0 0-84.5 0A35.25 35.25 0 0 1 69 148.82 40 40 0 0 0 88 224a39.48 39.48 0 0 0 15.52-3.13 64.09 64.09 0 0 1 48.87 0 40 40 0 0 0 34.73-72Z"
}));
export default PawPrint;
