// phosphor-react-native/lib/module/fill/Percent.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Percent = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "percent-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M176 172a12 12 0 1 1-12-12 12 12 0 0 1 12 12ZM92 96a12 12 0 1 0-12-12 12 12 0 0 0 12 12Zm132-48v160a16 16 0 0 1-16 16H48a16 16 0 0 1-16-16V48a16 16 0 0 1 16-16h160a16 16 0 0 1 16 16ZM64 84a28 28 0 1 0 28-28 28 28 0 0 0-28 28Zm128 88a28 28 0 1 0-28 28 28 28 0 0 0 28-28Zm-2.34-105.66a8 8 0 0 0-11.32 0l-112 112a8 8 0 0 0 11.32 11.32l112-112a8 8 0 0 0 0-11.32Z"
}));
export default Percent;
