// phosphor-react-native/lib/module/fill/PinterestLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PinterestLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "pinterest-logo-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M232 128.7c-.38 56.49-46.46 102.73-102.94 103.29a104.16 104.16 0 0 1-25.94-3 4 4 0 0 1-2.91-4.86l8.64-34.55A60.57 60.57 0 0 0 136 196c37 0 66.7-33.45 63.81-73.36A72 72 0 1 0 61.24 155 8 8 0 0 0 72 159.29a8.19 8.19 0 0 0 4-10.49 56 56 0 1 1 107.86-24.93C186 154.4 163.73 180 136 180a44.87 44.87 0 0 1-23.14-6.44l14.9-59.62a8 8 0 0 0-15.52-3.88L85.38 217.51a4 4 0 0 1-5.71 2.59A104 104 0 0 1 24 126.88C24.6 70.52 70.67 24.52 127 24a104 104 0 0 1 105 104.7Z"
}));
export default PinterestLogo;
