// phosphor-react-native/lib/module/fill/Plugs.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Plugs = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "plugs-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M149.66 149.66 131.31 168l18.35 18.34a8 8 0 0 1-11.32 11.32l-6.34-6.35-23.31 23.32a32.06 32.06 0 0 1-45.26 0l-5.37-5.38-28.4 28.41a8 8 0 0 1-11.32-11.32l28.41-28.4-5.38-5.37a32 32 0 0 1 0-45.26L64.69 124l-6.35-6.34a8 8 0 0 1 11.32-11.32L88 124.69l18.34-18.35a8 8 0 0 1 11.32 11.32L99.31 136 120 156.69l18.34-18.35a8 8 0 0 1 11.32 11.32Zm88-131.32a8 8 0 0 0-11.32 0l-28.4 28.41-5.37-5.38a32.05 32.05 0 0 0-45.26 0L124 64.69l-6.34-6.35a8 8 0 0 0-11.32 11.32l80 80a8 8 0 0 0 11.32-11.32l-6.35-6.34 23.32-23.31a32 32 0 0 0 0-45.26l-5.38-5.37 28.41-28.4a8 8 0 0 0 0-11.32Z"
}));
export default Plugs;
