// phosphor-react-native/lib/module/fill/Plug.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Plug = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "plug-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M237.66 77.66 203.31 112l26.35 26.34a8 8 0 0 1-11.32 11.32l-6.34-6.35-53 53a40 40 0 0 1-56.57 0l-15.68-15.74-49.09 49.09a8 8 0 0 1-11.32-11.32l49.09-49.09-15.71-15.71a40 40 0 0 1 0-56.57l53-53-6.35-6.34a8 8 0 0 1 11.32-11.32L144 52.69l34.34-34.35a8 8 0 1 1 11.32 11.32L155.31 64 192 100.69l34.34-34.35a8 8 0 0 1 11.32 11.32Z"
}));
export default Plug;
