// phosphor-react-native/lib/module/fill/QrCode.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Rect, Path } from "react-native-svg";
const QrCode = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "qr-code-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Rect, {
  width: 80,
  height: 80,
  x: 40,
  y: 40,
  rx: 16
}), /*#__PURE__*/React.createElement(Rect, {
  width: 80,
  height: 80,
  x: 40,
  y: 136,
  rx: 16
}), /*#__PURE__*/React.createElement(Rect, {
  width: 80,
  height: 80,
  x: 136,
  y: 40,
  rx: 16
}), /*#__PURE__*/React.createElement(Path, {
  d: "M144 184a8 8 0 0 0 8-8v-32a8 8 0 0 0-16 0v32a8 8 0 0 0 8 8Z"
}), /*#__PURE__*/React.createElement(Path, {
  d: "M208 152h-24v-8a8 8 0 0 0-16 0v56h-24a8 8 0 0 0 0 16h32a8 8 0 0 0 8-8v-40h24a8 8 0 0 0 0-16ZM208 184a8 8 0 0 0-8 8v16a8 8 0 0 0 16 0v-16a8 8 0 0 0-8-8Z"
}));
export default QrCode;
