// phosphor-react-native/lib/module/fill/Scissors.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Scissors = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "scissors-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M157.73 113.13a8 8 0 0 1 2.09-11.13l67.66-46.3a8 8 0 0 1 9 13.21l-67.67 46.3a8 8 0 0 1-11.12-2.09Zm80.87 85.07a8 8 0 0 1-11.12 2.09L136 137.69l-42.52 29.09a36 36 0 1 1-9-13.2L121.83 128l-37.38-25.58a36 36 0 1 1 9-13.2l143 97.86a8 8 0 0 1 2.15 11.12ZM72 180a12 12 0 1 0-12 12 12 12 0 0 0 12-12Zm0-104a12 12 0 1 0-12 12 12 12 0 0 0 12-12Z"
}));
export default Scissors;
