// phosphor-react-native/lib/module/fill/Screencast.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Screencast = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "screencast-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M56 208a8 8 0 0 1-16 0 8 8 0 0 0-8-8 8 8 0 0 1 0-16 24 24 0 0 1 24 24Zm-24-56a8 8 0 0 0 0 16 40 40 0 0 1 40 40 8 8 0 0 0 16 0 56.06 56.06 0 0 0-56-56Zm0-32a8 8 0 0 0 0 16 72.08 72.08 0 0 1 72 72 8 8 0 0 0 16 0 88.1 88.1 0 0 0-88-88Zm184-80H40a16 16 0 0 0-16 16v40a8 8 0 0 0 8 8 104.11 104.11 0 0 1 104 104 8 8 0 0 0 8 8h72a16 16 0 0 0 16-16V56a16 16 0 0 0-16-16Z"
}));
export default Screencast;
