// phosphor-react-native/lib/module/fill/ShieldCheckered.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ShieldCheckered = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "shield-checkered-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 40H48a16 16 0 0 0-16 16v58.77c0 89.62 75.82 119.34 91 124.38a15.44 15.44 0 0 0 10 0c15.2-5.05 91-34.77 91-124.39V56a16 16 0 0 0-16-16ZM48.66 128H128V56h80v58.77a130.51 130.51 0 0 1-.66 13.21H128v96c-12.74-4.25-72.41-27.72-79.34-95.98Z"
}));
export default ShieldCheckered;
