// phosphor-react-native/lib/module/fill/SortAscending.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SortAscending = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "sort-ascending-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 128a8 8 0 0 1-8 8H48a8 8 0 0 1 0-16h72a8 8 0 0 1 8 8ZM48 72h136a8 8 0 0 0 0-16H48a8 8 0 0 0 0 16Zm56 112H48a8 8 0 0 0 0 16h56a8 8 0 0 0 0-16Zm127.39-19.06A8 8 0 0 0 224 160h-32v-48a8 8 0 0 0-16 0v48h-32a8 8 0 0 0-5.66 13.66l40 40a8 8 0 0 0 11.32 0l40-40a8 8 0 0 0 1.73-8.72Z"
}));
export default SortAscending;
