// phosphor-react-native/lib/module/fill/SpeakerSimpleX.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SpeakerSimpleX = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "speaker-simple-x-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M155.52 24.81a8 8 0 0 0-8.43.88L77.25 80H32a16 16 0 0 0-16 16v64a16 16 0 0 0 16 16h45.25l69.84 54.31A7.94 7.94 0 0 0 152 232a8 8 0 0 0 8-8V32a8 8 0 0 0-4.48-7.19ZM227.31 128l18.35-18.34a8 8 0 0 0-11.32-11.32L216 116.69l-18.34-18.35a8 8 0 0 0-11.32 11.32L204.69 128l-18.35 18.34a8 8 0 0 0 11.32 11.32L216 139.31l18.34 18.35a8 8 0 0 0 11.32-11.32Z"
}));
export default SpeakerSimpleX;
