// phosphor-react-native/lib/module/fill/SpeakerSlash.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SpeakerSlash = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "speaker-slash-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M53.92 34.62a8 8 0 1 0-11.84 10.76L73.55 80H32a16 16 0 0 0-16 16v64a16 16 0 0 0 16 16h45.25l69.84 54.31A8 8 0 0 0 160 224v-48.91l42.08 46.29a8 8 0 1 0 11.84-10.76ZM32 96h40v64H32Zm154 47.87a24 24 0 0 0 0-31.72 8 8 0 1 1 12-10.57 40 40 0 0 1 0 52.88 8 8 0 0 1-12-10.59Zm-79.77-75.49a8 8 0 0 1 1-11.7l39.85-31A8 8 0 0 1 160 32v74.83a8 8 0 0 1-13.92 5.38ZM248 128a79.9 79.9 0 0 1-20.37 53.34 8 8 0 0 1-11.92-10.67 64 64 0 0 0 0-85.33 8 8 0 1 1 11.92-10.67A79.83 79.83 0 0 1 248 128Z"
}));
export default SpeakerSlash;
