// phosphor-react-native/lib/module/fill/SpotifyLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SpotifyLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "spotify-logo-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm31.07 145.74a8 8 0 0 1-10.81 3.33 42.79 42.79 0 0 0-40.52 0 8 8 0 0 1-7.48-14.14 59.33 59.33 0 0 1 55.48 0 8 8 0 0 1 3.33 10.81Zm16-28a8 8 0 0 1-10.82 3.3 77.07 77.07 0 0 0-72.48 0 8 8 0 0 1-7.52-14.12 93 93 0 0 1 87.52 0 8 8 0 0 1 3.29 10.84Zm16-28a8 8 0 0 1-10.83 3.29 110.62 110.62 0 0 0-104.46 0 8 8 0 0 1-7.54-14.12 126.67 126.67 0 0 1 119.54 0 8 8 0 0 1 3.28 10.85Z"
}));
export default SpotifyLogo;
