// phosphor-react-native/lib/module/fill/Stethoscope.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Stethoscope = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "stethoscope-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M232 152a32 32 0 1 0-39.93 31 8 8 0 0 0-.07 1 32 32 0 0 1-32 32h-24a32 32 0 0 1-32-32v-40.52c31.47-4 56-31.47 56-64.31V40a16 16 0 0 0-16-16h-16a8 8 0 0 0 0 16h16v39.17c0 26.58-21.25 48.49-47.36 48.83A48 48 0 0 1 48 80V40h16a8 8 0 0 0 0-16H48a16 16 0 0 0-16 16v40a64 64 0 0 0 56 63.49V184a48.05 48.05 0 0 0 48 48h24a48.05 48.05 0 0 0 48-48 8 8 0 0 0-.07-1A32 32 0 0 0 232 152Zm-32 8a8 8 0 1 1 8-8 8 8 0 0 1-8 8Z"
}));
export default Stethoscope;
