// phosphor-react-native/lib/module/fill/SunHorizon.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SunHorizon = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "sun-horizon-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M248 160a8 8 0 0 1-8 8H16a8 8 0 0 1 0-16h40.45a73.54 73.54 0 0 1-.45-8 72 72 0 0 1 144 0 73.54 73.54 0 0 1-.45 8H240a8 8 0 0 1 8 8Zm-40 32H48a8 8 0 0 0 0 16h160a8 8 0 0 0 0-16ZM80.84 59.58a8 8 0 0 0 14.32-7.16l-8-16a8 8 0 0 0-14.32 7.16Zm-60.42 43.58 16 8a8 8 0 1 0 7.16-14.31l-16-8a8 8 0 1 0-7.16 14.31ZM216 112a8 8 0 0 0 3.57-.84l16-8a8 8 0 1 0-7.16-14.31l-16 8A8 8 0 0 0 216 112Zm-51.58-48.84a8 8 0 0 0 10.74-3.58l8-16a8 8 0 0 0-14.32-7.16l-8 16a8 8 0 0 0 3.58 10.74Z"
}));
export default SunHorizon;
