// phosphor-react-native/lib/module/fill/Swatches.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Swatches = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "swatches-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M232 155.91a16 16 0 0 0-1-5.22l-19.06-52.21A16 16 0 0 0 191.49 89l-67.81 24.57 12.08-69A16 16 0 0 0 122.84 26l-54.67-9.75a15.94 15.94 0 0 0-18.47 13l-25 143.12A43.82 43.82 0 0 0 67.78 224H216a16 16 0 0 0 16-16ZM68 196a16 16 0 1 1 16-16 16 16 0 0 1-16 16Zm42.72-8.38 9.78-55.92 76.42-27.7L216 156.11 108.78 195a44.89 44.89 0 0 0 1.94-7.38ZM216 208h-96.26L216 173.11Z"
}));
export default Swatches;
