// phosphor-react-native/lib/module/fill/Tabs.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Tabs = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "tabs-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M256 168a8 8 0 0 1-8 8H8a8 8 0 0 1-7.66-10.3v-.05L22.63 91.4A15.89 15.89 0 0 1 38 80h84.1a15.89 15.89 0 0 1 15.32 11.4L158 160h31.3l-19.2-64H160a8 8 0 0 1 0-16h10.05a15.89 15.89 0 0 1 15.32 11.4L206 160h31.3l-19.2-64H208a8 8 0 0 1 0-16h10.05a15.89 15.89 0 0 1 15.32 11.4l22.26 74.19a.24.24 0 0 1 0 .08A8 8 0 0 1 256 168Z"
}));
export default Tabs;
