// phosphor-react-native/lib/module/fill/TextHFour.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TextHFour = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "text-h-four-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 32H48a16 16 0 0 0-16 16v160a16 16 0 0 0 16 16h160a16 16 0 0 0 16-16V48a16 16 0 0 0-16-16Zm-88 128a8 8 0 0 1-16 0v-32H72v32a8 8 0 0 1-16 0V80a8 8 0 0 1 16 0v32h32V80a8 8 0 0 1 16 0Zm80 0h-8v16a8 8 0 0 1-16 0v-16h-32a8 8 0 0 1-7.49-10.81l24-64a8 8 0 0 1 15 5.62l-20 53.19H176v-24a8 8 0 0 1 16 0v24h8a8 8 0 0 1 0 16Z"
}));
export default TextHFour;
