// phosphor-react-native/lib/module/fill/TextColumns.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TextColumns = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "text-columns-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M120 184v8a8 8 0 0 1-8 8H40a8 8 0 0 1-8-8v-8a8 8 0 0 1 8-8h72a8 8 0 0 1 8 8Zm-8-88H40a8 8 0 0 0-8 8v8a8 8 0 0 0 8 8h72a8 8 0 0 0 8-8v-8a8 8 0 0 0-8-8Zm0 40H40a8 8 0 0 0-8 8v8a8 8 0 0 0 8 8h72a8 8 0 0 0 8-8v-8a8 8 0 0 0-8-8Zm0-80H40a8 8 0 0 0-8 8v8a8 8 0 0 0 8 8h72a8 8 0 0 0 8-8v-8a8 8 0 0 0-8-8Zm104 40h-72a8 8 0 0 0-8 8v8a8 8 0 0 0 8 8h72a8 8 0 0 0 8-8v-8a8 8 0 0 0-8-8Zm0 80h-72a8 8 0 0 0-8 8v8a8 8 0 0 0 8 8h72a8 8 0 0 0 8-8v-8a8 8 0 0 0-8-8Zm0-120h-72a8 8 0 0 0-8 8v8a8 8 0 0 0 8 8h72a8 8 0 0 0 8-8v-8a8 8 0 0 0-8-8Zm0 80h-72a8 8 0 0 0-8 8v8a8 8 0 0 0 8 8h72a8 8 0 0 0 8-8v-8a8 8 0 0 0-8-8Z"
}));
export default TextColumns;
