// phosphor-react-native/lib/module/fill/Truck.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Truck = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "truck-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m247.43 117-14-35a15.93 15.93 0 0 0-14.85-10H184v-8a8 8 0 0 0-8-8H24A16 16 0 0 0 8 72v112a16 16 0 0 0 16 16h17a32 32 0 0 0 62 0h50a32 32 0 0 0 62 0h17a16 16 0 0 0 16-16v-64a8.13 8.13 0 0 0-.57-3ZM72 208a16 16 0 1 1 16-16 16 16 0 0 1-16 16Zm-48-72V72h144v64Zm160 72a16 16 0 1 1 16-16 16 16 0 0 1-16 16Zm0-96V88h34.58l9.6 24Z"
}));
export default Truck;
