// phosphor-react-native/lib/module/fill/Unite.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Unite = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "unite-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M64 160c0 3.2.16 6.37.47 9.49a80 80 0 1 1 105-105c-3.12-.31-6.29-.47-9.49-.47A96.11 96.11 0 0 0 64 160Zm112-64a79.22 79.22 0 0 0-1.37-14.63A79.22 79.22 0 0 0 160 80a80.09 80.09 0 0 0-80 80 79.22 79.22 0 0 0 1.37 14.63A79.22 79.22 0 0 0 96 176a80.09 80.09 0 0 0 80-80Zm15.53-9.49c.31 3.12.47 6.29.47 9.49a96.11 96.11 0 0 1-96 96c-3.2 0-6.37-.16-9.49-.47a80 80 0 1 0 105-105Z"
}));
export default Unite;
