// phosphor-react-native/lib/module/fill/VideoCameraSlash.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const VideoCameraSlash = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "video-camera-slash-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M213.92 210.62a8 8 0 1 1-11.84 10.76L182.64 200H32a16 16 0 0 1-16-16V72a16 16 0 0 1 16-16h19.73l-9.65-10.62a8 8 0 1 1 11.84-10.76ZM251.77 73a8 8 0 0 0-8.21.39l-32 21.34a8 8 0 0 0-3.56 6.65v53.34a8 8 0 0 0 3.56 6.65l32 21.34A8 8 0 0 0 248 184a8 8 0 0 0 8-8V80a8 8 0 0 0-4.23-7Zm-73.69 74.46A8 8 0 0 0 192 142V72a16 16 0 0 0-16-16h-62.94a8 8 0 0 0-5.92 13.38Z"
}));
export default VideoCameraSlash;
