// phosphor-react-native/lib/module/fill/VirtualReality.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const VirtualReality = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "virtual-reality-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M176 116a12 12 0 0 1-12 12h-12v-24h12a12 12 0 0 1 12 12Zm80 12a80.09 80.09 0 0 1-80 80H80a80 80 0 0 1 0-160h96a80.09 80.09 0 0 1 80 80ZM119 88.59A8 8 0 0 0 108.59 93L90 138.74 71.41 93a8 8 0 1 0-14.82 6l26 64a8 8 0 0 0 14.82 0l26-64A8 8 0 0 0 119 88.59Zm61.83 49.76A28 28 0 0 0 164 88h-20a8 8 0 0 0-8 8v64a8 8 0 0 0 16 0v-16h12c.54 0 1.08 0 1.61-.05l11.44 20A8 8 0 0 0 191 156Z"
}));
export default VirtualReality;
