// phosphor-react-native/lib/module/fill/Wall.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Wall = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "wall-fill__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M136 92h92a4 4 0 0 0 4-4V56a8 8 0 0 0-8-8h-84a4 4 0 0 0-4 4ZM184 152h44a4 4 0 0 0 4-4v-36a4 4 0 0 0-4-4h-44ZM88 108h80v44H88zM72 108H28a4 4 0 0 0-4 4v36a4 4 0 0 0 4 4h44ZM136 168v36a4 4 0 0 0 4 4h84a8 8 0 0 0 8-8v-28a4 4 0 0 0-4-4ZM120 92V52a4 4 0 0 0-4-4H32a8 8 0 0 0-8 8v32a4 4 0 0 0 4 4ZM120 168H28a4 4 0 0 0-4 4v28a8 8 0 0 0 8 8h84a4 4 0 0 0 4-4Z"
}));
export default Wall;
