// phosphor-react-native/lib/module/light/Anchor.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Anchor = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "anchor-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 138a6 6 0 0 0-6 6 42 42 0 0 1-42 42 45.86 45.86 0 0 0-34 15.08V126h34a6 6 0 0 0 0-12h-34V85.4a30 30 0 1 0-12 0V114H88a6 6 0 0 0 0 12h34v75.08A45.86 45.86 0 0 0 88 186a42 42 0 0 1-42-42 6 6 0 0 0-12 0 54.06 54.06 0 0 0 54 54 34 34 0 0 1 34 34 6 6 0 0 0 12 0 34 34 0 0 1 34-34 54.06 54.06 0 0 0 54-54 6 6 0 0 0-6-6ZM110 56a18 18 0 1 1 18 18 18 18 0 0 1-18-18Z"
}));
export default Anchor;
