// phosphor-react-native/lib/module/light/AngularLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const AngularLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "angular-logo-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m226.31 66.46-96-40a6.06 6.06 0 0 0-4.62 0l-96 40a6 6 0 0 0-3.64 6.33l16 120a6 6 0 0 0 3.27 4.58l80 40a6 6 0 0 0 5.36 0l80-40a6 6 0 0 0 3.27-4.58l16-120a6 6 0 0 0-3.64-6.33Zm-23.84 121.6L128 225.29l-74.47-37.23-15-112.29L128 38.5l89.44 37.27Zm-79.72-103-40 72a6 6 0 0 0 10.5 5.82L104.86 142h46.28l11.61 20.91A6 6 0 0 0 168 166a5.88 5.88 0 0 0 2.9-.76 6 6 0 0 0 2.34-8.15l-40-72a6 6 0 0 0-10.5 0ZM144.47 130h-32.94L128 100.35Z"
}));
export default AngularLogo;
