// phosphor-react-native/lib/module/light/ArrowFatLinesUp.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowFatLinesUp = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-fat-lines-up-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m228.24 115.76-96-96a6 6 0 0 0-8.48 0l-96 96A6 6 0 0 0 32 126h42v26a6 6 0 0 0 6 6h96a6 6 0 0 0 6-6v-26h42a6 6 0 0 0 4.24-10.24ZM176 114a6 6 0 0 0-6 6v26H86v-26a6 6 0 0 0-6-6H46.49L128 32.49 209.51 114Zm6 102a6 6 0 0 1-6 6H80a6 6 0 0 1 0-12h96a6 6 0 0 1 6 6Zm0-32a6 6 0 0 1-6 6H80a6 6 0 0 1 0-12h96a6 6 0 0 1 6 6Z"
}));
export default ArrowFatLinesUp;
