// phosphor-react-native/lib/module/light/ArrowsOutCardinal.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowsOutCardinal = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrows-out-cardinal-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M91.76 60.24a6 6 0 0 1 0-8.48l32-32a6 6 0 0 1 8.48 0l32 32a6 6 0 1 1-8.48 8.48L134 38.49V96a6 6 0 0 1-12 0V38.49l-21.76 21.75a6 6 0 0 1-8.48 0Zm64 135.52L134 217.51V160a6 6 0 0 0-12 0v57.51l-21.76-21.75a6 6 0 0 0-8.48 8.48l32 32a6 6 0 0 0 8.48 0l32-32a6 6 0 0 0-8.48-8.48Zm80.48-72-32-32a6 6 0 0 0-8.48 8.48L217.51 122H160a6 6 0 0 0 0 12h57.51l-21.75 21.76a6 6 0 1 0 8.48 8.48l32-32a6 6 0 0 0 0-8.48ZM38.49 134H96a6 6 0 0 0 0-12H38.49l21.75-21.76a6 6 0 0 0-8.48-8.48l-32 32a6 6 0 0 0 0 8.48l32 32a6 6 0 0 0 8.48-8.48Z"
}));
export default ArrowsOutCardinal;
