// phosphor-react-native/lib/module/light/BluetoothConnected.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BluetoothConnected = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "bluetooth-connected-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M187.6 171.2 130 128l57.6-43.2a6 6 0 0 0 0-9.6l-64-48A6 6 0 0 0 114 32v84L59.6 75.2a6 6 0 0 0-7.2 9.6L110 128l-57.6 43.2a6 6 0 0 0 7.2 9.6L114 140v84a6 6 0 0 0 9.6 4.8l64-48a6 6 0 0 0 0-9.6ZM126 44l48 36-48 36Zm0 168v-72l48 36Zm-74-74a10 10 0 1 1 10-10 10 10 0 0 1-10 10Zm154-10a10 10 0 1 1-10-10 10 10 0 0 1 10 10Z"
}));
export default BluetoothConnected;
