// phosphor-react-native/lib/module/light/BugDroid.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BugDroid = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "bug-droid-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m189 43.47 15.22-15.23a6 6 0 0 0-8.48-8.48L180 35.54a85.76 85.76 0 0 0-104 0L60.24 19.76a6 6 0 0 0-8.48 8.48L67 43.47A85.7 85.7 0 0 0 42 104v40a86 86 0 0 0 172 0v-40a85.7 85.7 0 0 0-25-60.53ZM128 30a74.09 74.09 0 0 1 74 74v10H54v-10a74.09 74.09 0 0 1 74-74Zm0 188a74.09 74.09 0 0 1-74-74v-18h148v18a74.09 74.09 0 0 1-74 74Zm18-134a10 10 0 1 1 10 10 10 10 0 0 1-10-10Zm-56 0a10 10 0 1 1 10 10 10 10 0 0 1-10-10Z"
}));
export default BugDroid;
