// phosphor-react-native/lib/module/light/Cardholder.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Cardholder = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cardholder-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 50H48a22 22 0 0 0-22 22v112a22 22 0 0 0 22 22h160a22 22 0 0 0 22-22V72a22 22 0 0 0-22-22ZM38 94h180v20h-50.81a14 14 0 0 0-13.71 11.21 26 26 0 0 1-51 0A14 14 0 0 0 88.81 114H38Zm10-32h160a10 10 0 0 1 10 10v10H38V72a10 10 0 0 1 10-10Zm160 132H48a10 10 0 0 1-10-10v-58h50.81a2 2 0 0 1 2 1.59 38 38 0 0 0 74.48 0 2 2 0 0 1 1.95-1.59H218v58a10 10 0 0 1-10 10Z"
}));
export default Cardholder;
