// phosphor-react-native/lib/module/light/ChalkboardSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChalkboardSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chalkboard-simple-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M26 168V56a14 14 0 0 1 14-14h176a14 14 0 0 1 14 14v112a6 6 0 0 1-12 0V56a2 2 0 0 0-2-2H40a2 2 0 0 0-2 2v112a6 6 0 0 1-12 0Zm220 32a6 6 0 0 1-6 6H16a6 6 0 0 1 0-12h98v-26a6 6 0 0 1 6-6h64a6 6 0 0 1 6 6v26h50a6 6 0 0 1 6 6Zm-120-6h52v-20h-52Z"
}));
export default ChalkboardSimple;
