// phosphor-react-native/lib/module/light/ChartScatter.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChartScatter = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chart-scatter-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M230 208a6 6 0 0 1-6 6H32a6 6 0 0 1-6-6V48a6 6 0 0 1 12 0v154h186a6 6 0 0 1 6 6Zm-98-50a10 10 0 1 0-10-10 10 10 0 0 0 10 10Zm-24-56a10 10 0 1 0-10-10 10 10 0 0 0 10 10Zm-32 72a10 10 0 1 0-10-10 10 10 0 0 0 10 10Zm96-48a10 10 0 1 0-10-10 10 10 0 0 0 10 10Zm24-40a10 10 0 1 0-10-10 10 10 0 0 0 10 10Zm-8 68a10 10 0 1 0 10 10 10 10 0 0 0-10-10Z"
}));
export default ChartScatter;
