// phosphor-react-native/lib/module/light/ChatsTeardrop.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChatsTeardrop = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chats-teardrop-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M168.16 74.42A78 78 0 0 0 18 104v66a12 12 0 0 0 12 12h58a78.15 78.15 0 0 0 72 48h66a12 12 0 0 0 12-12v-66a78 78 0 0 0-69.84-77.58ZM30 104a66 66 0 1 1 66 66H30Zm196 114h-66a66.13 66.13 0 0 1-58.89-36.19 77.92 77.92 0 0 0 71-94.68A66 66 0 0 1 226 152Z"
}));
export default ChatsTeardrop;
