// phosphor-react-native/lib/module/light/ChatTeardropText.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChatTeardropText = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chat-teardrop-text-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M166 112a6 6 0 0 1-6 6H96a6 6 0 0 1 0-12h64a6 6 0 0 1 6 6Zm-6 26H96a6 6 0 0 0 0 12h64a6 6 0 0 0 0-12Zm70-14a98.11 98.11 0 0 1-98 98H47.67A13.68 13.68 0 0 1 34 208.33V124a98 98 0 0 1 196 0Zm-12 0a86 86 0 0 0-172 0v84.33a1.67 1.67 0 0 0 1.67 1.67H132a86.1 86.1 0 0 0 86-86Z"
}));
export default ChatTeardropText;
