// phosphor-react-native/lib/module/light/ChatTeardropDots.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChatTeardropDots = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chat-teardrop-dots-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M132 26a98.11 98.11 0 0 0-98 98v84.33A13.68 13.68 0 0 0 47.67 222H132a98 98 0 0 0 0-196Zm0 184H47.67a1.67 1.67 0 0 1-1.67-1.67V124a86 86 0 1 1 86 86Zm10-82a10 10 0 1 1-10-10 10 10 0 0 1 10 10Zm-44 0a10 10 0 1 1-10-10 10 10 0 0 1 10 10Zm88 0a10 10 0 1 1-10-10 10 10 0 0 1 10 10Z"
}));
export default ChatTeardropDots;
