// phosphor-react-native/lib/module/light/CircleHalf.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CircleHalf = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "circle-half-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 26a102 102 0 1 0 102 102A102.12 102.12 0 0 0 128 26Zm6 12.2a89.86 89.86 0 0 1 20 3.63v172.34a89.86 89.86 0 0 1-20 3.63Zm32 8.23a90.48 90.48 0 0 1 20 12.81v137.52a90.48 90.48 0 0 1-20 12.81ZM38 128a90.12 90.12 0 0 1 84-89.8v179.6A90.12 90.12 0 0 1 38 128Zm160 56.5v-113a89.81 89.81 0 0 1 0 113Z"
}));
export default CircleHalf;
