// phosphor-react-native/lib/module/light/CirclesThree.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CirclesThree = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "circles-three-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M170 76a42 42 0 1 0-42 42 42 42 0 0 0 42-42Zm-42 30a30 30 0 1 1 30-30 30 30 0 0 1-30 30Zm60 24a42 42 0 1 0 42 42 42 42 0 0 0-42-42Zm0 72a30 30 0 1 1 30-30 30 30 0 0 1-30 30ZM68 130a42 42 0 1 0 42 42 42 42 0 0 0-42-42Zm0 72a30 30 0 1 1 30-30 30 30 0 0 1-30 30Z"
}));
export default CirclesThree;
