// phosphor-react-native/lib/module/light/ClockCountdown.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ClockCountdown = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "clock-countdown-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M230 136.49A102.12 102.12 0 1 1 119.51 26a6 6 0 0 1 1 12A90.13 90.13 0 1 0 218 135.51a6 6 0 1 1 12 1ZM122 72v56a6 6 0 0 0 6 6h56a6 6 0 0 0 0-12h-50V72a6 6 0 0 0-12 0Zm38-26a10 10 0 1 0-10-10 10 10 0 0 0 10 10Zm36 24a10 10 0 1 0-10-10 10 10 0 0 0 10 10Zm24 36a10 10 0 1 0-10-10 10 10 0 0 0 10 10Z"
}));
export default ClockCountdown;
