// phosphor-react-native/lib/module/light/ClosedCaptioning.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ClosedCaptioning = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "closed-captioning-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 50H32a14 14 0 0 0-14 14v128a14 14 0 0 0 14 14h192a14 14 0 0 0 14-14V64a14 14 0 0 0-14-14Zm2 142a2 2 0 0 1-2 2H32a2 2 0 0 1-2-2V64a2 2 0 0 1 2-2h192a2 2 0 0 1 2 2Zm-108.81-39.29a6 6 0 0 1-2.19 8.2 38 38 0 1 1 0-65.82 6 6 0 1 1-6 10.38 26 26 0 1 0 0 45.05 6 6 0 0 1 8.19 2.19Zm80 0a6 6 0 0 1-2.19 8.2 38 38 0 1 1 0-65.82 6 6 0 1 1-6 10.38 26 26 0 1 0 0 45.05 6 6 0 0 1 8.19 2.19Z"
}));
export default ClosedCaptioning;
