// phosphor-react-native/lib/module/light/CloudSnow.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CloudSnow = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cloud-snow-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M86 196a10 10 0 1 1-10-10 10 10 0 0 1 10 10Zm30 6a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm48-16a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm-96 40a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm88 0a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm74-134a74.09 74.09 0 0 1-74 74H76a50 50 0 1 1 10.2-99A74.08 74.08 0 0 1 230 92Zm-12 0a62.06 62.06 0 0 0-124-3.65 6 6 0 0 1-12-.7 75.84 75.84 0 0 1 1.07-9A38 38 0 1 0 76 154h80a62.07 62.07 0 0 0 62-62Z"
}));
export default CloudSnow;
