// phosphor-react-native/lib/module/light/CloudLightning.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CloudLightning = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cloud-lightning-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M156 18a74.21 74.21 0 0 0-69.89 49.3 6.18 6.18 0 0 0-1.83-.62A50.83 50.83 0 0 0 76 66a50 50 0 0 0 0 100h41.4l-18.55 30.91A6 6 0 0 0 104 206h29.4l-18.55 30.91a6 6 0 0 0 10.3 6.18l24-40A6 6 0 0 0 144 194h-29.4l16.8-28H156a74 74 0 0 0 0-148Zm0 136H76a38 38 0 1 1 6.31-75.48 6.82 6.82 0 0 0 .79.08 72.86 72.86 0 0 0-1.1 9.05 6 6 0 0 0 12 .7A62.06 62.06 0 1 1 156 154Z"
}));
export default CloudLightning;
