// phosphor-react-native/lib/module/light/CookingPot.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CookingPot = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cooking-pot-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M90 48V16a6 6 0 0 1 12 0v32a6 6 0 0 1-12 0Zm38 6a6 6 0 0 0 6-6V16a6 6 0 0 0-12 0v32a6 6 0 0 0 6 6Zm32 0a6 6 0 0 0 6-6V16a6 6 0 0 0-12 0v32a6 6 0 0 0 6 6Zm91.6 46.8L222 123v61a30 30 0 0 1-30 30H64a30 30 0 0 1-30-30v-61L4.4 100.8a6 6 0 0 1 7.2-9.6L34 108V88a14 14 0 0 1 14-14h160a14 14 0 0 1 14 14v20l22.4-16.8a6 6 0 0 1 7.2 9.6ZM210 88a2 2 0 0 0-2-2H48a2 2 0 0 0-2 2v96a18 18 0 0 0 18 18h128a18 18 0 0 0 18-18Z"
}));
export default CookingPot;
