// phosphor-react-native/lib/module/light/Couch.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Couch = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "couch-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M238 107.37V72a14 14 0 0 0-14-14H32a14 14 0 0 0-14 14v35.37A14 14 0 0 0 10 120v48a14 14 0 0 0 14 14h10v18a6 6 0 0 0 12 0v-18h164v18a6 6 0 0 0 12 0v-18h10a14 14 0 0 0 14-14v-48a14 14 0 0 0-8-12.63ZM226 72v34h-10a14 14 0 0 0-14 14v16a2 2 0 0 1-2 2h-66V70h90a2 2 0 0 1 2 2ZM32 70h90v68H56a2 2 0 0 1-2-2v-16a14 14 0 0 0-14-14H30V72a2 2 0 0 1 2-2Zm202 98a2 2 0 0 1-2 2H24a2 2 0 0 1-2-2v-48a2 2 0 0 1 2-2h16a2 2 0 0 1 2 2v16a14 14 0 0 0 14 14h144a14 14 0 0 0 14-14v-16a2 2 0 0 1 2-2h16a2 2 0 0 1 2 2Z"
}));
export default Couch;
