// phosphor-react-native/lib/module/light/Cross.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Cross = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cross-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M200 74h-40a2 2 0 0 1-2-2V32a14 14 0 0 0-14-14h-32a14 14 0 0 0-14 14v40a2 2 0 0 1-2 2H56a14 14 0 0 0-14 14v32a14 14 0 0 0 14 14h40a2 2 0 0 1 2 2v88a14 14 0 0 0 14 14h32a14 14 0 0 0 14-14v-88a2 2 0 0 1 2-2h40a14 14 0 0 0 14-14V88a14 14 0 0 0-14-14Zm2 46a2 2 0 0 1-2 2h-40a14 14 0 0 0-14 14v88a2 2 0 0 1-2 2h-32a2 2 0 0 1-2-2v-88a14 14 0 0 0-14-14H56a2 2 0 0 1-2-2V88a2 2 0 0 1 2-2h40a14 14 0 0 0 14-14V32a2 2 0 0 1 2-2h32a2 2 0 0 1 2 2v40a14 14 0 0 0 14 14h40a2 2 0 0 1 2 2Z"
}));
export default Cross;
