// phosphor-react-native/lib/module/light/CurrencyBtc.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CurrencyBtc = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "currency-btc-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M166.69 116.41A42 42 0 0 0 142 42.05V24a6 6 0 0 0-12 0v18h-20V24a6 6 0 0 0-12 0v18H64a6 6 0 0 0 0 12h10v140H64a6 6 0 0 0 0 12h34v18a6 6 0 0 0 12 0v-18h20v18a6 6 0 0 0 12 0v-18h10a46 46 0 0 0 14.69-89.59ZM170 84a30 30 0 0 1-30 30H86V54h54a30 30 0 0 1 30 30Zm-18 110H86v-68h66a34 34 0 0 1 0 68Z"
}));
export default CurrencyBtc;
