// phosphor-react-native/lib/module/light/Disc.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Disc = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "disc-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 26a102 102 0 1 0 102 102A102.12 102.12 0 0 0 128 26Zm37.52 96a37.79 37.79 0 0 0-6.76-16.27l37-36.95A89.61 89.61 0 0 1 217.8 122ZM154 128a26 26 0 1 1-26-26 26 26 0 0 1 26 26Zm-26 90a90 90 0 1 1 59.22-157.71l-36.95 36.95A38 38 0 1 0 165.52 134h52.28a90.12 90.12 0 0 1-89.8 84Z"
}));
export default Disc;
