// phosphor-react-native/lib/module/light/DotsNine.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const DotsNine = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "dots-nine-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M70 60a10 10 0 1 1-10-10 10 10 0 0 1 10 10Zm58-10a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm68 20a10 10 0 1 0-10-10 10 10 0 0 0 10 10ZM60 118a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm68 0a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm68 0a10 10 0 1 0 10 10 10 10 0 0 0-10-10ZM60 186a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm68 0a10 10 0 1 0 10 10 10 10 0 0 0-10-10Zm68 0a10 10 0 1 0 10 10 10 10 0 0 0-10-10Z"
}));
export default DotsNine;
