// phosphor-react-native/lib/module/light/DotsThreeCircleVertical.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const DotsThreeCircleVertical = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "dots-three-circle-vertical-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 26a102 102 0 1 0 102 102A102.12 102.12 0 0 0 128 26Zm0 192a90 90 0 1 1 90-90 90.1 90.1 0 0 1-90 90Zm10-134a10 10 0 1 1-10-10 10 10 0 0 1 10 10Zm0 44a10 10 0 1 1-10-10 10 10 0 0 1 10 10Zm0 44a10 10 0 1 1-10-10 10 10 0 0 1 10 10Z"
}));
export default DotsThreeCircleVertical;
