// phosphor-react-native/lib/module/light/Drop.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Drop = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "drop-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M172.53 49.06a252.86 252.86 0 0 0-41.09-38 6 6 0 0 0-6.88 0 252.86 252.86 0 0 0-41.09 38C56.34 80.26 42 113.09 42 144a86 86 0 0 0 172 0c0-30.91-14.34-63.74-41.47-94.94ZM128 218a74.09 74.09 0 0 1-74-74c0-59.62 59-108.93 74-120.51 15 11.58 74 60.89 74 120.51a74.09 74.09 0 0 1-74 74Zm53.92-65A55.58 55.58 0 0 1 137 197.92a7 7 0 0 1-1 .08 6 6 0 0 1-1-11.92c17.38-2.92 32.13-17.68 35.08-35.08a6 6 0 1 1 11.84 2Z"
}));
export default Drop;
