// phosphor-react-native/lib/module/light/EggCrack.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const EggCrack = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "egg-crack-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M185 60.67C167.18 34 145.87 18 128 18S88.82 34 71 60.67C52.57 88.32 42 121.61 42 152a86 86 0 0 0 172 0c0-30.39-10.57-63.68-29-91.33ZM128 226a74.09 74.09 0 0 1-74-74c0-28.08 9.84-58.94 27-84.67C96.11 44.65 114.56 30 128 30c10.52 0 24.12 9 36.78 23.77L123.52 100a6 6 0 0 0 3 9.82l26.48 6.61-6.87 34.39a6 6 0 0 0 4.7 7.06 6.08 6.08 0 0 0 1.17.12 6 6 0 0 0 5.87-4.82l8-40a6 6 0 0 0-4.42-7l-22.37-5.59 33.2-37.2q1.36 1.92 2.72 3.94c17.15 25.73 27 56.59 27 84.67a74.09 74.09 0 0 1-74 74Z"
}));
export default EggCrack;
