// phosphor-react-native/lib/module/light/Equalizer.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Equalizer = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "equalizer-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M78 96a6 6 0 0 1-6 6H24a6 6 0 0 1 0-12h48a6 6 0 0 1 6 6Zm-6 26H24a6 6 0 0 0 0 12h48a6 6 0 0 0 0-12Zm0 32H24a6 6 0 0 0 0 12h48a6 6 0 0 0 0-12Zm0 32H24a6 6 0 0 0 0 12h48a6 6 0 0 0 0-12Zm80-64h-48a6 6 0 0 0 0 12h48a6 6 0 0 0 0-12Zm0 32h-48a6 6 0 0 0 0 12h48a6 6 0 0 0 0-12Zm0 32h-48a6 6 0 0 0 0 12h48a6 6 0 0 0 0-12Zm80-96h-48a6 6 0 0 0 0 12h48a6 6 0 0 0 0-12Zm-48-20h48a6 6 0 0 0 0-12h-48a6 6 0 0 0 0 12Zm48 52h-48a6 6 0 0 0 0 12h48a6 6 0 0 0 0-12Zm0 32h-48a6 6 0 0 0 0 12h48a6 6 0 0 0 0-12Zm0 32h-48a6 6 0 0 0 0 12h48a6 6 0 0 0 0-12Z"
}));
export default Equalizer;
