// phosphor-react-native/lib/module/light/FadersHorizontal.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FadersHorizontal = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "faders-horizontal-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M34 80a6 6 0 0 1 6-6h32a6 6 0 0 1 0 12H40a6 6 0 0 1-6-6Zm182 90h-42v-18a6 6 0 0 0-12 0v48a6 6 0 0 0 12 0v-18h42a6 6 0 0 0 0-12Zm-80 0H40a6 6 0 0 0 0 12h96a6 6 0 0 0 0-12Zm-32-60a6 6 0 0 0 6-6V86h106a6 6 0 0 0 0-12H110V56a6 6 0 0 0-12 0v48a6 6 0 0 0 6 6Z"
}));
export default FadersHorizontal;
