// phosphor-react-native/lib/module/light/FastForward.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FastForward = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "fast-forward-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m239.59 116.35-88.18-56.17a14 14 0 0 0-14.22-.46A13.83 13.83 0 0 0 130 71.84v41L47.41 60.18a14 14 0 0 0-14.22-.46A13.83 13.83 0 0 0 26 71.84v112.32a13.83 13.83 0 0 0 7.19 12.12 14 14 0 0 0 14.22-.46L130 143.21v40.95a13.83 13.83 0 0 0 7.19 12.12 14 14 0 0 0 14.22-.46l88.18-56.17a13.79 13.79 0 0 0 0-23.3Zm-110.44 13.18L41 185.69a1.9 1.9 0 0 1-2 .06 1.73 1.73 0 0 1-1-1.59V71.84a1.73 1.73 0 0 1 1-1.59 2 2 0 0 1 1-.26 1.87 1.87 0 0 1 1 .32l88.19 56.16a1.8 1.8 0 0 1 0 3.06Zm104 0L145 185.69a1.9 1.9 0 0 1-2 .06 1.73 1.73 0 0 1-1-1.59V71.84a1.73 1.73 0 0 1 1-1.59 2 2 0 0 1 1-.26 1.87 1.87 0 0 1 1 .32l88.19 56.16a1.8 1.8 0 0 1 0 3.06Z"
}));
export default FastForward;
