// phosphor-react-native/lib/module/light/FastForwardCircle.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FastForwardCircle = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "fast-forward-circle-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 26a102 102 0 1 0 102 102A102.12 102.12 0 0 0 128 26Zm0 192a90 90 0 1 1 90-90 90.1 90.1 0 0 1-90 90Zm63.33-95-48-32a6 6 0 0 0-9.33 5v26.12L87.33 91A6 6 0 0 0 78 96v64a6 6 0 0 0 9.33 5L134 133.88V160a6 6 0 0 0 9.33 5l48-32a6 6 0 0 0 0-10ZM90 148.79v-41.58L121.18 128Zm56 0v-41.58L177.18 128Z"
}));
export default FastForwardCircle;
