// phosphor-react-native/lib/module/light/FigmaLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FigmaLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "figma-logo-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M180.45 96A38 38 0 0 0 160 26H88a38 38 0 0 0-20.45 70A38 38 0 0 0 69 160.89 42 42 0 1 0 134 196v-40.32A38 38 0 1 0 180.45 96ZM186 64a26 26 0 0 1-26 26h-26V38h26a26 26 0 0 1 26 26ZM62 64a26 26 0 0 1 26-26h34v52H88a26 26 0 0 1-26-26Zm26 90a26 26 0 0 1 0-52h34v52H88Zm34 42a30 30 0 1 1-30-30h30Zm38-42a26 26 0 1 1 26-26 26 26 0 0 1-26 26Z"
}));
export default FigmaLogo;
