// phosphor-react-native/lib/module/light/FilmReel.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FilmReel = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "film-reel-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 218h-48a102 102 0 1 0-48 12h96a6 6 0 0 0 0-12ZM38 128a90 90 0 1 1 90 90 90.1 90.1 0 0 1-90-90Zm90-26a22 22 0 1 0-22-22 22 22 0 0 0 22 22Zm0-32a10 10 0 1 1-10 10 10 10 0 0 1 10-10Zm22 106a22 22 0 1 0-22 22 22 22 0 0 0 22-22Zm-32 0a10 10 0 1 1 10 10 10 10 0 0 1-10-10Zm58-26a22 22 0 1 0-22-22 22 22 0 0 0 22 22Zm0-32a10 10 0 1 1-10 10 10 10 0 0 1 10-10Zm-96-12a22 22 0 1 0 22 22 22 22 0 0 0-22-22Zm0 32a10 10 0 1 1 10-10 10 10 0 0 1-10 10Z"
}));
export default FilmReel;
