// phosphor-react-native/lib/module/light/FramerLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FramerLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "framer-logo-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M206 96V32a6 6 0 0 0-6-6H56a6 6 0 0 0-4 10.48L112.22 90H56a6 6 0 0 0-6 6v64a6 6 0 0 0 1.76 4.24l72 72A6 6 0 0 0 134 232v-66h66a6 6 0 0 0 4-10.48L143.78 102H200a6 6 0 0 0 6-6Zm-21.78 58H128a6 6 0 0 0-6 6v57.51l-60-60V102h63.72ZM194 90h-63.72l-58.5-52H194Z"
}));
export default FramerLogo;
