// phosphor-react-native/lib/module/light/GitDiff.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GitDiff = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "git-diff-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M112 154a6 6 0 0 0-6 6v33.52l-31.7-31.7a41.75 41.75 0 0 1-12.3-29.7V93.4a30 30 0 1 0-12 0v38.72a53.65 53.65 0 0 0 15.82 38.18l31.7 31.7H64a6 6 0 0 0 0 12h48a6 6 0 0 0 6-6v-48a6 6 0 0 0-6-6ZM38 64a18 18 0 1 1 18 18 18 18 0 0 1-18-18Zm168 98.6v-38.72a53.65 53.65 0 0 0-15.82-38.18L158.48 54H192a6 6 0 0 0 0-12h-48a6 6 0 0 0-6 6v48a6 6 0 0 0 12 0V62.48l31.7 31.7a41.75 41.75 0 0 1 12.3 29.7v38.72a30 30 0 1 0 12 0Zm-6 47.4a18 18 0 1 1 18-18 18 18 0 0 1-18 18Z"
}));
export default GitDiff;
