// phosphor-react-native/lib/module/light/GitFork.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GitFork = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "git-fork-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M222 64a30 30 0 1 0-36 29.4V104a18 18 0 0 1-18 18H88a18 18 0 0 1-18-18V93.4a30 30 0 1 0-12 0V104a30 30 0 0 0 30 30h34v28.6a30 30 0 1 0 12 0V134h34a30 30 0 0 0 30-30V93.4A30.05 30.05 0 0 0 222 64ZM46 64a18 18 0 1 1 18 18 18 18 0 0 1-18-18Zm100 128a18 18 0 1 1-18-18 18 18 0 0 1 18 18Zm46-110a18 18 0 1 1 18-18 18 18 0 0 1-18 18Z"
}));
export default GitFork;
