// phosphor-react-native/lib/module/light/GooglePlayLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GooglePlayLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "google-play-logo-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M222.84 115.93 55 19.89a14 14 0 0 0-14.12 0A13.68 13.68 0 0 0 34 31.87v192.26a13.68 13.68 0 0 0 6.92 11.94 14 14 0 0 0 14.12 0l167.8-96a13.75 13.75 0 0 0 0-24.14ZM46 217.5v-179l89.51 89.5Zm98-81 22.19 22.19L62.4 218.07ZM62.4 37.93l103.79 59.4L144 119.52ZM217 129.58l-.1.06-39.9 22.85L152.49 128 177 103.51l39.94 22.85.1.06a1.76 1.76 0 0 1 0 3.16Z"
}));
export default GooglePlayLogo;
