// phosphor-react-native/lib/module/light/IceCream.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const IceCream = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "ice-cream-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M206 90.83V88a78 78 0 0 0-156 0v2.83A22 22 0 0 0 56 134h4.52l62.27 109a6 6 0 0 0 10.42 0l62.27-109H200a22 22 0 0 0 6-43.17Zm-78 137.08L74.34 134h23.89L140 207Zm8-93.91 22.85 40-11.94 20.91L112.05 134Zm29.76 27.91L149.77 134h31.89ZM200 122H56a10 10 0 0 1 0-20 6 6 0 0 0 6-6v-8a66 66 0 0 1 132 0v8a6 6 0 0 0 6 6 10 10 0 0 1 0 20Z"
}));
export default IceCream;
