// phosphor-react-native/lib/module/light/Lightning.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Lightning = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "lightning-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M213.84 118.63a6 6 0 0 0-3.73-4.25l-59.23-22.21 15-75a6 6 0 0 0-10.27-5.27l-112 120a6 6 0 0 0 2.28 9.71l59.23 22.21-15 75a6 6 0 0 0 3.14 6.52A6.07 6.07 0 0 0 96 246a6 6 0 0 0 4.39-1.91l112-120a6 6 0 0 0 1.45-5.46ZM106 220.46l11.85-59.28a6 6 0 0 0-3.77-6.8l-55.6-20.85 91.46-98-11.82 59.29a6 6 0 0 0 3.77 6.8l55.6 20.85Z"
}));
export default Lightning;
