// phosphor-react-native/lib/module/light/LinkBreak.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const LinkBreak = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "link-break-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M192 64a34 34 0 0 0-48-.05l-11.66 12.19a6 6 0 1 1-8.68-8.28l11.71-12.28.1-.11a46 46 0 0 1 65.06 65.06l-.11.1-12.28 11.71a6 6 0 0 1-8.28-8.68L192.09 112a34 34 0 0 0-.09-48Zm-68.38 115.9L112 192.09A34 34 0 0 1 63.91 144l12.23-11.67a6 6 0 1 0-8.28-8.68l-12.28 11.72-.11.1a46 46 0 0 0 65.06 65.06l.1-.11 11.71-12.28a6 6 0 1 0-8.68-8.28ZM208 154h-24a6 6 0 0 0 0 12h24a6 6 0 0 0 0-12ZM48 102h24a6 6 0 0 0 0-12H48a6 6 0 0 0 0 12Zm112 76a6 6 0 0 0-6 6v24a6 6 0 0 0 12 0v-24a6 6 0 0 0-6-6ZM96 78a6 6 0 0 0 6-6V48a6 6 0 0 0-12 0v24a6 6 0 0 0 6 6Z"
}));
export default LinkBreak;
