// phosphor-react-native/lib/module/light/ListDashes.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ListDashes = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "list-dashes-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M90 64a6 6 0 0 1 6-6h120a6 6 0 0 1 0 12H96a6 6 0 0 1-6-6Zm126 58H96a6 6 0 0 0 0 12h120a6 6 0 0 0 0-12Zm0 64H96a6 6 0 0 0 0 12h120a6 6 0 0 0 0-12ZM56 58H40a6 6 0 0 0 0 12h16a6 6 0 0 0 0-12Zm0 64H40a6 6 0 0 0 0 12h16a6 6 0 0 0 0-12Zm0 64H40a6 6 0 0 0 0 12h16a6 6 0 0 0 0-12Z"
}));
export default ListDashes;
