// phosphor-react-native/lib/module/light/ListMagnifyingGlass.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ListMagnifyingGlass = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "list-magnifying-glass-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M34 64a6 6 0 0 1 6-6h176a6 6 0 0 1 0 12H40a6 6 0 0 1-6-6Zm6 70h72a6 6 0 0 0 0-12H40a6 6 0 0 0 0 12Zm88 52H40a6 6 0 0 0 0 12h88a6 6 0 0 0 0-12Zm108.24 10.24a6 6 0 0 1-8.48 0l-21.49-21.48a38.06 38.06 0 1 1 8.49-8.49l21.48 21.49a6 6 0 0 1 0 8.48ZM184 170a26 26 0 1 0-26-26 26 26 0 0 0 26 26Z"
}));
export default ListMagnifyingGlass;
