// phosphor-react-native/lib/module/light/MedalMilitary.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const MedalMilitary = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "medal-military-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M207 34H49a15 15 0 0 0-15 15v49.21a15 15 0 0 0 8.79 13.65l66.4 30.14a46 46 0 1 0 37.62 0l66.4-30.18A15 15 0 0 0 222 98.21V49a15 15 0 0 0-15-15Zm-45 12v76l-34 15.45L94 122V46ZM46 98.21V49a3 3 0 0 1 3-3h33v70.5l-34.24-15.56A3 3 0 0 1 46 98.21ZM162 184a34 34 0 1 1-34-34 34 34 0 0 1 34 34Zm48-85.79a3 3 0 0 1-1.76 2.73L174 116.5V46h33a3 3 0 0 1 3 3Z"
}));
export default MedalMilitary;
