// phosphor-react-native/lib/module/light/MediumLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const MediumLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "medium-logo-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M72 66a62 62 0 1 0 62 62 62.07 62.07 0 0 0-62-62Zm0 112a50 50 0 1 1 50-50 50.06 50.06 0 0 1-50 50ZM184 66c-17.1 0-30 26.65-30 62s12.9 62 30 62 30-26.65 30-62-12.9-62-30-62Zm0 112c-7.34 0-18-19.48-18-50s10.66-50 18-50 18 19.48 18 50-10.66 50-18 50Zm62-106v112a6 6 0 0 1-12 0V72a6 6 0 0 1 12 0Z"
}));
export default MediumLogo;
