// phosphor-react-native/lib/module/light/Mountains.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Mountains = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "mountains-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M164 78a26 26 0 1 0-26-26 26 26 0 0 0 26 26Zm0-40a14 14 0 1 1-14 14 14 14 0 0 1 14-14Zm89.16 158.94-54.56-92.08a13.9 13.9 0 0 0-12-6.86 13.88 13.88 0 0 0-12 6.86l-27.88 47.05-46.56-79a14 14 0 0 0-24.13 0L2.83 197A6 6 0 0 0 8 206h240a6 6 0 0 0 5.16-9.06ZM86.27 79a2 2 0 0 1 3.46 0l25.34 43H60.93ZM18.5 194l35.36-60h68.29l19.3 32.77 16 27.2Zm152.93 0-17.85-30.29L184.83 111a2 2 0 0 1 1.72-1 1.93 1.93 0 0 1 1.72 1l49.2 83Z"
}));
export default Mountains;
