// phosphor-react-native/lib/module/light/MusicNotes.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const MusicNotes = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "music-notes-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M211.69 27.27a6 6 0 0 0-5.15-1.09l-128 32A6 6 0 0 0 74 64v114.11A34 34 0 1 0 86 204v-87.32l116-29v58.43A34 34 0 1 0 214 172V32a6 6 0 0 0-2.31-4.73ZM52 226a22 22 0 1 1 22-22 22 22 0 0 1-22 22Zm34-121.68V68.68l116-29v35.64ZM180 194a22 22 0 1 1 22-22 22 22 0 0 1-22 22Z"
}));
export default MusicNotes;
