// phosphor-react-native/lib/module/light/Notebook.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Notebook = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "notebook-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M182 112a6 6 0 0 1-6 6h-64a6 6 0 0 1 0-12h64a6 6 0 0 1 6 6Zm-6 26h-64a6 6 0 0 0 0 12h64a6 6 0 0 0 0-12Zm46-90v160a14 14 0 0 1-14 14H48a14 14 0 0 1-14-14V48a14 14 0 0 1 14-14h160a14 14 0 0 1 14 14ZM48 210h26V46H48a2 2 0 0 0-2 2v160a2 2 0 0 0 2 2ZM210 48a2 2 0 0 0-2-2H86v164h122a2 2 0 0 0 2-2Z"
}));
export default Notebook;
