// phosphor-react-native/lib/module/light/Notepad.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Notepad = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "notepad-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M166 128a6 6 0 0 1-6 6H96a6 6 0 0 1 0-12h64a6 6 0 0 1 6 6Zm-6 26H96a6 6 0 0 0 0 12h64a6 6 0 0 0 0-12Zm54-106v152a30 30 0 0 1-30 30H72a30 30 0 0 1-30-30V48a14 14 0 0 1 14-14h18V24a6 6 0 0 1 12 0v10h36V24a6 6 0 0 1 12 0v10h36V24a6 6 0 0 1 12 0v10h18a14 14 0 0 1 14 14Zm-12 0a2 2 0 0 0-2-2h-18v10a6 6 0 0 1-12 0V46h-36v10a6 6 0 0 1-12 0V46H86v10a6 6 0 0 1-12 0V46H56a2 2 0 0 0-2 2v152a18 18 0 0 0 18 18h112a18 18 0 0 0 18-18Z"
}));
export default Notepad;
