// phosphor-react-native/lib/module/light/NumberCircleEight.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const NumberCircleEight = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "number-circle-eight-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 26a102 102 0 1 0 102 102A102.12 102.12 0 0 0 128 26Zm0 192a90 90 0 1 1 90-90 90.1 90.1 0 0 1-90 90Zm18.57-94.46a30 30 0 1 0-37.14 0 34 34 0 1 0 37.14 0ZM110 100a18 18 0 1 1 18 18 18 18 0 0 1-18-18Zm18 74a22 22 0 1 1 22-22 22 22 0 0 1-22 22Z"
}));
export default NumberCircleEight;
