// phosphor-react-native/lib/module/light/PatreonLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PatreonLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "patreon-logo-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M176 42a62 62 0 1 0 62 62 62.07 62.07 0 0 0-62-62Zm0 112a50 50 0 1 1 50-50 50.06 50.06 0 0 1-50 50ZM80 42H64a14 14 0 0 0-14 14v152a14 14 0 0 0 14 14h16a14 14 0 0 0 14-14V56a14 14 0 0 0-14-14Zm2 166a2 2 0 0 1-2 2H64a2 2 0 0 1-2-2V56a2 2 0 0 1 2-2h16a2 2 0 0 1 2 2Z"
}));
export default PatreonLogo;
