// phosphor-react-native/lib/module/light/Percent.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Percent = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "percent-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m204.24 60.23-144 144a6 6 0 0 1-8.48-8.48l144-144a6 6 0 1 1 8.48 8.49ZM52 100a34 34 0 1 1 24 10 33.78 33.78 0 0 1-24-10Zm2-24a22 22 0 1 0 6.44-15.56A21.86 21.86 0 0 0 54 76Zm160 104a34 34 0 1 1-10-24 33.78 33.78 0 0 1 10 24Zm-12 0a21.87 21.87 0 0 0-6.44-15.56A22 22 0 1 0 202 180Z"
}));
export default Percent;
