// phosphor-react-native/lib/module/light/PictureInPicture.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PictureInPicture = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "picture-in-picture-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 50H40a14 14 0 0 0-14 14v128a14 14 0 0 0 14 14h176a14 14 0 0 0 14-14V64a14 14 0 0 0-14-14ZM38 192V64a2 2 0 0 1 2-2h176a2 2 0 0 1 2 2v58h-74a14 14 0 0 0-14 14v58H40a2 2 0 0 1-2-2Zm178 2h-74v-58a2 2 0 0 1 2-2h74v58a2 2 0 0 1-2 2Z"
}));
export default PictureInPicture;
