// phosphor-react-native/lib/module/light/Radio.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Radio = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "radio-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M102 112a6 6 0 0 1-6 6H64a6 6 0 0 1 0-12h32a6 6 0 0 1 6 6Zm-6 26H64a6 6 0 0 0 0 12h32a6 6 0 0 0 0-12Zm0 32H64a6 6 0 0 0 0 12h32a6 6 0 0 0 0-12Zm134-82v112a14 14 0 0 1-14 14H40a14 14 0 0 1-14-14V80a6 6 0 0 1 4.28-5.75l160-48a6 6 0 0 1 3.44 11.5L72.88 74H216a14 14 0 0 1 14 14Zm-12 0a2 2 0 0 0-2-2H38v114a2 2 0 0 0 2 2h176a2 2 0 0 0 2-2Zm-20 56a38 38 0 1 1-38-38 38 38 0 0 1 38 38Zm-12 0a26 26 0 1 0-26 26 26 26 0 0 0 26-26Z"
}));
export default Radio;
