// phosphor-react-native/lib/module/light/Robot.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Robot = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "robot-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M200 50h-66V16a6 6 0 0 0-12 0v34H56a30 30 0 0 0-30 30v112a30 30 0 0 0 30 30h144a30 30 0 0 0 30-30V80a30 30 0 0 0-30-30Zm18 142a18 18 0 0 1-18 18H56a18 18 0 0 1-18-18V80a18 18 0 0 1 18-18h144a18 18 0 0 1 18 18Zm-54-54H92a26 26 0 0 0 0 52h72a26 26 0 0 0 0-52Zm-26 12v28h-20v-28Zm-60 14a14 14 0 0 1 14-14h14v28H92a14 14 0 0 1-14-14Zm86 14h-14v-28h14a14 14 0 0 1 0 28Zm-90-70a10 10 0 1 1 10 10 10 10 0 0 1-10-10Zm88 0a10 10 0 1 1 10 10 10 10 0 0 1-10-10Z"
}));
export default Robot;
