// phosphor-react-native/lib/module/light/Rss.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Rss = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "rss-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M97.5 158.5A69.54 69.54 0 0 1 118 208a6 6 0 0 1-12 0 58 58 0 0 0-58-58 6 6 0 0 1 0-12 69.54 69.54 0 0 1 49.5 20.5ZM48 90a6 6 0 0 0 0 12 106 106 0 0 1 106 106 6 6 0 0 0 12 0A118 118 0 0 0 48 90Zm117.38.62A164.92 164.92 0 0 0 48 42a6 6 0 0 0 0 12 153 153 0 0 1 108.89 45.11A153 153 0 0 1 202 208a6 6 0 0 0 12 0 164.92 164.92 0 0 0-48.62-117.38ZM52 194a10 10 0 1 0 10 10 10 10 0 0 0-10-10Z"
}));
export default Rss;
