// phosphor-react-native/lib/module/light/Scooter.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Scooter = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "scooter-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M212 138a34.32 34.32 0 0 0-4.89.36l-11.27-33.82-22.15-66.44A6 6 0 0 0 168 34h-32a6 6 0 0 0 0 12h27.68l19.75 59.25L133.07 170H77.94a34 34 0 1 0-1.44 12H136a6 6 0 0 0 4.74-2.32L188 118.93l7.74 23.23A34 34 0 1 0 212 138ZM44 194a22 22 0 1 1 22-22 22 22 0 0 1-22 22Zm168 0a22 22 0 1 1 22-22 22 22 0 0 1-22 22Z"
}));
export default Scooter;
