// phosphor-react-native/lib/module/light/ShareNetwork.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ShareNetwork = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "share-network-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M176 162a37.91 37.91 0 0 0-28.3 12.67l-48.9-31.43a37.89 37.89 0 0 0 0-30.48l48.9-31.43a38 38 0 1 0-6.5-10.09l-48.9 31.43a38 38 0 1 0 0 50.66l48.9 31.43A38 38 0 1 0 176 162Zm0-132a26 26 0 1 1-26 26 26 26 0 0 1 26-26ZM64 154a26 26 0 1 1 26-26 26 26 0 0 1-26 26Zm112 72a26 26 0 1 1 26-26 26 26 0 0 1-26 26Z"
}));
export default ShareNetwork;
