// phosphor-react-native/lib/module/light/ShoppingBagOpen.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ShoppingBagOpen = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "shopping-bag-open-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 42H40a14 14 0 0 0-14 14v144a14 14 0 0 0 14 14h176a14 14 0 0 0 14-14V56a14 14 0 0 0-14-14ZM40 54h176a2 2 0 0 1 2 2v18H38V56a2 2 0 0 1 2-2Zm176 148H40a2 2 0 0 1-2-2V86h180v114a2 2 0 0 1-2 2Zm-42-90a46 46 0 0 1-92 0 6 6 0 0 1 12 0 34 34 0 0 0 68 0 6 6 0 0 1 12 0Z"
}));
export default ShoppingBagOpen;
