// phosphor-react-native/lib/module/light/SkipForwardCircle.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SkipForwardCircle = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "skip-forward-circle-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 26a102 102 0 1 0 102 102A102.12 102.12 0 0 0 128 26Zm0 192a90 90 0 1 1 90-90 90.1 90.1 0 0 1-90 90Zm32-136a6 6 0 0 0-6 6v29.17L99.18 82.91A6 6 0 0 0 90 88v80a6 6 0 0 0 9.18 5.09L154 138.83V168a6 6 0 0 0 12 0V88a6 6 0 0 0-6-6Zm-58 75.17V98.83L148.68 128Z"
}));
export default SkipForwardCircle;
