// phosphor-react-native/lib/module/light/SmileyXEyes.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SmileyXEyes = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "smiley-x-eyes-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 26a102 102 0 1 0 102 102A102.12 102.12 0 0 0 128 26Zm0 192a90 90 0 1 1 90-90 90.1 90.1 0 0 1-90 90Zm60.24-117.76L176.49 112l11.75 11.76a6 6 0 1 1-8.48 8.48L168 120.49l-11.76 11.75a6 6 0 0 1-8.48-8.48L159.51 112l-11.75-11.76a6 6 0 0 1 8.48-8.48L168 103.51l11.76-11.75a6 6 0 0 1 8.48 8.48Zm-80 0L96.49 112l11.75 11.76a6 6 0 1 1-8.48 8.48L88 120.49l-11.76 11.75a6 6 0 0 1-8.48-8.48L79.51 112l-11.75-11.76a6 6 0 0 1 8.48-8.48L88 103.51l11.76-11.75a6 6 0 0 1 8.48 8.48ZM138 180a10 10 0 1 1-10-10 10 10 0 0 1 10 10Z"
}));
export default SmileyXEyes;
