// phosphor-react-native/lib/module/light/SmileyNervous.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SmileyNervous = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "smiley-nervous-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M180.24 163.76a6 6 0 1 1-8.48 8.48L160 160.49l-11.76 11.75a6 6 0 0 1-8.48 0L128 160.49l-11.76 11.75a6 6 0 0 1-8.48 0L96 160.49l-11.76 11.75a6 6 0 0 1-8.48-8.48l16-16a6 6 0 0 1 8.48 0L112 159.51l11.76-11.75a6 6 0 0 1 8.48 0L144 159.51l11.76-11.75a6 6 0 0 1 8.48 0ZM230 128A102 102 0 1 1 128 26a102.12 102.12 0 0 1 102 102Zm-12 0a90 90 0 1 0-90 90 90.1 90.1 0 0 0 90-90ZM92 118a10 10 0 1 0-10-10 10 10 0 0 0 10 10Zm72 0a10 10 0 1 0-10-10 10 10 0 0 0 10 10Z"
}));
export default SmileyNervous;
