// phosphor-react-native/lib/module/light/StarFour.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const StarFour = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "star-four-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m228.81 114.89-63.45-23.08a1.94 1.94 0 0 1-1.17-1.17l-23.08-63.45a13.95 13.95 0 0 0-26.22 0L91.81 90.64a1.94 1.94 0 0 1-1.17 1.17l-63.45 23.08a13.95 13.95 0 0 0 0 26.22l63.45 23.08a1.94 1.94 0 0 1 1.17 1.17l23.08 63.45a13.95 13.95 0 0 0 26.22 0l23.08-63.45a1.94 1.94 0 0 1 1.17-1.17l63.45-23.08a13.95 13.95 0 0 0 0-26.22Zm-4.1 15-63.45 23.07a14 14 0 0 0-8.35 8.35l-23.07 63.45a2 2 0 0 1-3.68 0l-23.07-63.45a14 14 0 0 0-8.35-8.35l-63.45-23.12a2 2 0 0 1 0-3.68l63.45-23.07a14 14 0 0 0 8.35-8.35l23.07-63.45a2 2 0 0 1 3.68 0l23.07 63.45a14 14 0 0 0 8.35 8.35l63.45 23.07a2 2 0 0 1 0 3.68Z"
}));
export default StarFour;
