// phosphor-react-native/lib/module/light/Stop.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Stop = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "stop-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M200.73 42H55.27A13.28 13.28 0 0 0 42 55.27v145.46A13.28 13.28 0 0 0 55.27 214h145.46A13.28 13.28 0 0 0 214 200.73V55.27A13.28 13.28 0 0 0 200.73 42ZM202 200.73a1.27 1.27 0 0 1-1.27 1.27H55.27a1.27 1.27 0 0 1-1.27-1.27V55.27A1.27 1.27 0 0 1 55.27 54h145.46a1.27 1.27 0 0 1 1.27 1.27Z"
}));
export default Stop;
