// phosphor-react-native/lib/module/light/Storefront.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Storefront = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "storefront-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M230 96a6.19 6.19 0 0 0-.22-1.65l-14.34-50.2A14.07 14.07 0 0 0 202 34H54a14.07 14.07 0 0 0-13.43 10.15l-14.34 50.2A6.19 6.19 0 0 0 26 96v16a38 38 0 0 0 16 31v65a14 14 0 0 0 14 14h144a14 14 0 0 0 14-14v-65a38 38 0 0 0 16-31ZM52.11 47.45A2 2 0 0 1 54 46h148a2 2 0 0 1 1.92 1.45L216.05 90H40ZM102 102h52v10a26 26 0 0 1-52 0Zm-64 0h52v10a26 26 0 0 1-52 0Zm164 106a2 2 0 0 1-2 2H56a2 2 0 0 1-2-2v-59.34a38 38 0 0 0 42-16.21 37.95 37.95 0 0 0 64 0 38 38 0 0 0 42 16.21Zm-10-70a26 26 0 0 1-26-26v-10h52v10a26 26 0 0 1-26 26Z"
}));
export default Storefront;
