// phosphor-react-native/lib/module/light/TagChevron.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TagChevron = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "tag-chevron-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m235.64 120.23-42.64-64A14 14 0 0 0 181.33 50H24a6 6 0 0 0-4.94 9.41l46.6 67.48a2 2 0 0 1 0 2.14l-46.6 67.56A6 6 0 0 0 24 206h157.33a14 14 0 0 0 11.67-6.23l42.67-64a14 14 0 0 0-.03-15.54Zm-10 8.88-42.67 64a2 2 0 0 1-1.66.89H35.43l40.21-58.23a14 14 0 0 0 0-15.62L35.43 62h145.9a2 2 0 0 1 1.66.89l42.67 64a2 2 0 0 1 0 2.22Z"
}));
export default TagChevron;
