// phosphor-react-native/lib/module/light/TextHFour.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TextHFour = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "text-h-four-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M254 176a6 6 0 0 1-6 6h-10v26a6 6 0 0 1-12 0v-26h-42a6 6 0 0 1-5.69-7.9l24-72a6 6 0 1 1 11.38 3.8L192.32 170H226v-26a6 6 0 0 1 12 0v26h10a6 6 0 0 1 6 6ZM144 50a6 6 0 0 0-6 6v54H46V56a6 6 0 0 0-12 0v120a6 6 0 0 0 12 0v-54h92v54a6 6 0 0 0 12 0V56a6 6 0 0 0-6-6Z"
}));
export default TextHFour;
