// phosphor-react-native/lib/module/light/TextColumns.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TextColumns = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "text-columns-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M118 64a6 6 0 0 1-6 6H40a6 6 0 0 1 0-12h72a6 6 0 0 1 6 6Zm-6 34H40a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Zm0 40H40a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Zm0 40H40a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Zm32-108h72a6 6 0 0 0 0-12h-72a6 6 0 0 0 0 12Zm72 28h-72a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Zm0 40h-72a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Zm0 40h-72a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Z"
}));
export default TextColumns;
