// phosphor-react-native/lib/module/light/Ticket.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Ticket = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "ticket-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M226.79 102.52A14 14 0 0 0 238 88.81V64a14 14 0 0 0-14-14H32a14 14 0 0 0-14 14v24.81a14 14 0 0 0 11.21 13.71 26 26 0 0 1 0 51A14 14 0 0 0 18 167.19V192a14 14 0 0 0 14 14h192a14 14 0 0 0 14-14v-24.81a14 14 0 0 0-11.21-13.71 26 26 0 0 1 0-51ZM30 192v-24.81a2 2 0 0 1 1.59-1.95 38 38 0 0 0 0-74.48 2 2 0 0 1-1.59-2V64a2 2 0 0 1 2-2h58v132H32a2 2 0 0 1-2-2Zm194.41-26.76a2 2 0 0 1 1.59 1.95V192a2 2 0 0 1-2 2H102V62h122a2 2 0 0 1 2 2v24.81a2 2 0 0 1-1.59 2 38 38 0 0 0 0 74.48Z"
}));
export default Ticket;
