// phosphor-react-native/lib/module/light/Tram.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Tram = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "tram-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M184 50h-50V22h34a6 6 0 0 0 0-12H88a6 6 0 0 0 0 12h34v28H72a30 30 0 0 0-30 30v104a30 30 0 0 0 30 30h12l-16.8 22.4a6 6 0 1 0 9.6 7.2L99 214h58l22.2 29.6a6 6 0 0 0 9.6-7.2L172 214h12a30 30 0 0 0 30-30V80a30 30 0 0 0-30-30ZM72 62h112a18 18 0 0 1 18 18v42H54V80a18 18 0 0 1 18-18Zm112 140H72a18 18 0 0 1-18-18v-50h148v50a18 18 0 0 1-18 18Zm-90-30a10 10 0 1 1-10-10 10 10 0 0 1 10 10Zm88 0a10 10 0 1 1-10-10 10 10 0 0 1 10 10Z"
}));
export default Tram;
