// phosphor-react-native/lib/module/light/UserList.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const UserList = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "user-list-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M146 80a6 6 0 0 1 6-6h96a6 6 0 0 1 0 12h-96a6 6 0 0 1-6-6Zm102 42h-96a6 6 0 0 0 0 12h96a6 6 0 0 0 0-12Zm0 48h-72a6 6 0 0 0 0 12h72a6 6 0 0 0 0-12Zm-98.19 20.5a6 6 0 1 1-11.62 3C131.7 168.29 107.23 150 80 150s-51.7 18.29-58.19 43.49a6 6 0 1 1-11.62-3c5.74-22.28 23-40.07 44.67-48a46 46 0 1 1 50.28 0c21.65 7.94 38.94 25.73 44.67 48.01ZM80 138a34 34 0 1 0-34-34 34 34 0 0 0 34 34Z"
}));
export default UserList;
