// phosphor-react-native/lib/module/light/Vibrate.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Vibrate = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "vibrate-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M160 34H96a22 22 0 0 0-22 22v144a22 22 0 0 0 22 22h64a22 22 0 0 0 22-22V56a22 22 0 0 0-22-22Zm10 166a10 10 0 0 1-10 10H96a10 10 0 0 1-10-10V56a10 10 0 0 1 10-10h64a10 10 0 0 1 10 10Zm44-112v80a6 6 0 0 1-12 0V88a6 6 0 0 1 12 0Zm32 16v48a6 6 0 0 1-12 0v-48a6 6 0 0 1 12 0ZM54 88v80a6 6 0 0 1-12 0V88a6 6 0 0 1 12 0Zm-32 16v48a6 6 0 0 1-12 0v-48a6 6 0 0 1 12 0Z"
}));
export default Vibrate;
