// phosphor-react-native/lib/module/light/YinYang.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const YinYang = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "yin-yang-light__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 26a102 102 0 1 0 102 102A102.12 102.12 0 0 0 128 26ZM38 128a90.1 90.1 0 0 1 90-90 42 42 0 0 1 0 84 54 54 0 0 0-44.88 84A90.06 90.06 0 0 1 38 128Zm90 90a42 42 0 0 1 0-84 54 54 0 0 0 44.88-84A90 90 0 0 1 128 218Zm12-42a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm-22-96a10 10 0 1 1 10 10 10 10 0 0 1-10-10Z"
}));
export default YinYang;
