// phosphor-react-native/lib/module/thin/Asterisk.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Asterisk = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "asterisk-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M211.43 178.06a4 4 0 0 1-5.49 1.37L132 135.06V216a4 4 0 0 1-8 0v-80.94l-73.94 44.37a4 4 0 0 1-4.12-6.86L120.22 128 45.94 83.43a4 4 0 0 1 4.12-6.86L124 120.94V40a4 4 0 0 1 8 0v80.94l73.94-44.37a4 4 0 1 1 4.12 6.86L135.78 128l74.28 44.57a4 4 0 0 1 1.37 5.49Z"
}));
export default Asterisk;
