// phosphor-react-native/lib/module/thin/Barricade.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Barricade = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "barricade-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 68H32a12 12 0 0 0-12 12v72a12 12 0 0 0 12 12h28v36a4 4 0 0 0 8 0v-36h120v36a4 4 0 0 0 8 0v-36h28a12 12 0 0 0 12-12V80a12 12 0 0 0-12-12Zm4 12v58.34L165.66 76H224a4 4 0 0 1 4 4ZM32 76h50.34l80 80h-60.68L28 82.34V80a4 4 0 0 1 4-4Zm-4 76V93.66L90.34 156H32a4 4 0 0 1-4-4Zm196 4h-50.34l-80-80h60.68L228 149.66V152a4 4 0 0 1-4 4Z"
}));
export default Barricade;
