// phosphor-react-native/lib/module/thin/BellSlash.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BellSlash = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "bell-slash-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M51 37.31a4 4 0 0 0-6 5.38l18.8 20.63A75.52 75.52 0 0 0 52 104c0 36.13-8.58 64-14.36 73.95A12 12 0 0 0 48 196h44.23a36 36 0 0 0 71.54 0h20.64L205 218.69a4 4 0 1 0 5.92-5.38ZM128 220a28 28 0 0 1-27.71-24h55.42A28 28 0 0 1 128 220Zm-80-32a3.89 3.89 0 0 1-3.43-2 4 4 0 0 1 0-4C52 169.17 60 139.32 60 104a67.58 67.58 0 0 1 9.4-34.51L177.14 188Zm164.55-12.48a3.94 3.94 0 0 1-1.46.28 4 4 0 0 1-3.72-2.54C200.24 155.17 196 129.28 196 104A68 68 0 0 0 94.46 44.83a4 4 0 1 1-4-6.95A76 76 0 0 1 204 104c0 36.05 8.26 59.89 10.79 66.34a4 4 0 0 1-2.26 5.18Z"
}));
export default BellSlash;
