// phosphor-react-native/lib/module/thin/BluetoothConnected.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BluetoothConnected = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "bluetooth-connected-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M186.4 172.8 126.67 128l59.73-44.8a4 4 0 0 0 0-6.4l-64-48A4 4 0 0 0 116 32v88L58.4 76.8a4 4 0 0 0-4.8 6.4l59.73 44.8-59.73 44.8a4 4 0 0 0 4.8 6.4L116 136v88a4 4 0 0 0 6.4 3.2l64-48a4 4 0 0 0 0-6.4ZM124 40l53.33 40L124 120Zm0 176v-80l53.33 40Zm-72-80a8 8 0 1 1 8-8 8 8 0 0 1-8 8Zm152-8a8 8 0 1 1-8-8 8 8 0 0 1 8 8Z"
}));
export default BluetoothConnected;
