// phosphor-react-native/lib/module/thin/Cactus.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Cactus = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cactus-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 212h-52v-32h8a64.07 64.07 0 0 0 64-64 24 24 0 0 0-48 0 16 16 0 0 1-16 16h-8V56a36 36 0 0 0-72 0v36h-8a16 16 0 0 1-16-16 24 24 0 0 0-48 0 64.07 64.07 0 0 0 64 64h8v72H40a4 4 0 0 0 0 8h176a4 4 0 0 0 0-8ZM96 132H84a56.06 56.06 0 0 1-56-56 16 16 0 0 1 32 0 24 24 0 0 0 24 24h12a4 4 0 0 0 4-4V56a28 28 0 0 1 56 0v80a4 4 0 0 0 4 4h12a24 24 0 0 0 24-24 16 16 0 0 1 32 0 56.06 56.06 0 0 1-56 56h-12a4 4 0 0 0-4 4v36h-56v-76a4 4 0 0 0-4-4Z"
}));
export default Cactus;
