// phosphor-react-native/lib/module/thin/Calculator.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Calculator = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "calculator-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M176 60H80a4 4 0 0 0-4 4v48a4 4 0 0 0 4 4h96a4 4 0 0 0 4-4V64a4 4 0 0 0-4-4Zm-4 48H84V68h88Zm28-80H56a12 12 0 0 0-12 12v176a12 12 0 0 0 12 12h144a12 12 0 0 0 12-12V40a12 12 0 0 0-12-12Zm4 188a4 4 0 0 1-4 4H56a4 4 0 0 1-4-4V40a4 4 0 0 1 4-4h144a4 4 0 0 1 4 4ZM96 148a8 8 0 1 1-8-8 8 8 0 0 1 8 8Zm40 0a8 8 0 1 1-8-8 8 8 0 0 1 8 8Zm40 0a8 8 0 1 1-8-8 8 8 0 0 1 8 8Zm-80 40a8 8 0 1 1-8-8 8 8 0 0 1 8 8Zm40 0a8 8 0 1 1-8-8 8 8 0 0 1 8 8Zm40 0a8 8 0 1 1-8-8 8 8 0 0 1 8 8Z"
}));
export default Calculator;
