// phosphor-react-native/lib/module/thin/ChalkboardSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChalkboardSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chalkboard-simple-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M28 168V56a12 12 0 0 1 12-12h176a12 12 0 0 1 12 12v112a4 4 0 0 1-8 0V56a4 4 0 0 0-4-4H40a4 4 0 0 0-4 4v112a4 4 0 0 1-8 0Zm216 32a4 4 0 0 1-4 4H16a4 4 0 0 1 0-8h100v-28a4 4 0 0 1 4-4h64a4 4 0 0 1 4 4v28h52a4 4 0 0 1 4 4Zm-120-4h56v-24h-56Z"
}));
export default ChalkboardSimple;
