// phosphor-react-native/lib/module/thin/ChartLineDown.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChartLineDown = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chart-line-down-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M228 208a4 4 0 0 1-4 4H32a4 4 0 0 1-4-4V48a4 4 0 0 1 8 0v14.34l60 60 29.17-29.17a4 4 0 0 1 5.66 0L196 158.34V128a4 4 0 0 1 8 0v40a4.13 4.13 0 0 1-.08.78 3.37 3.37 0 0 1-.1.34 2.8 2.8 0 0 1-.13.41 2.87 2.87 0 0 1-.2.39c-.05.1-.1.2-.16.3a4.19 4.19 0 0 1-1.11 1.11l-.31.16a3.48 3.48 0 0 1-.38.2 2.8 2.8 0 0 1-.41.13 3.37 3.37 0 0 1-.34.1 4.13 4.13 0 0 1-.78.08h-40a4 4 0 0 1 0-8h30.34L128 101.66l-29.17 29.17a4 4 0 0 1-5.66 0L36 73.66V204h188a4 4 0 0 1 4 4Z"
}));
export default ChartLineDown;
