// phosphor-react-native/lib/module/thin/ChatTeardropText.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChatTeardropText = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chat-teardrop-text-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M164 112a4 4 0 0 1-4 4H96a4 4 0 0 1 0-8h64a4 4 0 0 1 4 4Zm-4 28H96a4 4 0 0 0 0 8h64a4 4 0 0 0 0-8Zm68-16a96.11 96.11 0 0 1-96 96H47.67A11.68 11.68 0 0 1 36 208.33V124a96 96 0 0 1 192 0Zm-8 0a88 88 0 0 0-176 0v84.33a3.67 3.67 0 0 0 3.67 3.67H132a88.1 88.1 0 0 0 88-88Z"
}));
export default ChatTeardropText;
