// phosphor-react-native/lib/module/thin/CircleHalf.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CircleHalf = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "circle-half-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 28a100 100 0 1 0 100 100A100.11 100.11 0 0 0 128 28Zm4 8.09a91.58 91.58 0 0 1 24 4.27v175.28a91.58 91.58 0 0 1-24 4.27Zm32 7.25a92.21 92.21 0 0 1 24 15v139.35a92.21 92.21 0 0 1-24 15ZM36 128a92.11 92.11 0 0 1 88-91.91v183.82A92.11 92.11 0 0 1 36 128Zm160 61.9V66.1a91.83 91.83 0 0 1 0 123.8Z"
}));
export default CircleHalf;
