// phosphor-react-native/lib/module/thin/ClosedCaptioning.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ClosedCaptioning = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "closed-captioning-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 52H32a12 12 0 0 0-12 12v128a12 12 0 0 0 12 12h192a12 12 0 0 0 12-12V64a12 12 0 0 0-12-12Zm4 140a4 4 0 0 1-4 4H32a4 4 0 0 1-4-4V64a4 4 0 0 1 4-4h192a4 4 0 0 1 4 4Zm-112.54-38.29a4 4 0 0 1-1.46 5.47 36 36 0 1 1 0-62.36 4 4 0 0 1-4 6.92 28 28 0 1 0 0 48.52 4 4 0 0 1 5.46 1.45Zm80 0a4 4 0 0 1-1.46 5.47 36 36 0 1 1 0-62.36 4 4 0 0 1-4 6.92 28 28 0 1 0 0 48.52 4 4 0 0 1 5.46 1.45Z"
}));
export default ClosedCaptioning;
