// phosphor-react-native/lib/module/thin/CloudFog.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CloudFog = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cloud-fog-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M120 196H72a4 4 0 0 1 0-8h48a4 4 0 0 1 0 8Zm64-8h-24a4 4 0 0 0 0 8h24a4 4 0 0 0 0-8Zm-24 32h-56a4 4 0 0 0 0 8h56a4 4 0 0 0 0-8Zm68-128a72.08 72.08 0 0 1-72 72H76a48 48 0 1 1 11.51-94.61A72.08 72.08 0 0 1 228 92Zm-8 0a64.06 64.06 0 0 0-128-3.77 4 4 0 0 1-8-.46 71.63 71.63 0 0 1 1.42-10.65A40 40 0 1 0 76 156h80a64.07 64.07 0 0 0 64-64Z"
}));
export default CloudFog;
