// phosphor-react-native/lib/module/thin/CookingPot.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CookingPot = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cooking-pot-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M92 48V16a4 4 0 0 1 8 0v32a4 4 0 0 1-8 0Zm36 4a4 4 0 0 0 4-4V16a4 4 0 0 0-8 0v32a4 4 0 0 0 4 4Zm32 0a4 4 0 0 0 4-4V16a4 4 0 0 0-8 0v32a4 4 0 0 0 4 4Zm90.4 47.2L220 122v62a28 28 0 0 1-28 28H64a28 28 0 0 1-28-28v-62L5.6 99.2a4 4 0 0 1 4.8-6.4L36 112V88a12 12 0 0 1 12-12h160a12 12 0 0 1 12 12v24l25.6-19.2a4 4 0 1 1 4.8 6.4ZM212 88a4 4 0 0 0-4-4H48a4 4 0 0 0-4 4v96a20 20 0 0 0 20 20h128a20 20 0 0 0 20-20Z"
}));
export default CookingPot;
