// phosphor-react-native/lib/module/thin/Couch.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Couch = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "couch-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M236 108.7V72a12 12 0 0 0-12-12H32a12 12 0 0 0-12 12v36.7a12 12 0 0 0-8 11.3v48a12 12 0 0 0 12 12h12v20a4 4 0 0 0 8 0v-20h168v20a4 4 0 0 0 8 0v-20h12a12 12 0 0 0 12-12v-48a12 12 0 0 0-8-11.3ZM228 72v36h-12a12 12 0 0 0-12 12v16a4 4 0 0 1-4 4h-68V68h92a4 4 0 0 1 4 4ZM32 68h92v72H56a4 4 0 0 1-4-4v-16a12 12 0 0 0-12-12H28V72a4 4 0 0 1 4-4Zm204 100a4 4 0 0 1-4 4H24a4 4 0 0 1-4-4v-48a4 4 0 0 1 4-4h16a4 4 0 0 1 4 4v16a12 12 0 0 0 12 12h144a12 12 0 0 0 12-12v-16a4 4 0 0 1 4-4h16a4 4 0 0 1 4 4Z"
}));
export default Couch;
