// phosphor-react-native/lib/module/thin/Cross.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Cross = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cross-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M200 76h-40a4 4 0 0 1-4-4V32a12 12 0 0 0-12-12h-32a12 12 0 0 0-12 12v40a4 4 0 0 1-4 4H56a12 12 0 0 0-12 12v32a12 12 0 0 0 12 12h40a4 4 0 0 1 4 4v88a12 12 0 0 0 12 12h32a12 12 0 0 0 12-12v-88a4 4 0 0 1 4-4h40a12 12 0 0 0 12-12V88a12 12 0 0 0-12-12Zm4 44a4 4 0 0 1-4 4h-40a12 12 0 0 0-12 12v88a4 4 0 0 1-4 4h-32a4 4 0 0 1-4-4v-88a12 12 0 0 0-12-12H56a4 4 0 0 1-4-4V88a4 4 0 0 1 4-4h40a12 12 0 0 0 12-12V32a4 4 0 0 1 4-4h32a4 4 0 0 1 4 4v40a12 12 0 0 0 12 12h40a4 4 0 0 1 4 4Z"
}));
export default Cross;
