// phosphor-react-native/lib/module/thin/Cube.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Cube = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cube-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m221.76 69.66-88-48.18a12 12 0 0 0-11.52 0l-88 48.18A12 12 0 0 0 28 80.18v95.64a12 12 0 0 0 6.24 10.52l88 48.18a11.95 11.95 0 0 0 11.52 0l88-48.18a12 12 0 0 0 6.24-10.52V80.18a12 12 0 0 0-6.24-10.52ZM126.08 28.5a3.94 3.94 0 0 1 3.84 0L216.67 76 128 124.52 39.33 76Zm-88 150.83a4 4 0 0 1-2.08-3.51V83.29l88 48.16v94.91Zm179.84 0-85.92 47v-94.88l88-48.16v92.53a4 4 0 0 1-2.08 3.5Z"
}));
export default Cube;
