// phosphor-react-native/lib/module/thin/CurrencyBtc.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CurrencyBtc = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "currency-btc-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M162.27 117.21A40 40 0 0 0 140 44V24a4 4 0 0 0-8 0v20h-24V24a4 4 0 0 0-8 0v20H64a4 4 0 0 0 0 8h12v144H64a4 4 0 0 0 0 8h36v20a4 4 0 0 0 8 0v-20h24v20a4 4 0 0 0 8 0v-20h12a44 44 0 0 0 10.27-86.79ZM84 52h56a32 32 0 0 1 0 64H84Zm68 144H84v-72h68a36 36 0 0 1 0 72Z"
}));
export default CurrencyBtc;
