// phosphor-react-native/lib/module/thin/DesktopTower.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const DesktopTower = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "desktop-tower-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M20 96v72a12 12 0 0 0 12 12h80a4 4 0 0 1 0 8H92v24h20a4 4 0 0 1 0 8H64a4 4 0 0 1 0-8h20v-24H32a20 20 0 0 1-20-20V96a20 20 0 0 1 20-20h80a4 4 0 0 1 0 8H32a12 12 0 0 0-12 12Zm188-28h-32a4 4 0 0 0 0 8h32a4 4 0 0 0 0-8Zm0 32h-32a4 4 0 0 0 0 8h32a4 4 0 0 0 0-8Zm36-52v160a12 12 0 0 1-12 12h-80a12 12 0 0 1-12-12V48a12 12 0 0 1 12-12h80a12 12 0 0 1 12 12Zm-8 0a4 4 0 0 0-4-4h-80a4 4 0 0 0-4 4v160a4 4 0 0 0 4 4h80a4 4 0 0 0 4-4Zm-44 124a8 8 0 1 0 8 8 8 8 0 0 0-8-8Z"
}));
export default DesktopTower;
