// phosphor-react-native/lib/module/thin/EscalatorDown.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const EscalatorDown = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "escalator-down-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M173.17 82.83a4 4 0 0 1 5.66-5.66L196 94.34V48a4 4 0 0 1 8 0v46.34l17.17-17.17a4 4 0 1 1 5.66 5.66l-24 24a4 4 0 0 1-5.66 0ZM236 160v40a12 12 0 0 1-12 12h-52.5a12.06 12.06 0 0 1-8.82-3.86l-91.24-98.85A4 4 0 0 0 68.5 108H32a12 12 0 0 1-12-12V56a12 12 0 0 1 12-12h52.5a12.06 12.06 0 0 1 8.82 3.86l91.24 98.85a4 4 0 0 0 2.94 1.29H224a12 12 0 0 1 12 12Zm-8 0a4 4 0 0 0-4-4h-36.5a12.06 12.06 0 0 1-8.82-3.86L87.44 53.29A4 4 0 0 0 84.5 52H32a4 4 0 0 0-4 4v40a4 4 0 0 0 4 4h36.5a12.06 12.06 0 0 1 8.82 3.86l91.24 98.85a4 4 0 0 0 2.94 1.29H224a4 4 0 0 0 4-4Z"
}));
export default EscalatorDown;
