// phosphor-react-native/lib/module/thin/FadersHorizontal.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FadersHorizontal = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "faders-horizontal-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M36 80a4 4 0 0 1 4-4h32a4 4 0 0 1 0 8H40a4 4 0 0 1-4-4Zm180 92h-44v-20a4 4 0 0 0-8 0v48a4 4 0 0 0 8 0v-20h44a4 4 0 0 0 0-8Zm-80 0H40a4 4 0 0 0 0 8h96a4 4 0 0 0 0-8Zm-32-64a4 4 0 0 0 4-4V84h108a4 4 0 0 0 0-8H108V56a4 4 0 0 0-8 0v48a4 4 0 0 0 4 4Z"
}));
export default FadersHorizontal;
