// phosphor-react-native/lib/module/thin/FilePdf.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FilePdf = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "file-pdf-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M220 152a4 4 0 0 1-4 4h-28v24h20a4 4 0 0 1 0 8h-20v20a4 4 0 0 1-8 0v-56a4 4 0 0 1 4-4h32a4 4 0 0 1 4 4ZM88 172a24 24 0 0 1-24 24H52v12a4 4 0 0 1-8 0v-56a4 4 0 0 1 4-4h16a24 24 0 0 1 24 24Zm-8 0a16 16 0 0 0-16-16H52v32h12a16 16 0 0 0 16-16Zm80 8a32 32 0 0 1-32 32h-16a4 4 0 0 1-4-4v-56a4 4 0 0 1 4-4h16a32 32 0 0 1 32 32Zm-8 0a24 24 0 0 0-24-24h-12v48h12a24 24 0 0 0 24-24ZM44 112V40a12 12 0 0 1 12-12h96a4 4 0 0 1 2.83 1.17l56 56A4 4 0 0 1 212 88v24a4 4 0 0 1-8 0V92h-52a4 4 0 0 1-4-4V36H56a4 4 0 0 0-4 4v72a4 4 0 0 1-8 0Zm112-28h42.34L156 41.65Z"
}));
export default FilePdf;
