// phosphor-react-native/lib/module/thin/FilmReel.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FilmReel = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "film-reel-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 220h-56.82a100 100 0 1 0-39.18 8h96a4 4 0 0 0 0-8ZM36 128a92 92 0 1 1 92 92 92.1 92.1 0 0 1-92-92Zm92-28a20 20 0 1 0-20-20 20 20 0 0 0 20 20Zm0-32a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm20 108a20 20 0 1 0-20 20 20 20 0 0 0 20-20Zm-32 0a12 12 0 1 1 12 12 12 12 0 0 1-12-12Zm60-28a20 20 0 1 0-20-20 20 20 0 0 0 20 20Zm0-32a12 12 0 1 1-12 12 12 12 0 0 1 12-12Zm-96-8a20 20 0 1 0 20 20 20 20 0 0 0-20-20Zm0 32a12 12 0 1 1 12-12 12 12 0 0 1-12 12Z"
}));
export default FilmReel;
