// phosphor-react-native/lib/module/thin/FlagBanner.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FlagBanner = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "flag-banner-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M227.38 53.86A4 4 0 0 0 224 52H32a4 4 0 0 0-2.7 7l45 41.29-45.24 49A4 4 0 0 0 32 156h140l-31.56 66.28a4 4 0 0 0 1.89 5.33 3.92 3.92 0 0 0 1.67.39 4 4 0 0 0 3.61-2.28l80-168a4 4 0 0 0-.23-3.86ZM175.76 148H41.14l41.8-45.29a4 4 0 0 0-.24-5.66L42.28 60h175.39Z"
}));
export default FlagBanner;
