// phosphor-react-native/lib/module/thin/Gif.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Gif = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "gif-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M140 72v112a4 4 0 0 1-8 0V72a4 4 0 0 1 8 0Zm84-4h-48a4 4 0 0 0-4 4v112a4 4 0 0 0 8 0v-52h36a4 4 0 0 0 0-8h-36V76h44a4 4 0 0 0 0-8ZM96 124H72a4 4 0 0 0 0 8h20v20a28 28 0 0 1-56 0v-48a28 28 0 0 1 28-28c13 0 25 9 28.13 21a4 4 0 1 0 7.74-2C95.85 79.36 80.76 68 64 68a36 36 0 0 0-36 36v48a36 36 0 0 0 72 0v-24a4 4 0 0 0-4-4Z"
}));
export default Gif;
