// phosphor-react-native/lib/module/thin/GitBranch.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GitBranch = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "git-branch-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M220 64a28 28 0 1 0-32 27.71V104a20 20 0 0 1-20 20H96a27.9 27.9 0 0 0-20 8.43V91.71a28 28 0 1 0-8 0v72.58a28 28 0 1 0 8 0V152a20 20 0 0 1 20-20h72a28 28 0 0 0 28-28V91.71A28 28 0 0 0 220 64ZM52 64a20 20 0 1 1 20 20 20 20 0 0 1-20-20Zm40 128a20 20 0 1 1-20-20 20 20 0 0 1 20 20ZM192 84a20 20 0 1 1 20-20 20 20 0 0 1-20 20Z"
}));
export default GitBranch;
