// phosphor-react-native/lib/module/thin/GooglePlayLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GooglePlayLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "google-play-logo-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M221.89 117.69 54.05 21.62a12 12 0 0 0-12.13 0A11.69 11.69 0 0 0 36 31.87v192.26a11.69 11.69 0 0 0 5.92 10.21 12 12 0 0 0 12.13 0l167.77-96a11.76 11.76 0 0 0 .07-20.66Zm-52.44-20.8L144 122.34 50.4 28.75ZM44 222.33V33.67L138.34 128Zm6.4 4.92 93.6-93.59 25.45 25.45Zm167.51-95.88L176.65 155l-27-27 27-27L218 124.66a3.77 3.77 0 0 1-.07 6.71Z"
}));
export default GooglePlayLogo;
