// phosphor-react-native/lib/module/thin/Gradient.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Gradient = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "gradient-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M28 104a4 4 0 0 1 4-4h80a4 4 0 0 1 0 8H32a4 4 0 0 1-4-4Zm196-4h-80a4 4 0 0 0 0 8h80a4 4 0 0 0 0-8ZM72 140H32a4 4 0 0 0 0 8h40a4 4 0 0 0 0-8Zm152 0h-40a4 4 0 0 0 0 8h40a4 4 0 0 0 0-8Zm-124 4a4 4 0 0 0 4 4h48a4 4 0 0 0 0-8h-48a4 4 0 0 0-4 4Zm-44 36H32a4 4 0 0 0 0 8h24a4 4 0 0 0 0-8Zm56 0H88a4 4 0 0 0 0 8h24a4 4 0 0 0 0-8Zm56 0h-24a4 4 0 0 0 0 8h24a4 4 0 0 0 0-8Zm56 0h-24a4 4 0 0 0 0 8h24a4 4 0 0 0 0-8ZM32 68h192a4 4 0 0 0 0-8H32a4 4 0 0 0 0 8Z"
}));
export default Gradient;
