// phosphor-react-native/lib/module/thin/HouseLine.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const HouseLine = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "house-line-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M240 212h-20v-96.46a12 12 0 0 0-3.87-8.82l-80.06-75.59a12 12 0 0 0-16.2.05l-79.94 75.49a12 12 0 0 0-3.93 8.87V212H16a4 4 0 0 0 0 8h224a4 4 0 0 0 0-8ZM44 115.54a4.09 4.09 0 0 1 1.36-3l79.94-75.49a4 4 0 0 1 5.33 0l80.06 75.58a4 4 0 0 1 1.31 3V212h-56v-52a12 12 0 0 0-12-12h-32a12 12 0 0 0-12 12v52H44ZM148 212h-40v-52a4 4 0 0 1 4-4h32a4 4 0 0 1 4 4Z"
}));
export default HouseLine;
