// phosphor-react-native/lib/module/thin/HouseSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const HouseSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "house-simple-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m216.13 106.72-80-75.54-.05-.05a12 12 0 0 0-16.2.05l-79.95 75.49a12 12 0 0 0-3.93 8.87V208a12 12 0 0 0 12 12h160a12 12 0 0 0 12-12v-92.46a12 12 0 0 0-3.87-8.82ZM212 208a4 4 0 0 1-4 4H48a4 4 0 0 1-4-4v-92.46a4.09 4.09 0 0 1 1.36-3l79.94-75.49a4 4 0 0 1 5.36 0l80 75.55a4 4 0 0 1 1.31 3Z"
}));
export default HouseSimple;
