// phosphor-react-native/lib/module/thin/IceCream.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const IceCream = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "ice-cream-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M204 92.4V88a76 76 0 0 0-152 0v4.4a20 20 0 0 0 4 39.6h5.68l62.85 110a4 4 0 0 0 6.94 0l62.85-110H200a20 20 0 0 0 4-39.6Zm-76 139.54L70.89 132h28.5l42.86 75Zm9.11-99.94 24 42-14.25 24.94L108.61 132Zm28.6 33.94L146.32 132h38.79ZM200 124H56a12 12 0 0 1 0-24 4 4 0 0 0 4-4v-8a68 68 0 0 1 136 0v8a4 4 0 0 0 4 4 12 12 0 0 1 0 24Z"
}));
export default IceCream;
