// phosphor-react-native/lib/module/thin/Key.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Key = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "key-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M160 20a76 76 0 0 0-71.49 101.84l-59.34 59.33A4 4 0 0 0 28 184v40a4 4 0 0 0 4 4h40a4 4 0 0 0 4-4v-20h20a4 4 0 0 0 4-4v-20h20a4 4 0 0 0 2.83-1.17l11.33-11.34A76 76 0 1 0 160 20Zm0 144a67.52 67.52 0 0 1-25.21-4.83 4 4 0 0 0-4.45.83l-12 12H96a4 4 0 0 0-4 4v20H72a4 4 0 0 0-4 4v20H36v-34.34l60-60a4 4 0 0 0 .83-4.45A68 68 0 1 1 160 164Zm28-88a8 8 0 1 1-8-8 8 8 0 0 1 8 8Z"
}));
export default Key;
