// phosphor-react-native/lib/module/thin/ListDashes.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ListDashes = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "list-dashes-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M92 64a4 4 0 0 1 4-4h120a4 4 0 0 1 0 8H96a4 4 0 0 1-4-4Zm124 60H96a4 4 0 0 0 0 8h120a4 4 0 0 0 0-8Zm0 64H96a4 4 0 0 0 0 8h120a4 4 0 0 0 0-8ZM56 60H40a4 4 0 0 0 0 8h16a4 4 0 0 0 0-8Zm0 64H40a4 4 0 0 0 0 8h16a4 4 0 0 0 0-8Zm0 64H40a4 4 0 0 0 0 8h16a4 4 0 0 0 0-8Z"
}));
export default ListDashes;
