// phosphor-react-native/lib/module/thin/ListBullets.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ListBullets = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "list-bullets-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M84 64a4 4 0 0 1 4-4h128a4 4 0 0 1 0 8H88a4 4 0 0 1-4-4Zm132 60H88a4 4 0 0 0 0 8h128a4 4 0 0 0 0-8Zm0 64H88a4 4 0 0 0 0 8h128a4 4 0 0 0 0-8ZM44 120a8 8 0 1 0 8 8 8 8 0 0 0-8-8Zm0-64a8 8 0 1 0 8 8 8 8 0 0 0-8-8Zm0 128a8 8 0 1 0 8 8 8 8 0 0 0-8-8Z"
}));
export default ListBullets;
