// phosphor-react-native/lib/module/thin/Microphone.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Microphone = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "microphone-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 172a44.05 44.05 0 0 0 44-44V64a44 44 0 0 0-88 0v64a44.05 44.05 0 0 0 44 44ZM92 64a36 36 0 0 1 72 0v64a36 36 0 0 1-72 0Zm40 139.89V232a4 4 0 0 1-8 0v-28.11A76.09 76.09 0 0 1 52 128a4 4 0 0 1 8 0 68 68 0 0 0 136 0 4 4 0 0 1 8 0 76.09 76.09 0 0 1-72 75.89Z"
}));
export default Microphone;
