// phosphor-react-native/lib/module/thin/MusicNotesPlus.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const MusicNotesPlus = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "music-notes-plus-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M228 56a4 4 0 0 1-4 4h-20v20a4 4 0 0 1-8 0V60h-20a4 4 0 0 1 0-8h20V32a4 4 0 0 1 8 0v20h20a4 4 0 0 1 4 4ZM84 115.12V204a32.06 32.06 0 1 1-8-21.13V64a4 4 0 0 1 3-3.88l56-14a4 4 0 0 1 2 7.76L84 67.12v39.76l75-18.76a4 4 0 0 1 2 7.76ZM76 204a24 24 0 1 0-24 24 24 24 0 0 0 24-24Zm136-84v52a32.06 32.06 0 1 1-8-21.13V120a4 4 0 0 1 8 0Zm-8 52a24 24 0 1 0-24 24 24 24 0 0 0 24-24Z"
}));
export default MusicNotesPlus;
