// phosphor-react-native/lib/module/thin/Notebook.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Notebook = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "notebook-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M180 112a4 4 0 0 1-4 4h-64a4 4 0 0 1 0-8h64a4 4 0 0 1 4 4Zm-4 28h-64a4 4 0 0 0 0 8h64a4 4 0 0 0 0-8Zm44-92v160a12 12 0 0 1-12 12H48a12 12 0 0 1-12-12V48a12 12 0 0 1 12-12h160a12 12 0 0 1 12 12ZM48 212h28V44H48a4 4 0 0 0-4 4v160a4 4 0 0 0 4 4ZM212 48a4 4 0 0 0-4-4H84v168h124a4 4 0 0 0 4-4Z"
}));
export default Notebook;
