// phosphor-react-native/lib/module/thin/NumberCircleEight.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const NumberCircleEight = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "number-circle-eight-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 28a100 100 0 1 0 100 100A100.11 100.11 0 0 0 128 28Zm0 192a92 92 0 1 1 92-92 92.1 92.1 0 0 1-92 92Zm14.9-96.31a28 28 0 1 0-29.8 0 32 32 0 1 0 29.8 0ZM108 100a20 20 0 1 1 20 20 20 20 0 0 1-20-20Zm20 76a24 24 0 1 1 24-24 24 24 0 0 1-24 24Z"
}));
export default NumberCircleEight;
