// phosphor-react-native/lib/module/thin/PaintBrushHousehold.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PaintBrushHousehold = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "paint-brush-household-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M227.8 28.2a28 28 0 0 0-39.6 0l-.15.16-54.9 61.17a4 4 0 0 1-5.81.16l-9.2-9.2a20 20 0 0 0-28.28 0l-76.69 76.68a4 4 0 0 0 0 5.66l80 80a4 4 0 0 0 5.66 0l76.68-76.69a20 20 0 0 0 0-28.28l-9.2-9.2a4 4 0 0 1-1.17-2.94 4 4 0 0 1 1.33-2.87L227.64 68l.16-.15a28 28 0 0 0 0-39.65ZM96 234.34 73.66 212l25.17-25.17a4 4 0 0 0-5.66-5.66L68 206.34 49.66 188l25.17-25.17a4 4 0 0 0-5.66-5.66L44 182.34 21.66 160 72 109.66 146.34 184ZM222.21 62.07l-61.09 54.83a12 12 0 0 0-.46 17.41l9.2 9.2a12 12 0 0 1 0 17L152 178.34 77.66 104l17.85-17.86a12 12 0 0 1 17 0l9.2 9.2a12 12 0 0 0 17.41-.46l54.83-61.09a20 20 0 0 1 28.28 28.28Z"
}));
export default PaintBrushHousehold;
