// phosphor-react-native/lib/module/thin/PaperPlane.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PaperPlane = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "paper-plane-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M234.43 202.08 138.35 34.14a12 12 0 0 0-20.92 0l-95.88 168A12 12 0 0 0 32 220a12 12 0 0 0 4-.7l90.48-31.05h.05a4.09 4.09 0 0 1 2.74 0l90.66 31a12 12 0 0 0 14.49-17.2Zm-7.43 8.48a3.93 3.93 0 0 1-4.45 1.17l-90.59-31V120a4 4 0 0 0-8 0v60.65h-.13l-90.5 31.06a4 4 0 0 1-4.85-5.7l95.87-168a4 4 0 0 1 7 0l96.08 168a3.89 3.89 0 0 1-.43 4.55Z"
}));
export default PaperPlane;
