// phosphor-react-native/lib/module/thin/Password.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Password = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "password-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M44 56v144a4 4 0 0 1-8 0V56a4 4 0 0 1 8 0Zm89.2 58.31-25.2 8.18V96a4 4 0 0 0-8 0v26.49l-25.2-8.18a4 4 0 1 0-2.47 7.61l25.2 8.18L82 151.54a4 4 0 1 0 6.47 4.7L104 134.81l15.57 21.43a4 4 0 0 0 3.24 1.65 4 4 0 0 0 3.23-6.35l-15.57-21.44 25.2-8.18a4 4 0 0 0-2.47-7.61Zm101 2.57a4 4 0 0 0-5-2.57l-25.2 8.18V96a4 4 0 0 0-8 0v26.49l-25.2-8.18a4 4 0 0 0-2.47 7.61l25.2 8.18L178 151.54a4 4 0 1 0 6.47 4.7L200 134.81l15.57 21.43a4 4 0 0 0 3.24 1.65 4 4 0 0 0 3.23-6.35l-15.57-21.44 25.2-8.18a4 4 0 0 0 2.57-5.04Z"
}));
export default Password;
