// phosphor-react-native/lib/module/thin/PatreonLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PatreonLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "patreon-logo-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M176 44a60 60 0 1 0 60 60 60.07 60.07 0 0 0-60-60Zm0 112a52 52 0 1 1 52-52 52.06 52.06 0 0 1-52 52ZM80 44H64a12 12 0 0 0-12 12v152a12 12 0 0 0 12 12h16a12 12 0 0 0 12-12V56a12 12 0 0 0-12-12Zm4 164a4 4 0 0 1-4 4H64a4 4 0 0 1-4-4V56a4 4 0 0 1 4-4h16a4 4 0 0 1 4 4Z"
}));
export default PatreonLogo;
