// phosphor-react-native/lib/module/thin/PictureInPicture.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PictureInPicture = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "picture-in-picture-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 52H40a12 12 0 0 0-12 12v128a12 12 0 0 0 12 12h176a12 12 0 0 0 12-12V64a12 12 0 0 0-12-12ZM36 192V64a4 4 0 0 1 4-4h176a4 4 0 0 1 4 4v60h-76a12 12 0 0 0-12 12v60H40a4 4 0 0 1-4-4Zm180 4h-76v-60a4 4 0 0 1 4-4h76v60a4 4 0 0 1-4 4Z"
}));
export default PictureInPicture;
