// phosphor-react-native/lib/module/thin/Rss.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Rss = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "rss-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M96.08 159.92A67.52 67.52 0 0 1 116 208a4 4 0 0 1-8 0 60 60 0 0 0-60-60 4 4 0 0 1 0-8 67.52 67.52 0 0 1 48.08 19.92ZM48 92a4 4 0 0 0 0 8 108 108 0 0 1 108 108 4 4 0 0 0 8 0A116 116 0 0 0 48 92Zm116 0A162.92 162.92 0 0 0 48 44a4 4 0 0 0 0 8 155 155 0 0 1 110.31 45.69A155 155 0 0 1 204 208a4 4 0 0 0 8 0 162.92 162.92 0 0 0-48-116ZM52 196a8 8 0 1 0 8 8 8 8 0 0 0-8-8Z"
}));
export default Rss;
