// phosphor-react-native/lib/module/thin/Scooter.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Scooter = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "scooter-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M212 140a31.29 31.29 0 0 0-6.24.62l-11.82-35.46-22.15-66.42A4 4 0 0 0 168 36h-32a4 4 0 0 0 0 8h29.12l20.54 61.63L134 172H76a32 32 0 1 0-1 8h61a4 4 0 0 0 3.16-1.54l49.54-63.7 9.47 28.39A32 32 0 1 0 212 140ZM44 196a24 24 0 1 1 24-24 24 24 0 0 1-24 24Zm168 0a24 24 0 1 1 24-24 24 24 0 0 1-24 24Z"
}));
export default Scooter;
