// phosphor-react-native/lib/module/thin/ShareNetwork.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ShareNetwork = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "share-network-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M176 164a36 36 0 0 0-27.92 13.3L96.25 144a35.92 35.92 0 0 0 0-32l51.83-33.3a35.93 35.93 0 1 0-4.33-6.7l-51.83 33.3a36 36 0 1 0 0 45.4l51.83 33.3A36 36 0 1 0 176 164Zm0-136a28 28 0 1 1-28 28 28 28 0 0 1 28-28ZM64 156a28 28 0 1 1 28-28 28 28 0 0 1-28 28Zm112 72a28 28 0 1 1 28-28 28 28 0 0 1-28 28Z"
}));
export default ShareNetwork;
