// phosphor-react-native/lib/module/thin/ShieldPlus.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ShieldPlus = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "shield-plus-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 44H48a12 12 0 0 0-12 12v58.77c0 86.87 73.54 115.7 88.28 120.59a11.47 11.47 0 0 0 7.44 0c14.74-4.87 88.28-33.7 88.28-120.57V56a12 12 0 0 0-12-12Zm4 70.79c0 81.38-69 108.41-82.81 113a3.51 3.51 0 0 1-2.39 0C113 223.2 44 196.17 44 114.79V56a4 4 0 0 1 4-4h160a4 4 0 0 1 4 4ZM164 128a4 4 0 0 1-4 4h-28v28a4 4 0 0 1-8 0v-28H96a4 4 0 0 1 0-8h28V96a4 4 0 0 1 8 0v28h28a4 4 0 0 1 4 4Z"
}));
export default ShieldPlus;
