// phosphor-react-native/lib/module/thin/ShoppingBagOpen.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ShoppingBagOpen = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "shopping-bag-open-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 44H40a12 12 0 0 0-12 12v144a12 12 0 0 0 12 12h176a12 12 0 0 0 12-12V56a12 12 0 0 0-12-12ZM40 52h176a4 4 0 0 1 4 4v20H36V56a4 4 0 0 1 4-4Zm176 152H40a4 4 0 0 1-4-4V84h184v116a4 4 0 0 1-4 4Zm-44-92a44 44 0 0 1-88 0 4 4 0 0 1 8 0 36 36 0 0 0 72 0 4 4 0 0 1 8 0Z"
}));
export default ShoppingBagOpen;
