// phosphor-react-native/lib/module/thin/SignOut.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SignOut = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "sign-out-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M108 216a4 4 0 0 1-4 4H48a12 12 0 0 1-12-12V48a12 12 0 0 1 12-12h56a4 4 0 0 1 0 8H48a4 4 0 0 0-4 4v160a4 4 0 0 0 4 4h56a4 4 0 0 1 4 4Zm110.83-90.83-40-40a4 4 0 0 0-5.66 5.66L206.34 124H104a4 4 0 0 0 0 8h102.34l-33.17 33.17a4 4 0 0 0 5.66 5.66l40-40a4 4 0 0 0 0-5.66Z"
}));
export default SignOut;
