// phosphor-react-native/lib/module/thin/SpotifyLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SpotifyLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "spotify-logo-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 28a100 100 0 1 0 100 100A100.11 100.11 0 0 0 128 28Zm0 192a92 92 0 1 1 92-92 92.1 92.1 0 0 1-92 92Zm27.54-52.13a4 4 0 0 1-5.41 1.67 46.73 46.73 0 0 0-44.26 0 4.07 4.07 0 0 1-1.87.46 4 4 0 0 1-1.87-7.54 55.39 55.39 0 0 1 51.74 0 4 4 0 0 1 1.67 5.41Zm32-56a4 4 0 0 1-5.41 1.65 114.67 114.67 0 0 0-108.24 0 4 4 0 0 1-3.76-7.06 122.65 122.65 0 0 1 115.76 0 4 4 0 0 1 1.64 5.41Zm-16 28a4 4 0 0 1-5.41 1.65 81 81 0 0 0-76.24 0 4 4 0 0 1-3.76-7.06 89 89 0 0 1 83.76 0 4 4 0 0 1 1.64 5.42Z"
}));
export default SpotifyLogo;
