// phosphor-react-native/lib/module/thin/Stop.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Stop = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "stop-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M200.73 44H55.27A11.29 11.29 0 0 0 44 55.27v145.46A11.29 11.29 0 0 0 55.27 212h145.46A11.29 11.29 0 0 0 212 200.73V55.27A11.29 11.29 0 0 0 200.73 44ZM204 200.73a3.28 3.28 0 0 1-3.27 3.27H55.27a3.28 3.28 0 0 1-3.27-3.27V55.27A3.28 3.28 0 0 1 55.27 52h145.46a3.28 3.28 0 0 1 3.27 3.27Z"
}));
export default Stop;
