// phosphor-react-native/lib/module/thin/Sunglasses.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Sunglasses = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "sunglasses-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M200 44a4 4 0 0 0 0 8 20 20 0 0 1 20 20v60H36V72a20 20 0 0 1 20-20 4 4 0 0 0 0-8 28 28 0 0 0-28 28v92a40 40 0 0 0 80 0v-24h40v24a40 40 0 0 0 80 0V72a28 28 0 0 0-28-28ZM36 164v-24h2.34l49.27 49.26A32 32 0 0 1 36 164Zm64 0a31.83 31.83 0 0 1-6.74 19.61L49.66 140H100Zm56 0v-24h2.34l49.27 49.26A32 32 0 0 1 156 164Zm57.26 19.61L169.66 140H220v24a31.83 31.83 0 0 1-6.74 19.61Z"
}));
export default Sunglasses;
