// phosphor-react-native/lib/module/thin/TagChevron.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TagChevron = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "tag-chevron-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m234 121.34-42.67-64a12 12 0 0 0-10-5.34H24a4 4 0 0 0-3.29 6.27l46.61 67.51a4 4 0 0 1 0 4.39l-46.61 67.56A4 4 0 0 0 24 204h157.33a12 12 0 0 0 10-5.34l42.67-64a12 12 0 0 0 0-13.32Zm-6.66 8.88-42.66 64a4 4 0 0 1-3.33 1.78H31.62L74 134.66a12 12 0 0 0 0-13.37L31.62 60h149.71a4 4 0 0 1 3.33 1.78l42.66 64a4 4 0 0 1 0 4.44Z"
}));
export default TagChevron;
