// phosphor-react-native/lib/module/thin/TextColumns.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TextColumns = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "text-columns-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M116 64a4 4 0 0 1-4 4H40a4 4 0 0 1 0-8h72a4 4 0 0 1 4 4Zm-4 36H40a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Zm0 40H40a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Zm0 40H40a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Zm32-112h72a4 4 0 0 0 0-8h-72a4 4 0 0 0 0 8Zm72 32h-72a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Zm0 40h-72a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Zm0 40h-72a4 4 0 0 0 0 8h72a4 4 0 0 0 0-8Z"
}));
export default TextColumns;
