// phosphor-react-native/lib/module/thin/TextHFour.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TextHFour = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "text-h-four-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M252 176a4 4 0 0 1-4 4h-12v28a4 4 0 0 1-8 0v-28h-44a4 4 0 0 1-3.79-5.26l24-72a4 4 0 1 1 7.58 2.52L189.55 172H228v-28a4 4 0 0 1 8 0v28h12a4 4 0 0 1 4 4ZM144 52a4 4 0 0 0-4 4v56H44V56a4 4 0 0 0-8 0v120a4 4 0 0 0 8 0v-56h96v56a4 4 0 0 0 8 0V56a4 4 0 0 0-4-4Z"
}));
export default TextHFour;
