// phosphor-react-native/lib/module/thin/ToteSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ToteSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "tote-simple-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M233 72.06a12.11 12.11 0 0 0-9.08-4.06H172v-4a44 44 0 0 0-88 0v4H32.08A12.11 12.11 0 0 0 23 72.06a12 12 0 0 0-2.94 9.35l14.26 120a12 12 0 0 0 12 10.59h163.35a12 12 0 0 0 12-10.59l14.26-120a12 12 0 0 0-2.93-9.35ZM92 64a36 36 0 0 1 72 0v4H92Zm136 16.46-14.25 120a4 4 0 0 1-4.05 3.54H46.33a4 4 0 0 1-4-3.54L28 80.46a3.9 3.9 0 0 1 1-3.09A4.08 4.08 0 0 1 32.08 76h191.84a4.08 4.08 0 0 1 3.08 1.37 3.9 3.9 0 0 1 1 3.09Z"
}));
export default ToteSimple;
