// phosphor-react-native/lib/module/thin/Trophy.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Trophy = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "trophy-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M232 68h-28V56a12 12 0 0 0-12-12H64a12 12 0 0 0-12 12v12H24a12 12 0 0 0-12 12v16a36 36 0 0 0 36 36h6.66A76 76 0 0 0 124 187.89V220H96a4 4 0 0 0 0 8h64a4 4 0 0 0 0-8h-28v-32.12c32.93-1.74 60.41-24.91 69.11-55.88H208a36 36 0 0 0 36-36V80a12 12 0 0 0-12-12ZM48 124a28 28 0 0 1-28-28V80a4 4 0 0 1 4-4h28v36a77 77 0 0 0 1 12Zm148-12.9c0 37.71-30.27 68.62-67.48 68.9H128a68 68 0 0 1-68-68V56a4 4 0 0 1 4-4h128a4 4 0 0 1 4 4ZM236 96a28 28 0 0 1-28 28h-5.1a77.35 77.35 0 0 0 1.1-12.9V76h28a4 4 0 0 1 4 4Z"
}));
export default Trophy;
