// phosphor-react-native/lib/module/thin/VinylRecord.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const VinylRecord = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "vinyl-record-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 28a100 100 0 1 0 100 100A100.11 100.11 0 0 0 128 28Zm0 192a92 92 0 1 1 92-92 92.1 92.1 0 0 1-92 92Zm0-152a60.07 60.07 0 0 0-60 60 4 4 0 0 1-8 0 68.07 68.07 0 0 1 68-68 4 4 0 0 1 0 8Zm68 60a68.07 68.07 0 0 1-68 68 4 4 0 0 1 0-8 60.07 60.07 0 0 0 60-60 4 4 0 0 1 8 0Zm-40 0a28 28 0 1 0-28 28 28 28 0 0 0 28-28Zm-48 0a20 20 0 1 1 20 20 20 20 0 0 1-20-20Z"
}));
export default VinylRecord;
