// phosphor-react-native/lib/module/thin/WindowsLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const WindowsLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "windows-logo-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 140h-80a4 4 0 0 0-4 4v57.45a4 4 0 0 0 3.28 3.94l80 14.55a4.37 4.37 0 0 0 .72.06 4 4 0 0 0 2.56-.93A4 4 0 0 0 220 216v-72a4 4 0 0 0-4-4Zm-4 71.21-72-13.09V148h72ZM104 140H40a4 4 0 0 0-4 4v40a4 4 0 0 0 3.28 3.94l64 11.63a3.51 3.51 0 0 0 .72.07 4 4 0 0 0 4-4V144a4 4 0 0 0-4-4Zm-4 50.84-56-10.18V148h56ZM218.56 36.93a4 4 0 0 0-3.28-.87l-80 14.55a4 4 0 0 0-3.28 3.94V112a4 4 0 0 0 4 4h80a4 4 0 0 0 4-4V40a4 4 0 0 0-1.44-3.07ZM212 108h-72V57.88l72-13.09ZM103.28 56.43l-64 11.63A4 4 0 0 0 36 72v40a4 4 0 0 0 4 4h64a4 4 0 0 0 4-4V60.36a4 4 0 0 0-4.72-3.93ZM100 108H44V75.34l56-10.18Z"
}));
export default WindowsLogo;
