// phosphor-react-native/lib/module/thin/YinYang.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const YinYang = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "yin-yang-thin__svg-icon-phosphor",
  fill: "currentColor",
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 28a100 100 0 1 0 100 100A100.11 100.11 0 0 0 128 28ZM36 128a92.1 92.1 0 0 1 92-92 44 44 0 0 1 0 88 52 52 0 0 0-37.44 88A92.14 92.14 0 0 1 36 128Zm92 92a44 44 0 0 1 0-88 52 52 0 0 0 37.44-88A92 92 0 0 1 128 220Zm8-44a8 8 0 1 1-8-8 8 8 0 0 1 8 8Zm-16-96a8 8 0 1 1 8 8 8 8 0 0 1-8-8Z"
}));
export default YinYang;
