// phosphor-react-native/lib/module/duotone/AirplaneLanding.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const AirplaneLanding = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "airplane-landing-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 148.32V184L47.37 134.54A32 32 0 0 1 24 103.73V48a8 8 0 0 1 10.53-7.59L40 42.24l12 33.22L96 88V48a8 8 0 0 1 10.53-7.59l5.47 1.83 24 57.2 64.56 18A32 32 0 0 1 224 148.32Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M248 216a8 8 0 0 1-8 8H96a8 8 0 0 1 0-16h144a8 8 0 0 1 8 8Zm-26.16-24.3L45.21 142.24A40.12 40.12 0 0 1 16 103.72V48a16 16 0 0 1 21.06-15.18l5.47 1.82a8 8 0 0 1 5 4.87l10.6 29.37L88 77.39V48a16 16 0 0 1 21.06-15.18l5.47 1.82a8 8 0 0 1 4.85 4.5l22.5 53.63 60.84 17A40.13 40.13 0 0 1 232 148.32V184a8 8 0 0 1-10.16 7.7ZM216 148.32a24.09 24.09 0 0 0-17.58-23.13l-64.57-18a8 8 0 0 1-5.23-4.61L106 48.67l-2-.67v40a8 8 0 0 1-10.19 7.7l-44-12.54a8 8 0 0 1-5.33-5L33.79 48.59 32 48v55.72a24.09 24.09 0 0 0 17.53 23.12L216 173.45Z"
}));
export default AirplaneLanding;
