// phosphor-react-native/lib/module/duotone/Alarm.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Alarm = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "alarm-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 128a88 88 0 1 1-88-88 88 88 0 0 1 88 88Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M128 32a96 96 0 1 0 96 96 96.11 96.11 0 0 0-96-96Zm0 176a80 80 0 1 1 80-80 80.09 80.09 0 0 1-80 80ZM61.66 29.66l-32 32a8 8 0 0 1-11.32-11.32l32-32a8 8 0 1 1 11.32 11.32Zm176 32a8 8 0 0 1-11.32 0l-32-32a8 8 0 0 1 11.32-11.32l32 32a8 8 0 0 1 0 11.32ZM184 120a8 8 0 0 1 0 16h-56a8 8 0 0 1-8-8V72a8 8 0 0 1 16 0v48Z"
}));
export default Alarm;
