// phosphor-react-native/lib/module/duotone/ArrowElbowRightUp.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowElbowRightUp = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-elbow-right-up-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 96h-96l48-48Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m229.66 90.34-48-48a8 8 0 0 0-11.32 0l-48 48A8 8 0 0 0 128 104h40v80H32a8 8 0 0 0 0 16h144a8 8 0 0 0 8-8v-88h40a8 8 0 0 0 5.66-13.66ZM147.31 88 176 59.31 204.69 88Z"
}));
export default ArrowElbowRightUp;
