// phosphor-react-native/lib/module/duotone/ArrowDownRight.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowDownRight = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-down-right-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M192 88v104H88Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M195.06 80.61a8 8 0 0 0-8.72 1.73L140 128.69 69.66 58.34a8 8 0 0 0-11.32 11.32L128.69 140l-46.35 46.34A8 8 0 0 0 88 200h104a8 8 0 0 0 8-8V88a8 8 0 0 0-4.94-7.39ZM184 184h-76.69l38.34-38.34L184 107.31Z"
}));
export default ArrowDownRight;
