// phosphor-react-native/lib/module/duotone/ArrowBendLeftUp.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowBendLeftUp = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-bend-left-up-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M152 80H56l48-48Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M200 216a88.1 88.1 0 0 1-88-88V88h40a8 8 0 0 0 5.66-13.66l-48-48a8 8 0 0 0-11.32 0l-48 48A8 8 0 0 0 56 88h40v40a104.11 104.11 0 0 0 104 104 8 8 0 0 0 0-16ZM104 43.31 132.69 72H75.31Z"
}));
export default ArrowBendLeftUp;
