// phosphor-react-native/lib/module/duotone/ArrowBendDownRight.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowBendDownRight = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-bend-down-right-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m224 152-48 48v-96Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m229.66 146.34-48-48A8 8 0 0 0 168 104v40h-40a88.1 88.1 0 0 1-88-88 8 8 0 0 0-16 0 104.11 104.11 0 0 0 104 104h40v40a8 8 0 0 0 13.66 5.66l48-48a8 8 0 0 0 0-11.32ZM184 180.69v-57.38L212.69 152Z"
}));
export default ArrowBendDownRight;
