// phosphor-react-native/lib/module/duotone/ArrowsOutLineHorizontal.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowsOutLineHorizontal = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrows-out-line-horizontal-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m16 128 32-32v64Zm192-32v64l32-32Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M136 40v176a8 8 0 0 1-16 0V40a8 8 0 0 1 16 0Zm-32 88a8 8 0 0 1-8 8H56v24a8 8 0 0 1-13.66 5.66l-32-32a8 8 0 0 1 0-11.32l32-32A8 8 0 0 1 56 96v24h40a8 8 0 0 1 8 8Zm-64-12.69L27.31 128 40 140.69Zm205.66 18.35-32 32A8 8 0 0 1 200 160v-24h-40a8 8 0 0 1 0-16h40V96a8 8 0 0 1 13.66-5.66l32 32a8 8 0 0 1 0 11.32Zm-17-5.66L216 115.31v25.38Z"
}));
export default ArrowsOutLineHorizontal;
