// phosphor-react-native/lib/module/duotone/ArrowElbowUpLeft.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowElbowUpLeft = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-elbow-up-left-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M96 32v96L48 80Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M192 72h-88V32a8 8 0 0 0-13.66-5.66l-48 48a8 8 0 0 0 0 11.32l48 48A8 8 0 0 0 104 128V88h80v136a8 8 0 0 0 16 0V80a8 8 0 0 0-8-8ZM88 108.69 59.31 80 88 51.31Z"
}));
export default ArrowElbowUpLeft;
