// phosphor-react-native/lib/module/duotone/ArrowUDownLeft.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowUDownLeft = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-u-down-left-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M80 120v96l-48-48Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M168 48H80a8 8 0 0 0 0 16h88a48 48 0 0 1 0 96H88v-40a8 8 0 0 0-13.66-5.66l-48 48a8 8 0 0 0 0 11.32l48 48A8 8 0 0 0 88 216v-40h80a64 64 0 0 0 0-128ZM72 196.69 43.31 168 72 139.31Z"
}));
export default ArrowUDownLeft;
