// phosphor-react-native/lib/module/duotone/ArrowArcLeft.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowArcLeft = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-arc-left-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M88 152H24V88Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M201.54 110.46a104 104 0 0 0-145.4-1.63L29.66 82.34A8 8 0 0 0 16 88v64a8 8 0 0 0 8 8h64a8 8 0 0 0 5.66-13.66l-26.19-26.18A88 88 0 0 1 216 184a8 8 0 0 0 16 0 103.33 103.33 0 0 0-30.46-73.54ZM32 144v-36.69L68.69 144Z"
}));
export default ArrowArcLeft;
