// phosphor-react-native/lib/module/duotone/ArrowCounterClockwise.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowCounterClockwise = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-counter-clockwise-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M72 104H24V56Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M195.88 60.08A96.08 96.08 0 0 0 60.25 60L49.31 70 29.66 50.3A8 8 0 0 0 16 56v48a8 8 0 0 0 8 8h48a8 8 0 0 0 5.66-13.66l-17-17 10.54-9.65a3.07 3.07 0 0 0 .26-.25 80 80 0 1 1 1.65 114.78 8 8 0 0 0-11 11.63A95.38 95.38 0 0 0 128 224h1.32a96 96 0 0 0 66.56-163.92ZM32 96V75.28L52.69 96Z"
}));
export default ArrowCounterClockwise;
