// phosphor-react-native/lib/module/duotone/ArrowsHorizontal.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowsHorizontal = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrows-horizontal-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m24 128 32-32v64Zm176-32v64l32-32Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m237.66 122.34-32-32A8 8 0 0 0 192 96v24H64V96a8 8 0 0 0-13.66-5.66l-32 32a8 8 0 0 0 0 11.32l32 32A8 8 0 0 0 64 160v-24h128v24a8 8 0 0 0 13.66 5.66l32-32a8 8 0 0 0 0-11.32ZM48 140.69 35.31 128 48 115.31Zm160 0v-25.38L220.69 128Z"
}));
export default ArrowsHorizontal;
