// phosphor-react-native/lib/module/duotone/ArrowsOutLineVertical.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowsOutLineVertical = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrows-out-line-vertical-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M160 48H96l32-32Zm-32 192 32-32H96Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M224 128a8 8 0 0 1-8 8H40a8 8 0 0 1 0-16h176a8 8 0 0 1 8 8ZM88.61 51.06a8 8 0 0 1 1.73-8.72l32-32a8 8 0 0 1 11.32 0l32 32A8 8 0 0 1 160 56h-24v40a8 8 0 0 1-16 0V56H96a8 8 0 0 1-7.39-4.94ZM115.31 40h25.38L128 27.31Zm52.08 164.94a8 8 0 0 1-1.73 8.72l-32 32a8 8 0 0 1-11.32 0l-32-32A8 8 0 0 1 96 200h24v-40a8 8 0 0 1 16 0v40h24a8 8 0 0 1 7.39 4.94ZM140.69 216h-25.38L128 228.69Z"
}));
export default ArrowsOutLineVertical;
