// phosphor-react-native/lib/module/duotone/ArrowBendRightUp.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowBendRightUp = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-bend-right-up-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M200 80h-96l48-48Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m205.66 74.34-48-48a8 8 0 0 0-11.32 0l-48 48A8 8 0 0 0 104 88h40v40a88.1 88.1 0 0 1-88 88 8 8 0 0 0 0 16 104.11 104.11 0 0 0 104-104V88h40a8 8 0 0 0 5.66-13.66ZM123.31 72 152 43.31 180.69 72Z"
}));
export default ArrowBendRightUp;
