// phosphor-react-native/lib/module/duotone/ArrowsInLineVertical.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowsInLineVertical = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrows-in-line-vertical-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 96 96 64h64Zm-32 96h64l-32-32Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M224 128a8 8 0 0 1-8 8H40a8 8 0 0 1 0-16h176a8 8 0 0 1 8 8ZM90.34 69.66A8 8 0 0 1 96 56h24V16a8 8 0 0 1 16 0v40h24a8 8 0 0 1 5.66 13.66l-32 32a8 8 0 0 1-11.32 0Zm25 2.34L128 84.69 140.69 72Zm50.35 114.34A8 8 0 0 1 160 200h-24v40a8 8 0 0 1-16 0v-40H96a8 8 0 0 1-5.66-13.66l32-32a8 8 0 0 1 11.32 0Zm-25-2.34L128 171.31 115.31 184Z"
}));
export default ArrowsInLineVertical;
