// phosphor-react-native/lib/module/duotone/ArrowElbowLeftUp.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowElbowLeftUp = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-elbow-left-up-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 96H32l48-48Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M224 184H88v-80h40a8 8 0 0 0 5.66-13.66l-48-48a8 8 0 0 0-11.32 0l-48 48A8 8 0 0 0 32 104h40v88a8 8 0 0 0 8 8h144a8 8 0 0 0 0-16ZM80 59.31 108.69 88H51.31Z"
}));
export default ArrowElbowLeftUp;
