// phosphor-react-native/lib/module/duotone/ArrowElbowUpRight.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowElbowUpRight = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-elbow-up-right-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m208 80-48 48V32Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m213.66 74.34-48-48A8 8 0 0 0 152 32v40H64a8 8 0 0 0-8 8v144a8 8 0 0 0 16 0V88h80v40a8 8 0 0 0 13.66 5.66l48-48a8 8 0 0 0 0-11.32ZM168 108.69V51.31L196.69 80Z"
}));
export default ArrowElbowUpRight;
