// phosphor-react-native/lib/module/duotone/ArrowElbowLeftDown.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowElbowLeftDown = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-elbow-left-down-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m128 160-48 48-48-48Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M224 56H80a8 8 0 0 0-8 8v88H32a8 8 0 0 0-5.66 13.66l48 48a8 8 0 0 0 11.32 0l48-48A8 8 0 0 0 128 152H88V72h136a8 8 0 0 0 0-16ZM80 196.69 51.31 168h57.38Z"
}));
export default ArrowElbowLeftDown;
