// phosphor-react-native/lib/module/duotone/ArrowUpLeft.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowUpLeft = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrow-up-left-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M168 64 64 168V64Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M197.66 186.34 127.31 116l46.35-46.34A8 8 0 0 0 168 56H64a8 8 0 0 0-8 8v104a8 8 0 0 0 13.66 5.66L116 127.31l70.34 70.35a8 8 0 0 0 11.32-11.32ZM72 72h76.69l-38.34 38.34L72 148.69Z"
}));
export default ArrowUpLeft;
