// phosphor-react-native/lib/module/duotone/ArrowsInSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowsInSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrows-in-simple-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M192 112h-48V64Zm-80 80v-48H64Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M213.66 42.34a8 8 0 0 0-11.32 0L168 76.69l-18.34-18.35A8 8 0 0 0 136 64v48a8 8 0 0 0 8 8h48a8 8 0 0 0 5.66-13.66L179.31 88l34.35-34.34a8 8 0 0 0 0-11.32ZM152 104V83.31L172.69 104Zm-40 32H64a8 8 0 0 0-5.66 13.66L76.69 168l-34.35 34.34a8 8 0 0 0 11.32 11.32L88 179.31l18.34 18.35A8 8 0 0 0 120 192v-48a8 8 0 0 0-8-8Zm-8 36.69L83.31 152H104Z"
}));
export default ArrowsInSimple;
