// phosphor-react-native/lib/module/duotone/ArrowsOutSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ArrowsOutSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "arrows-out-simple-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 48v48l-48-48ZM48 208h48l-48-48Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M208 40h-48a8 8 0 0 0-5.66 13.66L172.69 72l-34.35 34.34a8 8 0 0 0 11.32 11.32L184 83.31l18.34 18.35A8 8 0 0 0 216 96V48a8 8 0 0 0-8-8Zm-8 36.69L179.31 56H200Zm-93.66 61.65L72 172.69l-18.34-18.35A8 8 0 0 0 40 160v48a8 8 0 0 0 8 8h48a8 8 0 0 0 5.66-13.66L83.31 184l34.35-34.34a8 8 0 0 0-11.32-11.32ZM56 200v-20.69L76.69 200Z"
}));
export default ArrowsOutSimple;
