// phosphor-react-native/lib/module/duotone/Bird.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Bird = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "bird-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m232 80-24 16v24a96 96 0 0 1-96 96H8l96-120V76a52 52 0 0 1 102.33-13.11Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M176 68a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm-50.88 61.85a8 8 0 0 0-11.27 1l-40 48a8 8 0 0 0 12.3 10.24l40-48a8 8 0 0 0-1.03-11.24ZM240 80a8 8 0 0 1-3.56 6.66L216 100.28V120a104.11 104.11 0 0 1-104 104H8a8 8 0 0 1-6.25-13L96 93.19V76a60 60 0 0 1 117.21-18.14l23.23 15.48A8 8 0 0 1 240 80Zm-22.42 0L201.9 69.54a8 8 0 0 1-3.31-4.64A44 44 0 0 0 112 76v20a8 8 0 0 1-1.75 5L24.64 208H112a88.1 88.1 0 0 0 88-88V96a8 8 0 0 1 3.56-6.66Z"
}));
export default Bird;
