// phosphor-react-native/lib/module/duotone/Bluetooth.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Bluetooth = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "bluetooth-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m184 80-64 48V32Zm-64 144 64-48-64-48Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M188.8 169.6 133.33 128l55.47-41.6a8 8 0 0 0 0-12.8l-64-48A8 8 0 0 0 112 32v80L60.8 73.6a8 8 0 0 0-9.6 12.8l55.47 41.6-55.47 41.6a8 8 0 1 0 9.6 12.8L112 144v80a8 8 0 0 0 12.8 6.4l64-48a8 8 0 0 0 0-12.8ZM128 48l42.67 32L128 112Zm0 160v-64l42.67 32Z"
}));
export default Bluetooth;
