// phosphor-react-native/lib/module/duotone/BriefcaseMetal.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const BriefcaseMetal = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "briefcase-metal-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 72v32H32V72a8 8 0 0 1 8-8h176a8 8 0 0 1 8 8ZM32 200a8 8 0 0 0 8 8h176a8 8 0 0 0 8-8v-32H32Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M216 56h-40v-8a24 24 0 0 0-24-24h-48a24 24 0 0 0-24 24v8H40a16 16 0 0 0-16 16v128a16 16 0 0 0 16 16h176a16 16 0 0 0 16-16V72a16 16 0 0 0-16-16ZM40 112h176v48H40Zm56-64a8 8 0 0 1 8-8h48a8 8 0 0 1 8 8v8H96Zm120 24v24H40V72Zm0 128H40v-24h176v24Z"
}));
export default BriefcaseMetal;
