// phosphor-react-native/lib/module/duotone/CaretDoubleUp.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CaretDoubleUp = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "caret-double-up-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 208H48l80-80Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M133.66 122.34a8 8 0 0 0-11.32 0l-80 80A8 8 0 0 0 48 216h160a8 8 0 0 0 5.66-13.66ZM67.31 200 128 139.31 188.69 200Zm-25-66.34a8 8 0 0 1 0-11.32l80-80a8 8 0 0 1 11.32 0l80 80a8 8 0 0 1-11.32 11.32L128 59.31l-74.34 74.35a8 8 0 0 1-11.32 0Z"
}));
export default CaretDoubleUp;
