// phosphor-react-native/lib/module/duotone/CaretDoubleDown.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CaretDoubleDown = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "caret-double-down-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m208 48-80 80-80-80Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m213.66 133.66-80 80a8 8 0 0 1-11.32 0l-80-80a8 8 0 0 1 11.32-11.32L128 196.69l74.34-74.35a8 8 0 0 1 11.32 11.32Zm-171.32-80A8 8 0 0 1 48 40h160a8 8 0 0 1 5.66 13.66l-80 80a8 8 0 0 1-11.32 0Zm25 2.34L128 116.69 188.69 56Z"
}));
export default CaretDoubleDown;
