// phosphor-react-native/lib/module/duotone/CellSignalLow.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const CellSignalLow = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "cell-signal-low-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M80 140.7V208H32a8 8 0 0 1-5.66-13.66Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M198.12 25.23a16 16 0 0 0-17.44 3.46l-160 160A16 16 0 0 0 32 216h160a16 16 0 0 0 16-16V40a15.94 15.94 0 0 0-9.88-14.77ZM72 160v40H32Zm120 40H88v-56L192 40Z"
}));
export default CellSignalLow;
