// phosphor-react-native/lib/module/duotone/ChalkboardTeacher.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChalkboardTeacher = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chalkboard-teacher-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M192 80v96h-88a32 32 0 1 0-32-32h-8V80Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M216 40H40a16 16 0 0 0-16 16v144a16 16 0 0 0 16 16h13.39a8 8 0 0 0 7.23-4.57 48 48 0 0 1 86.76 0 8 8 0 0 0 7.23 4.57H216a16 16 0 0 0 16-16V56a16 16 0 0 0-16-16ZM80 144a24 24 0 1 1 24 24 24 24 0 0 1-24-24Zm136 56h-56.57a64.39 64.39 0 0 0-28.83-26.16 40 40 0 1 0-53.2 0A64.39 64.39 0 0 0 48.57 200H40V56h176ZM56 96V80a8 8 0 0 1 8-8h128a8 8 0 0 1 8 8v96a8 8 0 0 1-8 8h-16a8 8 0 0 1 0-16h8V88H72v8a8 8 0 0 1-16 0Z"
}));
export default ChalkboardTeacher;
