// phosphor-react-native/lib/module/duotone/ChatTeardrop.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ChatTeardrop = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "chat-teardrop-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 124a92 92 0 0 1-92 92H47.67a7.66 7.66 0 0 1-7.67-7.67V124a92 92 0 0 1 92-92 92 92 0 0 1 92 92Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M132 24A100.11 100.11 0 0 0 32 124v84.33A15.69 15.69 0 0 0 47.67 224H132a100 100 0 0 0 0-200Zm0 184H48v-84a84 84 0 1 1 84 84Z"
}));
export default ChatTeardrop;
