// phosphor-react-native/lib/module/duotone/DotsNine.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const DotsNine = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "dots-nine-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 48v160a16 16 0 0 1-16 16H48a16 16 0 0 1-16-16V48a16 16 0 0 1 16-16h160a16 16 0 0 1 16 16Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M72 60a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm56-12a12 12 0 1 0 12 12 12 12 0 0 0-12-12Zm68 24a12 12 0 1 0-12-12 12 12 0 0 0 12 12ZM60 116a12 12 0 1 0 12 12 12 12 0 0 0-12-12Zm68 0a12 12 0 1 0 12 12 12 12 0 0 0-12-12Zm68 0a12 12 0 1 0 12 12 12 12 0 0 0-12-12ZM60 184a12 12 0 1 0 12 12 12 12 0 0 0-12-12Zm68 0a12 12 0 1 0 12 12 12 12 0 0 0-12-12Zm68 0a12 12 0 1 0 12 12 12 12 0 0 0-12-12Z"
}));
export default DotsNine;
