// phosphor-react-native/lib/module/duotone/Equalizer.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Equalizer = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "equalizer-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M24 128h48v64H24Zm80 64h48v-32h-48Zm80-96v96h48V96Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M80 96a8 8 0 0 1-8 8H24a8 8 0 0 1 0-16h48a8 8 0 0 1 8 8Zm72 24h-48a8 8 0 0 0 0 16h48a8 8 0 0 0 0-16Zm32-48h48a8 8 0 0 0 0-16h-48a8 8 0 0 0 0 16ZM80 128v64a8 8 0 0 1-8 8H24a8 8 0 0 1-8-8v-64a8 8 0 0 1 8-8h48a8 8 0 0 1 8 8Zm-16 8H32v48h32Zm96 24v32a8 8 0 0 1-8 8h-48a8 8 0 0 1-8-8v-32a8 8 0 0 1 8-8h48a8 8 0 0 1 8 8Zm-16 8h-32v16h32Zm96-72v96a8 8 0 0 1-8 8h-48a8 8 0 0 1-8-8V96a8 8 0 0 1 8-8h48a8 8 0 0 1 8 8Zm-16 8h-32v80h32Z"
}));
export default Equalizer;
