// phosphor-react-native/lib/module/duotone/FadersHorizontal.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FadersHorizontal = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "faders-horizontal-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 80v96H40V80Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M32 80a8 8 0 0 1 8-8h32a8 8 0 0 1 0 16H40a8 8 0 0 1-8-8Zm184 88h-40v-16a8 8 0 0 0-16 0v48a8 8 0 0 0 16 0v-16h40a8 8 0 0 0 0-16Zm-80 0H40a8 8 0 0 0 0 16h96a8 8 0 0 0 0-16Zm-32-56a8 8 0 0 0 8-8V88h104a8 8 0 0 0 0-16H112V56a8 8 0 0 0-16 0v48a8 8 0 0 0 8 8Z"
}));
export default FadersHorizontal;
