// phosphor-react-native/lib/module/duotone/FireExtinguisher.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const FireExtinguisher = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "fire-extinguisher-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M176 168v64a8 8 0 0 1-8 8h-64a8 8 0 0 1-8-8v-64Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m218.3 48.34-60.68-18.2 30-15a8 8 0 0 0-7.2-14.29L134 24.05a80.08 80.08 0 0 0-78 80V208a8 8 0 0 0 16 0v-32h16v56a16 16 0 0 0 16 16h64a16 16 0 0 0 16-16V104a48.07 48.07 0 0 0-40-47.32V42.75l69.7 20.91a8 8 0 1 0 4.6-15.32ZM88 104v56H72v-56a64.07 64.07 0 0 1 56-63.48v16.16A48.07 48.07 0 0 0 88 104Zm80 128h-64v-56h64Zm0-128v56h-64v-56a32 32 0 0 1 64 0Z"
}));
export default FireExtinguisher;
