// phosphor-react-native/lib/module/duotone/GenderNonbinary.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GenderNonbinary = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "gender-nonbinary-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M192 168a64 64 0 1 1-64-64 64 64 0 0 1 64 64Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M136 96.46V62.13l27.88 16.73a8 8 0 1 0 8.24-13.72L143.55 48l28.57-17.14a8 8 0 0 0-8.24-13.72L128 38.67 92.12 17.14a8 8 0 0 0-8.24 13.72L112.45 48 83.88 65.14a8 8 0 0 0 8.24 13.72L120 62.13v34.33a72 72 0 1 0 16 0ZM128 224a56 56 0 1 1 56-56 56.06 56.06 0 0 1-56 56Z"
}));
export default GenderNonbinary;
