// phosphor-react-native/lib/module/duotone/GenderNeuter.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GenderNeuter = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "gender-neuter-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M200 104a72 72 0 1 1-72-72 72 72 0 0 1 72 72Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M208 104a80 80 0 1 0-88 79.6V232a8 8 0 0 0 16 0v-48.4a80.11 80.11 0 0 0 72-79.6Zm-80 64a64 64 0 1 1 64-64 64.07 64.07 0 0 1-64 64Z"
}));
export default GenderNeuter;
