// phosphor-react-native/lib/module/duotone/GenderMale.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GenderMale = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "gender-male-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M154.91 202.91a72 72 0 1 1 0-101.82 72 72 0 0 1 0 101.82Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M216 32h-48a8 8 0 0 0 0 16h28.69l-42.07 42.07a80 80 0 1 0 11.31 11.31L208 59.32V88a8 8 0 0 0 16 0V40a8 8 0 0 0-8-8Zm-66.76 165.29a64 64 0 1 1 0-90.53 64.1 64.1 0 0 1 0 90.53Z"
}));
export default GenderMale;
