// phosphor-react-native/lib/module/duotone/GlobeSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GlobeSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "globe-simple-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 128a96 96 0 1 1-96-96 96 96 0 0 1 96 96Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm87.63 96H175.8c-1.41-28.46-10.27-55.47-25.12-77a88.2 88.2 0 0 1 64.95 77Zm-119.4 16h63.54c-1.68 30.87-13 59.62-31.77 79.89-18.73-20.27-30.09-49.02-31.77-79.89Zm0-16c1.68-30.87 13-59.62 31.77-79.89 18.73 20.27 30.09 49 31.77 79.89Zm9.09-77C90.47 64.53 81.61 91.54 80.2 120H40.37a88.2 88.2 0 0 1 64.95-77Zm-64.95 93H80.2c1.41 28.46 10.27 55.47 25.12 77a88.2 88.2 0 0 1-64.95-77Zm110.31 77c14.85-21.56 23.71-48.57 25.12-77h39.83a88.2 88.2 0 0 1-64.95 77Z"
}));
export default GlobeSimple;
