// phosphor-react-native/lib/module/duotone/GoogleChromeLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const GoogleChromeLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "google-chrome-logo-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M168 128a40 40 0 1 1-40-40 40 40 0 0 1 40 40Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm0 16a88 88 0 0 1 73.72 40H128a48.08 48.08 0 0 0-45.6 33L59.32 73A87.89 87.89 0 0 1 128 40Zm32 88a32 32 0 1 1-32-32 32 32 0 0 1 32 32Zm-45.28 87A88 88 0 0 1 49.56 88.14L86.43 152c.06.1.13.19.19.28a48 48 0 0 0 51.2 22.72Zm18 .87L169.57 152c.08-.14.14-.28.22-.42a47.88 47.88 0 0 0-6-55.58H210a88 88 0 0 1-77.29 119.87Z"
}));
export default GoogleChromeLogo;
