// phosphor-react-native/lib/module/duotone/HighHeel.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const HighHeel = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "high-heel-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M248 195.31V200a8 8 0 0 1-8 8h-89.28a8 8 0 0 1-7.42-4.92C125.51 159.28 82.38 128 32 128c0-31 12.59-58.78 32-80l112 112 53.21 12A23.92 23.92 0 0 1 248 195.31Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m231 164.19-51-11.48L69.66 42.34a8 8 0 0 0-11.56.26C36.11 66.64 24 97 24 128v72a16 16 0 0 0 16 16h32a16 16 0 0 0 16-16v-48.94c2.49 1.45 4.94 3 7.34 4.64a112.45 112.45 0 0 1 40.55 50.39 15.9 15.9 0 0 0 14.83 9.91H240a16 16 0 0 0 16-16v-4.73a31.72 31.72 0 0 0-25-31.08ZM72 200H40v-63.71a110.88 110.88 0 0 1 32 7.12Zm168 0h-89.32a128.36 128.36 0 0 0-46.27-57.46 126.9 126.9 0 0 0-64.12-22.26 110.67 110.67 0 0 1 24.17-60.5l105.88 105.88a8 8 0 0 0 3.9 2.14l53.24 12A15.81 15.81 0 0 1 240 195.31Z"
}));
export default HighHeel;
