// phosphor-react-native/lib/module/duotone/HourglassSimpleHigh.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const HourglassSimpleHigh = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "hourglass-simple-high-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M186.48 72 128 128 69.52 72Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M211.18 196.56 139.57 128l71.61-68.56a1.59 1.59 0 0 1 .13-.13A16 16 0 0 0 200 32H56a16 16 0 0 0-11.31 27.31 1.59 1.59 0 0 1 .13.13L116.43 128l-71.61 68.56a1.59 1.59 0 0 1-.13.13A16 16 0 0 0 56 224h144a16 16 0 0 0 11.32-27.31 1.59 1.59 0 0 1-.14-.13ZM56 48Zm33.43 32h77.14L128 116.92ZM200 48l-16.7 16H72.72L56 48ZM56 208l72-68.92L200 208Z"
}));
export default HourglassSimpleHigh;
