// phosphor-react-native/lib/module/duotone/ListDashes.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ListDashes = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "list-dashes-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 64v128H40V64Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M88 64a8 8 0 0 1 8-8h120a8 8 0 0 1 0 16H96a8 8 0 0 1-8-8Zm128 56H96a8 8 0 1 0 0 16h120a8 8 0 0 0 0-16Zm0 64H96a8 8 0 1 0 0 16h120a8 8 0 0 0 0-16ZM56 56H40a8 8 0 0 0 0 16h16a8 8 0 0 0 0-16Zm0 64H40a8 8 0 1 0 0 16h16a8 8 0 0 0 0-16Zm0 64H40a8 8 0 1 0 0 16h16a8 8 0 0 0 0-16Z"
}));
export default ListDashes;
