// phosphor-react-native/lib/module/duotone/MarkerCircle.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const MarkerCircle = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "marker-circle-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M176 211.16Zm-96 0ZM224 128a96 96 0 1 0-144 83.16V176a8 8 0 0 1 8-8h8v-24a8 8 0 0 1 8-8h4.16l12-57.63c1.77-8.49 13.89-8.49 15.66 0l12 57.63H152a8 8 0 0 1 8 8v24h8a8 8 0 0 1 8 8v35.16A96 96 0 0 0 224 128Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M201.54 54.46A104 104 0 0 0 54.46 201.54 104 104 0 0 0 201.54 54.46ZM88 206.37V176h80v30.37a88.38 88.38 0 0 1-80 0ZM104 144h48v16h-48Zm14-16 10-48 10 48Zm72.22 62.23c-2 2-4.08 3.87-6.22 5.64V176a16 16 0 0 0-16-16v-16a16 16 0 0 0-13.61-15.8l-10.73-51.46a16 16 0 0 0-31.32 0l-10.73 51.46A16 16 0 0 0 88 144v16a16 16 0 0 0-16 16v19.87a88.917 88.917 0 0 1-6.22-5.64 88 88 0 1 1 124.44 0Z"
}));
export default MarkerCircle;
