// phosphor-react-native/lib/module/duotone/MouseSimple.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const MouseSimple = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "mouse-simple-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M200 80v96a56 56 0 0 1-56 56h-32a56 56 0 0 1-56-56V80a56 56 0 0 1 56-56h32a56 56 0 0 1 56 56Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M144 16h-32a64.07 64.07 0 0 0-64 64v96a64.07 64.07 0 0 0 64 64h32a64.07 64.07 0 0 0 64-64V80a64.07 64.07 0 0 0-64-64Zm48 160a48.05 48.05 0 0 1-48 48h-32a48.05 48.05 0 0 1-48-48V80a48.05 48.05 0 0 1 48-48h32a48.05 48.05 0 0 1 48 48ZM136 64v48a8 8 0 0 1-16 0V64a8 8 0 0 1 16 0Z"
}));
export default MouseSimple;
