// phosphor-react-native/lib/module/duotone/MusicNote.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const MusicNote = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "music-note-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 184a40 40 0 1 1-40-40 40 40 0 0 1 40 40Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m210.3 56.34-80-24A8 8 0 0 0 120 40v108.26A48 48 0 1 0 136 184V98.75l69.7 20.91A8 8 0 0 0 216 112V64a8 8 0 0 0-5.7-7.66ZM88 216a32 32 0 1 1 32-32 32 32 0 0 1-32 32Zm112-114.75-64-19.2v-31.3L200 70Z"
}));
export default MusicNote;
