// phosphor-react-native/lib/module/duotone/Notification.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Notification = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "notification-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 60a28 28 0 1 1-28-28 28 28 0 0 1 28 28Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M216 128v80a16 16 0 0 1-16 16H48a16 16 0 0 1-16-16V56a16 16 0 0 1 16-16h80a8 8 0 0 1 0 16H48v152h152v-80a8 8 0 0 1 16 0Zm16-68a36 36 0 1 1-36-36 36 36 0 0 1 36 36Zm-16 0a20 20 0 1 0-20 20 20 20 0 0 0 20-20Z"
}));
export default Notification;
