// phosphor-react-native/lib/module/duotone/NumberEight.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const NumberEight = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "number-eight-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 40v176a16 16 0 0 1-16 16H56a16 16 0 0 1-16-16V40a16 16 0 0 1 16-16h144a16 16 0 0 1 16 16Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M155.55 119.27a48 48 0 1 0-55.1 0 56 56 0 1 0 55.1 0ZM96 80a32 32 0 1 1 32 32 32 32 0 0 1-32-32Zm32 128a40 40 0 1 1 40-40 40 40 0 0 1-40 40Z"
}));
export default NumberEight;
