// phosphor-react-native/lib/module/duotone/NumberCircleEight.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const NumberCircleEight = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "number-circle-eight-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 128a96 96 0 1 1-96-96 96 96 0 0 1 96 96Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm0 192a88 88 0 1 1 88-88 88.1 88.1 0 0 1-88 88Zm21.81-92.61a32 32 0 1 0-43.62 0 36 36 0 1 0 43.62 0ZM112 100a16 16 0 1 1 16 16 16 16 0 0 1-16-16Zm16 72a20 20 0 1 1 20-20 20 20 0 0 1-20 20Z"
}));
export default NumberCircleEight;
