// phosphor-react-native/lib/module/duotone/PaperPlaneRight.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PaperPlaneRight = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "paper-plane-right-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M219.91 134.86 51.93 231a8 8 0 0 1-11.44-9.67l31-90.71a7.89 7.89 0 0 0 0-5.38l-31-90.47a8 8 0 0 1 11.44-9.67l168 95.85a8 8 0 0 1-.02 13.91Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m223.87 114-168-95.89a16 16 0 0 0-22.94 19.21l31 90.47a.42.42 0 0 0 0 .1.3.3 0 0 0 0 .1l-31 90.67A16 16 0 0 0 48 240a16.14 16.14 0 0 0 7.92-2.1l167.91-96.05a16 16 0 0 0 .05-27.89ZM48 224v-.09L78.14 136H136a8 8 0 0 0 0-16H78.22L48.06 32.12 48 32l168 95.83Z"
}));
export default PaperPlaneRight;
