// phosphor-react-native/lib/module/duotone/Parachute.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Parachute = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "parachute-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24s-40 24-40 96H32a96 96 0 0 1 96-96Zm0 0s40 24 40 96h56a96 96 0 0 0-96-96Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M232 120A104.12 104.12 0 0 0 128 16 104.12 104.12 0 0 0 24 120a8 8 0 0 0 3.12 6.33l.08.07L120 196v20h-8a8 8 0 0 0 0 16h32a8 8 0 0 0 0-16h-8v-20l92.8-69.6a8 8 0 0 0 3.2-6.4Zm-16.36-8h-39.81c-1.54-37.95-13.91-62.43-25.11-77a88.2 88.2 0 0 1 64.92 77ZM128 34a76.89 76.89 0 0 1 13.88 16.22C149.49 62 158.45 81.87 159.82 112H96.18c1.37-30.13 10.33-50 17.94-61.74A76.92 76.92 0 0 1 128 34Zm26.4 94L128 175.53 101.6 128Zm-71.11 0 19.5 35.09L56 128Zm89.42 0H200l-46.79 35.09Zm-67.43-93c-11.2 14.57-23.57 39.05-25.11 77H40.36a88.2 88.2 0 0 1 64.92-77Z"
}));
export default Parachute;
