// phosphor-react-native/lib/module/duotone/PhosphorLogo.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const PhosphorLogo = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "phosphor-logo-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 96a64 64 0 0 1-64 64h-8V32h8a64 64 0 0 1 64 64ZM64 160h72L64 32Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M144 24H64a8 8 0 0 0-8 8v128a80.09 80.09 0 0 0 80 80 8 8 0 0 0 8-8v-64a72 72 0 0 0 0-144ZM72 62.54 122.32 152H72Zm56 161A64.14 64.14 0 0 1 72.51 168H128Zm0-94L77.68 40H128ZM144 152V40a56 56 0 0 1 0 112Z"
}));
export default PhosphorLogo;
