// phosphor-react-native/lib/module/duotone/Pi.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Pi = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "pi-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M196 200H88V64h80v108a28 28 0 0 0 28 28Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M232 172a36 36 0 0 1-72 0V72H96v128a8 8 0 0 1-16 0V72h-8a40 40 0 0 0-40 40 8 8 0 0 1-16 0 56.06 56.06 0 0 1 56-56h152a8 8 0 0 1 0 16h-48v100a20 20 0 0 0 40 0 8 8 0 0 1 16 0Z"
}));
export default Pi;
