// phosphor-react-native/lib/module/duotone/ProjectorScreenChart.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const ProjectorScreenChart = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "projector-screen-chart-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 72v112H48V72Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M88 144v-16a8 8 0 0 1 16 0v16a8 8 0 0 1-16 0Zm40 8a8 8 0 0 0 8-8v-24a8 8 0 0 0-16 0v24a8 8 0 0 0 8 8Zm32 0a8 8 0 0 0 8-8v-32a8 8 0 0 0-16 0v32a8 8 0 0 0 8 8Zm56-72v96h8a8 8 0 0 1 0 16h-88v17.38a24 24 0 1 1-16 0V192H32a8 8 0 0 1 0-16h8V80a16 16 0 0 1-16-16V48a16 16 0 0 1 16-16h176a16 16 0 0 1 16 16v16a16 16 0 0 1-16 16Zm-80 152a8 8 0 1 0-8 8 8 8 0 0 0 8-8ZM40 64h176V48H40Zm160 16H56v96h144Z"
}));
export default ProjectorScreenChart;
