// phosphor-react-native/lib/module/duotone/RewindCircle.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const RewindCircle = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "rewind-circle-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M128 32a96 96 0 1 0 96 96 96 96 0 0 0-96-96Zm-16 128-48-32 48-32Zm64 0-48-32 48-32Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24Zm0 192a88 88 0 1 1 88-88 88.1 88.1 0 0 1-88 88Zm51.77-127a8 8 0 0 0-8.21.39l-48 32a8 8 0 0 0 0 13.32l48 32A8 8 0 0 0 176 168a8 8 0 0 0 8-8V96a8 8 0 0 0-4.23-7ZM168 145.05 142.42 128 168 111ZM115.77 89a8 8 0 0 0-8.21.39l-48 32a8 8 0 0 0 0 13.32l48 32A8 8 0 0 0 112 168a8 8 0 0 0 8-8V96a8 8 0 0 0-4.23-7ZM104 145.05 78.42 128 104 111Z"
}));
export default RewindCircle;
