// phosphor-react-native/lib/module/duotone/Scooter.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Scooter = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "scooter-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M72 172a28 28 0 1 1-28-28 28 28 0 0 1 28 28Zm140-28a28 28 0 1 0 28 28 28 28 0 0 0-28-28Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M212 136c-1.18 0-2.35.06-3.51.17l-10.75-32.25-22.15-66.45A8 8 0 0 0 168 32h-32a8 8 0 0 0 0 16h26.23l19 56.87L132.09 168H79.77a36 36 0 1 0-1.83 16H136a8 8 0 0 0 6.31-3.09l45-57.8 6 18.13A36 36 0 1 0 212 136ZM44 192a20 20 0 1 1 20-20 20 20 0 0 1-20 20Zm168 0a20 20 0 1 1 20-20 20 20 0 0 1-20 20Z"
}));
export default Scooter;
