// phosphor-react-native/lib/module/duotone/Share.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Share = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "share-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m224 104-48 48V56Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m229.66 98.34-48-48A8 8 0 0 0 168 56v40h-3a103.94 103.94 0 0 0-100.75 78 8 8 0 0 0 15.5 4A88 88 0 0 1 165 112h3v40a8 8 0 0 0 13.66 5.66l48-48a8 8 0 0 0 0-11.32ZM184 132.69V75.31L212.69 104ZM200 216a8 8 0 0 1-8 8H40a16 16 0 0 1-16-16V88a8 8 0 0 1 16 0v120h152a8 8 0 0 1 8 8Z"
}));
export default Share;
