// phosphor-react-native/lib/module/duotone/SignIn.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SignIn = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "sign-in-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m136 128-40 40V88Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m141.66 122.34-40-40A8 8 0 0 0 88 88v32H24a8 8 0 0 0 0 16h64v32a8 8 0 0 0 13.66 5.66l40-40a8 8 0 0 0 0-11.32ZM104 148.69v-41.38L124.69 128ZM208 48v160a16 16 0 0 1-16 16h-56a8 8 0 0 1 0-16h56V48h-56a8 8 0 0 1 0-16h56a16 16 0 0 1 16 16Z"
}));
export default SignIn;
