// phosphor-react-native/lib/module/duotone/SortAscending.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SortAscending = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "sort-ascending-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m224 168-40 40-40-40Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M128 128a8 8 0 0 1-8 8H48a8 8 0 0 1 0-16h72a8 8 0 0 1 8 8ZM48 72h136a8 8 0 0 0 0-16H48a8 8 0 0 0 0 16Zm56 112H48a8 8 0 0 0 0 16h56a8 8 0 0 0 0-16Zm125.66-10.34-40 40a8 8 0 0 1-11.32 0l-40-40A8 8 0 0 1 144 160h32v-48a8 8 0 0 1 16 0v48h32a8 8 0 0 1 5.66 13.66Zm-25 2.34h-41.35L184 196.69Z"
}));
export default SortAscending;
