// phosphor-react-native/lib/module/duotone/Sparkle.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Sparkle = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "sparkle-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m194.82 151.43-51.66 19a7.88 7.88 0 0 0-4.69 4.69l-19 51.66a7.92 7.92 0 0 1-14.86 0l-19-51.66a7.88 7.88 0 0 0-4.69-4.69l-51.66-19a7.92 7.92 0 0 1 0-14.86l51.66-19a7.88 7.88 0 0 0 4.69-4.69l19-51.66a7.92 7.92 0 0 1 14.86 0l19 51.66a7.88 7.88 0 0 0 4.69 4.69l51.66 19a7.92 7.92 0 0 1 0 14.86Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m197.58 129.06-51.61-19-19-51.65a15.92 15.92 0 0 0-29.88 0L78.07 110l-51.65 19a15.92 15.92 0 0 0 0 29.88L78 178l19 51.62a15.92 15.92 0 0 0 29.88 0l19-51.61 51.65-19a15.92 15.92 0 0 0 0-29.88ZM140.39 163a15.87 15.87 0 0 0-9.43 9.43l-19 51.46L93 172.39a15.87 15.87 0 0 0-9.39-9.39l-51.46-19 51.46-19a15.87 15.87 0 0 0 9.39-9.39l19-51.46 19 51.46a15.87 15.87 0 0 0 9.43 9.43l51.46 19ZM144 40a8 8 0 0 1 8-8h16V16a8 8 0 0 1 16 0v16h16a8 8 0 0 1 0 16h-16v16a8 8 0 0 1-16 0V48h-16a8 8 0 0 1-8-8Zm104 48a8 8 0 0 1-8 8h-8v8a8 8 0 0 1-16 0v-8h-8a8 8 0 0 1 0-16h8v-8a8 8 0 0 1 16 0v8h8a8 8 0 0 1 8 8Z"
}));
export default Sparkle;
