// phosphor-react-native/lib/module/duotone/SplitVertical.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const SplitVertical = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "split-vertical-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M160 56H96l32-32Zm-32 176 32-32H96Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M208 144H48a8 8 0 0 0 0 16h72v32H96a8 8 0 0 0-5.66 13.66l32 32a8 8 0 0 0 11.32 0l32-32A8 8 0 0 0 160 192h-24v-32h72a8 8 0 0 0 0-16Zm-80 76.69L115.31 208h25.38ZM48 112h160a8 8 0 0 0 0-16h-72V64h24a8 8 0 0 0 5.66-13.66l-32-32a8 8 0 0 0-11.32 0l-32 32A8 8 0 0 0 96 64h24v32H48a8 8 0 0 0 0 16Zm80-76.69L140.69 48h-25.38Z"
}));
export default SplitVertical;
