// phosphor-react-native/lib/module/duotone/StarFour.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const StarFour = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "star-four-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m226.76 135.48-63.45 23.07a8 8 0 0 0-4.76 4.76l-23.07 63.45a8 8 0 0 1-15 0l-23.03-63.45a8 8 0 0 0-4.76-4.76l-63.45-23.07a8 8 0 0 1 0-15l63.45-23.03a8 8 0 0 0 4.76-4.76l23.07-63.45a8 8 0 0 1 15 0l23.07 63.45a8 8 0 0 0 4.76 4.76l63.45 23.07a8 8 0 0 1-.04 14.96Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m229.5 113-63.43-23L143 26.5a16 16 0 0 0-30 0L90 89.93 26.5 113a16 16 0 0 0 0 30l63.43 23L113 229.5a16 16 0 0 0 30 0l23.07-63.44L229.5 143a16 16 0 0 0 0-30Zm-68.93 38a16 16 0 0 0-9.54 9.54L128 223.9l-23-63.33a16 16 0 0 0-9.57-9.57L32.1 128l63.33-23a16 16 0 0 0 9.57-9.57l23-63.33 23 63.33a16 16 0 0 0 9.54 9.54l63.33 23Z"
}));
export default StarFour;
