// phosphor-react-native/lib/module/duotone/Stop.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const Stop = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "stop-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M208 55.27v145.46a7.28 7.28 0 0 1-7.27 7.27H55.27a7.28 7.28 0 0 1-7.27-7.27V55.27A7.28 7.28 0 0 1 55.27 48h145.46a7.28 7.28 0 0 1 7.27 7.27Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M200.73 40H55.27A15.29 15.29 0 0 0 40 55.27v145.46A15.29 15.29 0 0 0 55.27 216h145.46A15.29 15.29 0 0 0 216 200.73V55.27A15.29 15.29 0 0 0 200.73 40ZM200 200H56V56h144Z"
}));
export default Stop;
