// phosphor-react-native/lib/module/duotone/TagChevron.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TagChevron = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "tag-chevron-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m230.66 132.44-42.67 64a8 8 0 0 1-6.66 3.56H24l46.65-67.56a8 8 0 0 0 0-8.88L24 56h157.33a8 8 0 0 1 6.67 3.56l42.67 64a8 8 0 0 1-.01 8.88Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "m237.3 119.12-42.66-64A16 16 0 0 0 181.33 48H24a8 8 0 0 0-6.58 12.55L64 128l-46.58 67.45A8 8 0 0 0 24 208h157.33a16 16 0 0 0 13.31-7.12l42.66-64a16 16 0 0 0 0-17.76Zm-56 72.88H39.25l38-55 .08-.1a16 16 0 0 0 0-17.76l-.08-.1-38-55h142.08L224 128Z"
}));
export default TagChevron;
