// phosphor-react-native/lib/module/duotone/TextB.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TextB = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "text-b-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M192 160a40 40 0 0 1-40 40H72V48h68a36 36 0 0 1 0 72h12a40 40 0 0 1 40 40Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M170.48 115.7A44 44 0 0 0 140 40H72a8 8 0 0 0-8 8v152a8 8 0 0 0 8 8h80a48 48 0 0 0 18.48-92.3ZM80 56h60a28 28 0 0 1 0 56H80Zm72 136H80v-64h72a32 32 0 0 1 0 64Z"
}));
export default TextB;
