// phosphor-react-native/lib/module/duotone/TextIndent.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TextIndent = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "text-indent-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m80 96-40 40V56Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M224 128a8 8 0 0 1-8 8H112a8 8 0 0 1 0-16h104a8 8 0 0 1 8 8ZM112 72h104a8 8 0 0 0 0-16H112a8 8 0 0 0 0 16Zm104 112H40a8 8 0 0 0 0 16h176a8 8 0 0 0 0-16ZM32 136V56a8 8 0 0 1 13.66-5.66l40 40a8 8 0 0 1 0 11.32l-40 40A8 8 0 0 1 32 136Zm16-19.31L68.69 96 48 75.31Z"
}));
export default TextIndent;
