// phosphor-react-native/lib/module/duotone/TextColumns.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TextColumns = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "text-columns-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M216 64v120H40V64Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M120 64a8 8 0 0 1-8 8H40a8 8 0 0 1 0-16h72a8 8 0 0 1 8 8Zm-8 32H40a8 8 0 0 0 0 16h72a8 8 0 0 0 0-16Zm0 40H40a8 8 0 0 0 0 16h72a8 8 0 0 0 0-16Zm0 40H40a8 8 0 0 0 0 16h72a8 8 0 0 0 0-16Zm32-104h72a8 8 0 0 0 0-16h-72a8 8 0 0 0 0 16Zm72 24h-72a8 8 0 0 0 0 16h72a8 8 0 0 0 0-16Zm0 40h-72a8 8 0 0 0 0 16h72a8 8 0 0 0 0-16Zm0 40h-72a8 8 0 0 0 0 16h72a8 8 0 0 0 0-16Z"
}));
export default TextColumns;
