// phosphor-react-native/lib/module/duotone/TrafficSign.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TrafficSign = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "traffic-sign-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M237.49 134.05 134.05 237.49a8.54 8.54 0 0 1-12.1 0L18.51 134.05a8.54 8.54 0 0 1 0-12.1L122 18.51a8.54 8.54 0 0 1 12.1 0L237.49 122a8.54 8.54 0 0 1 0 12.05Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M243.15 116.29 139.71 12.85a16.56 16.56 0 0 0-23.42 0L12.85 116.29a16.56 16.56 0 0 0 0 23.42l103.44 103.44a16.56 16.56 0 0 0 23.42 0l103.44-103.44a16.56 16.56 0 0 0 0-23.42Zm-11.31 12.1L128.39 231.84a.56.56 0 0 1-.78 0L24.16 128.39a.56.56 0 0 1 0-.78L127.61 24.16A.52.52 0 0 1 128 24a.58.58 0 0 1 .4.16l103.44 103.45a.56.56 0 0 1 0 .78Zm-58.18-14a8 8 0 0 1 0 11.32l-24 24a8 8 0 0 1-11.32-11.32L148.69 128H112a16 16 0 0 0-16 16v8a8 8 0 0 1-16 0v-8a32 32 0 0 1 32-32h36.69l-10.35-10.34a8 8 0 0 1 11.32-11.32Z"
}));
export default TrafficSign;
