// phosphor-react-native/lib/module/duotone/TreeEvergreen.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TreeEvergreen = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "tree-evergreen-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "m32 192 56-72H48l80-104 80 104h-40l56 72Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M230.31 187.09 184.36 128H208a8 8 0 0 0 6.34-12.88l-80-104a8 8 0 0 0-12.68 0l-80 104A8 8 0 0 0 48 128h23.64l-45.95 59.09A8 8 0 0 0 32 200h88v40a8 8 0 0 0 16 0v-40h88a8 8 0 0 0 6.31-12.91ZM48.36 184l46-59.09A8 8 0 0 0 88 112H64.25L128 29.12 191.75 112H168a8 8 0 0 0-6.31 12.91L207.64 184Z"
}));
export default TreeEvergreen;
