// phosphor-react-native/lib/module/duotone/TrendUp.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const TrendUp = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "trend-up-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M232 56v64l-64-64Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M232 48h-64a8 8 0 0 0-5.66 13.66L188.69 88 136 140.69l-34.34-34.35a8 8 0 0 0-11.32 0l-72 72a8 8 0 0 0 11.32 11.32L96 123.31l34.34 34.35a8 8 0 0 0 11.32 0L200 99.31l26.34 26.35A8 8 0 0 0 240 120V56a8 8 0 0 0-8-8Zm-8 52.69L187.31 64H224Z"
}));
export default TrendUp;
