// phosphor-react-native/lib/module/duotone/YinYang.js
function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }
/* GENERATED FILE */
import * as React from "react";
import Svg, { Path } from "react-native-svg";
const YinYang = props => /*#__PURE__*/React.createElement(Svg, _extends({
  className: "yin-yang-duotone__svg-icon-phosphor",
  fill: props.color,
  viewBox: "0 0 256 256",
  width: props.size,
  height: props.size
}, props), /*#__PURE__*/React.createElement(Path, {
  d: "M224 128a96 96 0 0 1-96 96 48 48 0 0 1 0-96 48 48 0 0 0 0-96 96 96 0 0 1 96 96Z",
  opacity: 0.2
}), /*#__PURE__*/React.createElement(Path, {
  d: "M128 24a104 104 0 1 0 104 104A104.11 104.11 0 0 0 128 24ZM40 128a88.1 88.1 0 0 1 88-88 40 40 0 0 1 0 80 56 56 0 0 0-50.61 80A88 88 0 0 1 40 128Zm88 88a40 40 0 0 1 0-80 56 56 0 0 0 50.61-79.95A88 88 0 0 1 128 216Zm12-40a12 12 0 1 1-12-12 12 12 0 0 1 12 12Zm-24-96a12 12 0 1 1 12 12 12 12 0 0 1-12-12Z"
}));
export default YinYang;
